// @adyen/bento-design-tokens/dist/js/bento/es6-styleguide.js
export const BColorBackgroundPrimary = "#ffffff";
export const BColorBackgroundPrimaryHover = "#f7f7f8";
export const BColorBackgroundPrimaryActive = "#edfaf3";
export const BColorBackgroundSecondary = "#f7f7f7";
export const BColorBackgroundTertiary = "#eeeff1";
export const BColorBackgroundTertiaryHover = "#e3e5e9";
export const BColorBackgroundTertiaryActive = "#dbdee2";
export const BColorBackgroundQuaternary = "#c0c5cc";
export const BColorBackgroundQuaternaryHover = "#b8bdc6";
export const BColorBackgroundQuaternaryActive = "#afb5bf";
export const BColorBackgroundModal = "#ffffff";
export const BColorBackgroundModalHover = "#f7f7f8";
export const BColorBackgroundModalActive = "#eeeff1";
export const BColorBackgroundDisabled = "#eeeff1";
export const BColorBackgroundSelected = "#e3f0ff";
export const BColorBackgroundCriticalWeak = "#fef4f4";
export const BColorBackgroundCriticalStrong = "#e22d2d";
export const BColorBackgroundCriticalStrongHover = "#c72727";
export const BColorBackgroundCriticalStrongActive = "#791818";
export const BColorBackgroundWarningWeak = "#fff5e9";
export const BColorBackgroundWarningStrong = "#dd7d00";
export const BColorBackgroundSuccessWeak = "#edfaf3";
export const BColorBackgroundSuccessStrong = "#07893c";
export const BColorBackgroundHighlightWeak = "#f2f8ff";
export const BColorBackgroundHighlightStrong = "#0070f5";
export const BColorBackgroundInverse = "#03401c";
export const BColorBackgroundAlwaysLight = "#ffffff";
export const BColorBackgroundAlwaysLightHover = "#f7f7f8";
export const BColorBackgroundAlwaysLightActive = "#eeeff1";
export const BColorBackgroundAlwaysLightDisabled = "#eeeff1";
export const BColorBackgroundAlwaysLightSelected = "#e3f0ff";
export const BColorBackgroundAlwaysDark = "#00112c";
export const BColorBackgroundAlwaysDarkHover = "#5c687c";
export const BColorBackgroundAlwaysDarkActive = "#8d95a3";
export const BColorBackgroundAlwaysDarkDisabled = "#eeeff1";
export const BColorBackgroundAlwaysDarkSelected = "#002f68";
export const BColorBackgroundTip1 = "#edfaf3";
export const BColorBackgroundTip2 = "#f2f8ff";
export const BColorBackgroundTip3 = "#faf5ff";
export const BColorBackgroundTip4 = "#fff7dd";
export const BColorBackgroundTip5 = "#fff5e9";
export const BColorBackgroundTip6 = "#fef4f4";
export const BColorBackgroundDark = "#00112c";
export const BColorLabelPrimary = "#00112c";
export const BColorLabelPrimaryHover = "#5c687c";
export const BColorLabelPrimaryActive = "#03401c";
export const BColorLabelSecondary = "#5c687c";
export const BColorLabelTertiary = "#8d95a3";
export const BColorLabelDisabled = "#8d95a3";
export const BColorLabelCritical = "#e22d2d";
export const BColorLabelWarning = "#b06300";
export const BColorLabelHighlight = "#0070f5";
export const BColorLabelSuccess = "#0abf53";
export const BColorLabelOnColor = "#ffffff";
export const BColorLabelInversePrimary = "#ffffff";
export const BColorLabelInversePrimaryHover = "#f7f7f8";
export const BColorLabelInversePrimaryActive = "#eeeff1";
export const BColorLabelInverseSecondary = "#9ea6b1";
export const BColorLabelInverseDisabled = "#747f8f";
export const BColorLabelInverseCritical = "#f99c9c";
export const BColorLabelInverseCriticalHover = "#fbbdbd";
export const BColorLabelInverseCriticalActive = "#fddfdf";
export const BColorLabelOnBackgroundCriticalWeak = "#c72727";
export const BColorLabelOnBackgroundWarningWeak = "#9b5700";
export const BColorLabelOnBackgroundSuccessWeak = "#067934";
export const BColorLabelOnBackgroundHighlightWeak = "#0063d7";
export const BColorLabelAlwaysLight = "#ffffff";
export const BColorLabelAlwaysDark = "#00112c";
export const BColorLabelCodeGreen = "#83dfa8";
export const BColorLabelCodeBlue = "#7db9ff";
export const BColorLabelCodeYellow = "#ffe48f";
export const BColorLabelCodeOrange = "#ffcc8a";
export const BColorLabelCodePurple = "#cb96ff";
export const BColorLabelCodeRed = "#f99c9c";
export const BColorLabelOnDarkPrimary = "#ffffff";
export const BColorLabelOnDarkSecondary = "#c5c5c5";
export const BColorLinkPrimary = "#00112c";
export const BColorLinkPrimaryHover = "#5c687c";
export const BColorLinkPrimaryActive = "#8d95a3";
export const BColorLinkPrimaryDisabled = "#8d95a3";
export const BColorLinkPrimaryHovered = "#5c687c";
export const BColorLinkPrimaryPressed = "#8d95a3";
export const BColorOutlinePrimary = "#eeeff1";
export const BColorOutlineSecondary = "#dbdee2";
export const BColorOutlineTertiary = "#8d95a3";
export const BColorOutlineTertiaryHover = "#6d7789";
export const BColorOutlineTertiaryActive = "#00112c";
export const BColorOutlineDisabled = "#dbdee2";
export const BColorOutlineSelected = "#00112c";
export const BColorOutlineCritical = "#e22d2d";
export const BColorOutlineCriticalHover = "#c72727";
export const BColorOutlineCriticalActive = "#791818";
export const BColorOutlineSuccess = "#07893c";
export const BColorOutlineSuccessHover = "#067934";
export const BColorOutlineSuccessActive = "#04481f";
export const BColorOutlineInversePrimary = "#2f3e56";
export const BColorOutlineInversePrimaryHover = "#6d7789";
export const BColorOutlineInversePrimaryActive = "#9ea6b1";
export const BColorOutlineInverseDisabled = "#455368";
export const BColorOutlineInverseCritical = "#f99c9c";
export const BColorOutlineInverseCriticalHover = "#fbbdbd";
export const BColorOutlineInverseCriticalActive = "#fddfdf";
export const BColorSeparatorPrimary = "#eeeff1";
export const BColorSeparatorSecondary = "#dbdee2";
export const BColorSeparatorInversePrimary = "#2f3e56";
export const BColorSeparatorInverseSecondary = "#5c687c";
export const BColorDecorativeGreen = "#09ab4b";
export const BColorDecorativeBlue = "#0070f5";
export const BColorDecorativeYellow = "#f7bc00";
export const BColorDecorativeOrange = "#dd7d00";
export const BColorDecorativeRed = "#e22d2d";
export const BColorDecorativeGrey = "#c0c5cc";
export const BColorSupportOverlay = "#00112c";
export const BColorSupportShadow = "#00112c";
export const BFocusRingColor = "#0070f5";
export const BFocusRingSpacer = "1px";
export const BFocusRingOutline = "3px";
export const BTextCaptionFontFamily = "Inter, Roboto, Oxygen, 'Open Sans', 'Helvetica Neue', Arial, sans-serif";
export const BTextCaptionFontSize = "12px";
export const BTextCaptionFontWeight = 400;
export const BTextCaptionLineHeight = "18px";
export const BTextCaptionLetterSpacing = 0;
export const BTextCaptionWideLineHeight = "20px";
export const BTextCaptionStrongerFontWeight = 500;
export const BTextCaptionStrongerWideFontWeight = 500;
export const BTextCaptionStrongerWideLineHeight = "20px";
export const BTextBodyFontFamily = "Inter, Roboto, Oxygen, 'Open Sans', 'Helvetica Neue', Arial, sans-serif";
export const BTextBodyFontSize = "14px";
export const BTextBodyFontWeight = 400;
export const BTextBodyLineHeight = "20px";
export const BTextBodyLetterSpacing = 0;
export const BTextBodyWideLineHeight = "24px";
export const BTextBodyStrongerFontWeight = 500;
export const BTextBodyStrongerWideFontWeight = 500;
export const BTextBodyStrongerWideLineHeight = "24px";
export const BTextBodyStrongestFontWeight = 600;
export const BTextBodyStrongestWideFontWeight = 600;
export const BTextBodyStrongestWideLineHeight = "24px";
export const BTextBodyCodeFontFamily = "Menlo";
export const BTextBodyCodeFontSize = "16px";
export const BTextBodyCodeFontWeight = 400;
export const BTextBodyCodeLineHeight = "24px";
export const BTextBodyCodeLetterSpacing = 0;
export const BTextBodyCodeTagLineHeight = "20px";
export const BTextBodyDocsFontFamily = "Inter, Roboto, Oxygen, 'Open Sans', 'Helvetica Neue', Arial, sans-serif";
export const BTextBodyDocsFontSize = "16px";
export const BTextBodyDocsFontWeight = 400;
export const BTextBodyDocsLineHeight = "24px";
export const BTextBodyDocsLetterSpacing = 0;
export const BTextBodyDocsStrongerFontWeight = 500;
export const BTextBodyDocsStrongerWideFontWeight = 500;
export const BTextBodyDocsStrongerWideLineHeight = "28px";
export const BTextBodyDocsStrongestFontWeight = 600;
export const BTextBodyDocsStrongestWideFontWeight = 600;
export const BTextBodyDocsStrongestWideLineHeight = "28px";
export const BTextSubtitleFontFamily = "Inter, Roboto, Oxygen, 'Open Sans', 'Helvetica Neue', Arial, sans-serif";
export const BTextSubtitleFontSize = "16px";
export const BTextSubtitleFontWeight = 500;
export const BTextSubtitleLineHeight = "26px";
export const BTextSubtitleLetterSpacing = 0;
export const BTextSubtitleStrongerFontWeight = 600;
export const BTextTitleFontFamily = "Inter, Roboto, Oxygen, 'Open Sans', 'Helvetica Neue', Arial, sans-serif";
export const BTextTitleFontSize = "24px";
export const BTextTitleFontWeight = 600;
export const BTextTitleLineHeight = "34px";
export const BTextTitleLetterSpacing = 0;
export const BTextTitleMobileFontSize = "24px";
export const BTextTitleMobileLineHeight = "34px";
export const BTextTitleMFontFamily = "Inter, Roboto, Oxygen, 'Open Sans', 'Helvetica Neue', Arial, sans-serif";
export const BTextTitleMFontSize = "28px";
export const BTextTitleMFontWeight = 600;
export const BTextTitleMLineHeight = "30px";
export const BTextTitleMLetterSpacing = 0;
export const BTextTitleMMobileFontSize = "28px";
export const BTextTitleMMobileLineHeight = "34px";
export const BTextTitleLFontFamily = "Inter, Roboto, Oxygen, 'Open Sans', 'Helvetica Neue', Arial, sans-serif";
export const BTextTitleLFontSize = "44px";
export const BTextTitleLFontWeight = 600;
export const BTextTitleLLineHeight = "52px";
export const BTextTitleLLetterSpacing = "-2%";
export const BTextTitleLMobileFontSize = "44px";
export const BTextTitleLMobileLineHeight = "52px";
export const BZIndexGlobalScreen = "auto";
export const BZIndexGlobalElevated = 5;
export const BZIndexGlobalNavigation = 10;
export const BZIndexGlobalInstantInteraction = 15;
export const BZIndexLocalTier1 = 1;
export const BZIndexLocalTier2 = 2;
export const BZIndexLocalTier3 = 3;
export const BZIndexLocalTier4 = 4;
export const BAccordionItemBackgroundColor = "#ffffff";
export const BAccordionItemBorder = "1px solid #dbdee2";
export const BAccordionItemButtonGap = "16px";
export const BAccordionItemButtonPadding = "12px 16px 12px 0";
export const BAccordionItemButtonHoverBackgroundColor = "#f7f7f8";
export const BAccordionItemButtonActiveBackgroundColor = "#edfaf3";
export const BAccordionItemContentPadding = "12px 16px 16px 32px";
export const BAccordionItemAnimationDuration = "150ms";
export const BAccordionItemAnimationEasing = "cubic-bezier(0.2, 0, 0.4, 0.9)";
export const BActionBarBottom = "16px";
export const BActionBarBottomExtraSpacing = "48px";
export const BActionBarZIndex = 5;
export const BActionBarContainerBackgroundColor = "#00112c";
export const BActionBarContainerBorderRadius = "28px";
export const BActionBarContainerColor = "#ffffff";
export const BActionBarContainerPadding = "10px 24px";
export const BActionBarContainerWidth = "calc(100% - 16px - 16px)";
export const BActionBarSelectionMarginRight = "40px";
export const BActionBarSelectionMarginRightMobile = "16px";
export const BActionBarCloseButtonMarginRight = "24px";
export const BActionBarCloseButtonMarginRightMobile = "16px";
export const BActionBarAnimationDuration = "150ms";
export const BAlertBorderRadius = "8px";
export const BAlertPadding = "20px";
export const BAlertPaddingMobile = "16px";
export const BAlertGap = "20px";
export const BAlertGapMobile = "16px";
export const BAlertIconHeight = "24px";
export const BAlertDescriptionMarginTop = "4px";
export const BAlertActionsMarginTop = "10px";
export const BAlertActionsGap = "16px";
export const BAlertSuccessBackgroundColor = "#edfaf3";
export const BAlertSuccessColor = "#00112c";
export const BAlertSuccessIconColor = "#07893c";
export const BAlertHighlightBackgroundColor = "#f2f8ff";
export const BAlertHighlightColor = "#00112c";
export const BAlertHighlightIconColor = "#0070f5";
export const BAlertWarningBackgroundColor = "#fff5e9";
export const BAlertWarningColor = "#00112c";
export const BAlertWarningIconColor = "#dd7d00";
export const BAlertCriticalBackgroundColor = "#fef4f4";
export const BAlertCriticalColor = "#00112c";
export const BAlertCriticalIconColor = "#e22d2d";
export const BAlertTipPadding = "10px 12px";
export const BAlertTipGap = "12px";
export const BAlertTipIconHeight = "20px";
export const BAvatarImageHeight = "32px";
export const BAvatarImageWidth = "32px";
export const BAvatarImageSmallHeight = "24px";
export const BAvatarImageSmallWidth = "24px";
export const BAvatarImageYellowBackgroundColor = "#fff5e9";
export const BAvatarImageYellowColor = "#dd7d00";
export const BAvatarImageBlueBackgroundColor = "#f2f8ff";
export const BAvatarImageBlueColor = "#0070f5";
export const BAvatarImageGreenBackgroundColor = "#edfaf3";
export const BAvatarImageGreenColor = "#09ab4b";
export const BAvatarImageRedBackgroundColor = "#fef4f4";
export const BAvatarImageRedColor = "#e22d2d";
export const BAvatarLabelColor = "#00112c";
export const BAvatarLabelPadding = "0 0 0 12px";
export const BButtonActionsWithMenuVerticalStackRowGap = "8px";
export const BButtonActionsButtonsEndColumnGap = "8px";
export const BButtonActionsFillContainerColumnGap = "8px";
export const BButtonActionsVerticalStackRowGap = "8px";
export const BButtonBorderRadius = "8px";
export const BButtonBoxShadow = "inset 0 0 0 1px #000000";
export const BButtonLineHeight = "20px";
export const BButtonPadding = "8px 12px";
export const BButtonTransition = "100ms linear";
export const BButtonPrimaryBackgroundColor = "#00112c";
export const BButtonPrimaryBackgroundColorInverse = "#ffffff";
export const BButtonPrimaryColor = "#ffffff";
export const BButtonPrimaryColorInverse = "#00112c";
export const BButtonPrimaryActiveBackgroundColor = "#8d95a3";
export const BButtonPrimaryActiveBackgroundColorInverse = "#edfaf3";
export const BButtonPrimaryDisabledBackgroundColor = "#eeeff1";
export const BButtonPrimaryDisabledBackgroundColorInverse = "#283750";
export const BButtonPrimaryDisabledColor = "#8d95a3";
export const BButtonPrimaryDisabledColorInverse = "#747f8f";
export const BButtonPrimaryDisabledBoxShadow = "inset 0 0 0 1px #eeeff1";
export const BButtonPrimaryDisabledBoxShadowInverse = "inset 0 0 0 1px #455368";
export const BButtonPrimaryHoverBackgroundColor = "#5c687c";
export const BButtonPrimaryHoverBackgroundColorInverse = "#f7f7f8";
export const BButtonPrimaryCriticalBackgroundColor = "#e22d2d";
export const BButtonPrimaryCriticalBackgroundColorInverse = "#f99c9c";
export const BButtonPrimaryCriticalColor = "#ffffff";
export const BButtonPrimaryCriticalColorInverse = "#00112c";
export const BButtonPrimaryCriticalActiveBackgroundColor = "#791818";
export const BButtonPrimaryCriticalActiveBackgroundColorInverse = "#fddfdf";
export const BButtonPrimaryCriticalDisabledBackgroundColor = "#eeeff1";
export const BButtonPrimaryCriticalDisabledBackgroundColorInverse = "#283750";
export const BButtonPrimaryCriticalDisabledBoxShadow = "inset 0 0 0 1px #eeeff1";
export const BButtonPrimaryCriticalDisabledBoxShadowInverse = "inset 0 0 0 1px #455368";
export const BButtonPrimaryCriticalDisabledColor = "#8d95a3";
export const BButtonPrimaryCriticalDisabledColorInverse = "#747f8f";
export const BButtonPrimaryCriticalHoverBackgroundColor = "#c72727";
export const BButtonPrimaryCriticalHoverBackgroundColorInverse = "#fbbdbd";
export const BButtonSecondaryBackgroundColor = "#ffffff";
export const BButtonSecondaryBackgroundColorInverse = "#00112c";
export const BButtonSecondaryBorder = "1px solid #c9cdd3";
export const BButtonSecondaryBorderInverse = "1px solid #2f3e56";
export const BButtonSecondaryBoxShadow = "inset 0 0 0 1px #c9cdd3";
export const BButtonSecondaryBoxShadowInverse = "inset 0 0 0 1px #2f3e56";
export const BButtonSecondaryColor = "#00112c";
export const BButtonSecondaryColorInverse = "#ffffff";
export const BButtonSecondaryPadding = "calc(8px - 1px) calc(12px - 1px)";
export const BButtonSecondaryActiveBackgroundColor = "#edfaf3";
export const BButtonSecondaryActiveBackgroundColorInverse = "#8d95a3";
export const BButtonSecondaryActiveBorder = "1px solid #c9cdd3";
export const BButtonSecondaryActiveBorderInverse = "1px solid #9ea6b1";
export const BButtonSecondaryActiveBoxShadow = "inset 0 0 0 1px #c9cdd3";
export const BButtonSecondaryActiveBoxShadowInverse = "inset 0 0 0 1px #9ea6b1";
export const BButtonSecondaryActiveColor = "#03401c";
export const BButtonSecondaryActiveColorInverse = "#ffffff";
export const BButtonSecondaryHoverBackgroundColor = "#f7f7f8";
export const BButtonSecondaryHoverBackgroundColorInverse = "#5c687c";
export const BButtonSecondaryHoverBorder = "1px solid #c9cdd3";
export const BButtonSecondaryHoverBorderInverse = "1px solid #6d7789";
export const BButtonSecondaryHoverBoxShadow = "inset 0 0 0 1px #c9cdd3";
export const BButtonSecondaryHoverBoxShadowInverse = "inset 0 0 0 1px #6d7789";
export const BButtonSecondaryHoverColor = "#5c687c";
export const BButtonSecondaryHoverColorInverse = "#ffffff";
export const BButtonSecondaryDisabledBackgroundColor = "#ffffff";
export const BButtonSecondaryDisabledBackgroundColorInverse = "#00112c";
export const BButtonSecondaryDisabledBorder = "1px solid #dbdee2";
export const BButtonSecondaryDisabledBorderInverse = "1px solid #455368";
export const BButtonSecondaryDisabledBoxShadow = "inset 0 0 0 1px #dbdee2";
export const BButtonSecondaryDisabledBoxShadowInverse = "inset 0 0 0 1px #455368";
export const BButtonSecondaryDisabledColor = "#8d95a3";
export const BButtonSecondaryDisabledColorInverse = "#747f8f";
export const BButtonSecondaryCriticalBackgroundColor = "#ffffff";
export const BButtonSecondaryCriticalBackgroundColorInverse = "#00112c";
export const BButtonSecondaryCriticalBorder = "1px solid #e22d2d";
export const BButtonSecondaryCriticalBorderInverse = "1px solid #f99c9c";
export const BButtonSecondaryCriticalBoxShadow = "inset 0 0 0 1px #e22d2d";
export const BButtonSecondaryCriticalBoxShadowInverse = "inset 0 0 0 1px #f99c9c";
export const BButtonSecondaryCriticalColor = "#e22d2d";
export const BButtonSecondaryCriticalColorInverse = "#f99c9c";
export const BButtonSecondaryCriticalActiveBackgroundColor = "#fef4f4";
export const BButtonSecondaryCriticalActiveBackgroundColorInverse = "#611313";
export const BButtonSecondaryCriticalActiveBorder = "1px solid #791818";
export const BButtonSecondaryCriticalActiveBorderInverse = "1px solid #fddfdf";
export const BButtonSecondaryCriticalActiveBoxShadow = "inset 0 0 0 1px #791818";
export const BButtonSecondaryCriticalActiveBoxShadowInverse = "inset 0 0 0 1px #fddfdf";
export const BButtonSecondaryCriticalActiveColor = "#791818";
export const BButtonSecondaryCriticalActiveColorInverse = "#fddfdf";
export const BButtonSecondaryCriticalDisabledBackgroundColor = "#ffffff";
export const BButtonSecondaryCriticalDisabledBackgroundColorInverse = "#00112c";
export const BButtonSecondaryCriticalDisabledBorder = "1px solid #dbdee2";
export const BButtonSecondaryCriticalDisabledBorderInverse = "1px solid #455368";
export const BButtonSecondaryCriticalDisabledBoxShadow = "inset 0 0 0 1px #dbdee2";
export const BButtonSecondaryCriticalDisabledBoxShadowInverse = "inset 0 0 0 1px #455368";
export const BButtonSecondaryCriticalDisabledColor = "#8d95a3";
export const BButtonSecondaryCriticalDisabledColorInverse = "#747f8f";
export const BButtonSecondaryCriticalHoverBackgroundColor = "#fef4f4";
export const BButtonSecondaryCriticalHoverBackgroundColorInverse = "#611313";
export const BButtonSecondaryCriticalHoverBorder = "1px solid #c72727";
export const BButtonSecondaryCriticalHoverBorderInverse = "1px solid #fbbdbd";
export const BButtonSecondaryCriticalHoverBoxShadow = "inset 0 0 0 1px #c72727";
export const BButtonSecondaryCriticalHoverBoxShadowInverse = "inset 0 0 0 1px #fbbdbd";
export const BButtonSecondaryCriticalHoverColor = "#c72727";
export const BButtonSecondaryCriticalHoverColorInverse = "#fbbdbd";
export const BButtonTertiaryBackgroundColor = "#000000";
export const BButtonTertiaryBorderRadius = "2px";
export const BButtonTertiaryBoxShadow = "inset 0 0 0 1px #000000";
export const BButtonTertiaryColor = "#00112c";
export const BButtonTertiaryColorInverse = "#ffffff";
export const BButtonTertiaryHeight = "auto";
export const BButtonTertiaryPadding = "0px";
export const BButtonTertiaryTextDecoration = "underline";
export const BButtonTertiaryHoverBackgroundColor = "#000000";
export const BButtonTertiaryHoverColor = "#5c687c";
export const BButtonTertiaryHoverColorInverse = "#f7f7f8";
export const BButtonTertiaryHoverTextDecoration = "none";
export const BButtonTertiaryActiveBackgroundColor = "#000000";
export const BButtonTertiaryActiveColor = "#03401c";
export const BButtonTertiaryActiveColorInverse = "#eeeff1";
export const BButtonTertiaryActiveTextDecoration = "none";
export const BButtonTertiaryDisabledBackgroundColor = "#000000";
export const BButtonTertiaryDisabledColor = "#8d95a3";
export const BButtonTertiaryDisabledColorInverse = "#747f8f";
export const BButtonTertiaryCriticalBackgroundColor = "#000000";
export const BButtonTertiaryCriticalColor = "#e22d2d";
export const BButtonTertiaryCriticalColorInverse = "#f99c9c";
export const BButtonTertiaryCriticalActiveBackgroundColor = "#000000";
export const BButtonTertiaryCriticalActiveColor = "#791818";
export const BButtonTertiaryCriticalActiveColorInverse = "#fddfdf";
export const BButtonTertiaryCriticalDisabledBackgroundColor = "#000000";
export const BButtonTertiaryCriticalDisabledColor = "#8d95a3";
export const BButtonTertiaryCriticalDisabledColorInverse = "#747f8f";
export const BButtonTertiaryCriticalHoverBackgroundColor = "#000000";
export const BButtonTertiaryCriticalHoverBoxShadow = "unset";
export const BButtonTertiaryCriticalHoverColor = "#c72727";
export const BButtonTertiaryCriticalHoverColorInverse = "#fbbdbd";
export const BButtonCondensedBorderRadius = "8px";
export const BButtonCondensedFontSize = "14px";
export const BButtonCondensedLineHeight = "20px";
export const BButtonCondensedPadding = "4px 10px";
export const BButtonCondensedIconHeight = "auto";
export const BButtonCondensedIconWidth = "auto";
export const BButtonCondensedSecondaryPadding = "calc(4px - 1px) calc(10px - 1px)";
export const BButtonCondensedTertiaryHeight = "auto";
export const BButtonCondensedTertiaryPadding = "0px";
export const BButtonIconOnlyPadding = "10px";
export const BButtonIconOnlyCondensedPadding = "6px";
export const BButtonIconOnlySecondaryPadding = "calc(10px - 1px)";
export const BButtonIconOnlySecondaryPaddingCondensed = "calc(6px - 1px)";
export const BCalendarRangeGap = "24px";
export const BCalendarMonthDayOfTheWeekColor = "#5c687c";
export const BCalendarMonthDayOfTheWeekPaddingBottom = "6px";
export const BCalendarMonthMonthSelectorMarginBottom = "24px";
export const BCalendarMonthDayHeight = "40px";
export const BCalendarMonthDayWidth = "40px";
export const BCalendarMonthDayBeginningBorderRadius = "4px 0 0 4px";
export const BCalendarMonthDayCurrentDaySelectedColor = "#00112c";
export const BCalendarMonthDayCurrentDaySelectedWrapperBackground = "#ffffff";
export const BCalendarMonthDayCurrentDayUnselectedHoverWrapperBackground = "#ffffff";
export const BCalendarMonthDayCurrentDayUnselectedWrapperBackground = "#eeeff1";
export const BCalendarMonthDayDisabledSelectedColor = "#8d95a3";
export const BCalendarMonthDayDisabledSelectedBackground = "#eeeff1";
export const BCalendarMonthDayDisabledUnselectedColor = "#8d95a3";
export const BCalendarMonthDayEndBorderRadius = "0 4px 4px 0";
export const BCalendarMonthDayMiddleBackground = "#eeeff1";
export const BCalendarMonthDayMiddleColor = "#00112c";
export const BCalendarMonthDayMiddleHoverBackground = "#f7f7f8";
export const BCalendarMonthDayMiddleActiveBackground = "#ffffff";
export const BCalendarMonthDaySelectedBackground = "#00112c";
export const BCalendarMonthDaySelectedBorderRadius = "4px";
export const BCalendarMonthDaySelectedColor = "#ffffff";
export const BCalendarMonthDaySelectedHoverBackground = "#5c687c";
export const BCalendarMonthDaySelectedActiveBackground = "#8d95a3";
export const BCalendarMonthDayUnselectedHoverBackground = "#f7f7f8";
export const BCalendarMonthDayUnselectedActiveBackground = "#edfaf3";
export const BCardBackgroundColor = "#ffffff";
export const BCardBorderColor = "#dbdee2";
export const BCardBorderRadius = "12px";
export const BCardBorderWidth = "1px";
export const BCardHeaderBorderRadius = "12px";
export const BCardHeaderColor = "#00112c";
export const BCardHeaderMarginTop = "4px";
export const BCardHeaderPadding = "20px 24px 0px";
export const BCardHeaderPaddingBottom = "20px";
export const BCardHeaderHoverBackgroundColor = "#f7f7f8";
export const BCardHeaderActiveBackgroundColor = "#edfaf3";
export const BCardHeaderFocusBackgroundColor = "#f7f7f8";
export const BCardToggleHeight = "26px";
export const BCardToggleSmallTextHeight = "24px";
export const BCardBodyPadding = "0px 24px 20px";
export const BCardBodyPaddingTop = "4px";
export const BCardBodyExpandablePaddingTop = "4px";
export const BCardContentPadding = "16px 0px 0px";
export const BCardContentPaddingTop = "20px";
export const BCardDescriptionColor = "#5c687c";
export const BCardDescriptionMarginLeft = "32px";
export const BCardActionsMarginTop = "16px";
export const BCardActionsPaddingTop = "20px";
export const BCardActionsWithoutContentPaddingTop = "16px";
export const BCardExpandableHeaderPaddingBottom = "20px";
export const BCardDisabledBackgroundColor = "#f7f7f8";
export const BCardDisabledColor = "#8d95a3";
export const BCardSecondaryBackgroundColor = "#f7f7f8";
export const BCardSecondaryHoverBackgroundColor = "#eeeff1";
export const BCardSecondaryHeaderHoverBackgroundColor = "#eeeff1";
export const BCardSecondaryHeaderActiveBackgroundColor = "#e3e5e9";
export const BCardSecondaryHeaderFocusBackgroundColor = "#eeeff1";
export const BCardTitleWrapperGap = "16px";
export const BCardAnimationDuration = "150ms";
export const BCardAnimationEasing = "cubic-bezier(0.2, 0, 0.4, 0.9)";
export const BCheckboxGroupCheckboxesVerticalGap = "16px";
export const BCheckboxGroupCheckboxesHorizontalGap = "32px";
export const BCheckboxGroupErrorMessageMarginTop = "12px";
export const BCheckboxDescriptionColor = "#5c687c";
export const BCheckboxDescriptionDisabledColor = "#5c687c";
export const BCheckboxContentPadding = "12px 0px 8px";
export const BCheckboxIconColor = "#ffffff";
export const BCheckboxIconDisabledColor = "#8d95a3";
export const BCheckboxIconReadonlyColor = "#00112c";
export const BCheckboxOutlineBorderRadius = "4px";
export const BCheckboxOutlineBorderWidth = "1px";
export const BCheckboxOutlineTopOffset = "2px";
export const BCheckboxOutlineCheckedBackgroundColor = "#00112c";
export const BCheckboxOutlineCheckedBorderColor = "#00112c";
export const BCheckboxOutlineCheckedActiveBackgroundColor = "#8d95a3";
export const BCheckboxOutlineCheckedActiveBorderColor = "#8d95a3";
export const BCheckboxOutlineCheckedDisabledBackgroundColor = "#eeeff1";
export const BCheckboxOutlineCheckedDisabledBorderColor = "#dbdee2";
export const BCheckboxOutlineCheckedHoverBackgroundColor = "#5c687c";
export const BCheckboxOutlineCheckedHoverBorderColor = "#5c687c";
export const BCheckboxOutlineCheckedErrorBackgroundColor = "#e22d2d";
export const BCheckboxOutlineCheckedErrorBorderColor = "#e22d2d";
export const BCheckboxOutlineIndeterminateBackgroundColor = "#00112c";
export const BCheckboxOutlineIndeterminateBorderColor = "#00112c";
export const BCheckboxOutlineIndeterminateActiveBackgroundColor = "#8d95a3";
export const BCheckboxOutlineIndeterminateActiveBorderColor = "#8d95a3";
export const BCheckboxOutlineIndeterminateDisabledBackgroundColor = "#eeeff1";
export const BCheckboxOutlineIndeterminateDisabledBorderColor = "#dbdee2";
export const BCheckboxOutlineIndeterminateHoverBackgroundColor = "#5c687c";
export const BCheckboxOutlineIndeterminateHoverBorderColor = "#5c687c";
export const BCheckboxOutlineIndeterminateErrorBackgroundColor = "#e22d2d";
export const BCheckboxOutlineIndeterminateErrorBorderColor = "#e22d2d";
export const BCheckboxOutlineUncheckedBackgroundColor = "#ffffff";
export const BCheckboxOutlineUncheckedBorderColor = "#8d95a3";
export const BCheckboxOutlineUncheckedActiveBackgroundColor = "#edfaf3";
export const BCheckboxOutlineUncheckedActiveBorderColor = "#8d95a3";
export const BCheckboxOutlineUncheckedDisabledBackgroundColor = "#eeeff1";
export const BCheckboxOutlineUncheckedDisabledBorderColor = "#dbdee2";
export const BCheckboxOutlineUncheckedHoverBackgroundColor = "#f7f7f8";
export const BCheckboxOutlineUncheckedHoverBorderColor = "#8d95a3";
export const BCheckboxOutlineUncheckedErrorBackgroundColor = "#e22d2d";
export const BCheckboxOutlineUncheckedErrorBorder = "1.5px solid #e22d2d";
export const BCheckboxOutlineReadonlyBackgroundColor = "#eeeff1";
export const BCheckboxOutlineReadonlyBorderColor = "#dbdee2";
export const BCheckboxTextColor = "#00112c";
export const BCheckboxTextGap = "4px";
export const BCheckboxTextPaddingLeft = "16px";
export const BCodeSnippetActionBarBackgroundColor = "#eeeff1";
export const BCodeSnippetActionBarPadding = "8px 12px";
export const BCodeSnippetActionBarGap = "8px";
export const BCodeSnippetActionBarBorder = "0 solid #dbdee2";
export const BCodeSnippetActionBarBorderRadius = "8px 8px 0 0";
export const BCodeSnippetLanguagePickerBackgroundColor = "#ffffff";
export const BCodeSnippetActionButtonPadding = "0 4px 0 4px";
export const BCodeSnippetCodeAreaBackgroundColor = "#f7f7f8";
export const BCodeSnippetCodeAreaMargin = 0;
export const BCodeSnippetCodeAreaBorderColor = "#dbdee2";
export const BCodeSnippetCodeAreaBorderRadius = "0 0 8px 8px";
export const BCodeSnippetCodeAreaBorderWidth = "1px";
export const BCodeSnippetCodeAreaContentFontFamily = "Menlo, 'Roboto Mono', 'Oxygen Mono', 'IBM Plex Mono', 'Courier New', monospace";
export const BCodeSnippetCodeAreaContentPadding = "12px";
export const BCodeSnippetCodeAreaContentPaddingLeft = "8px";
export const BCodeSnippetCodeAreaGuttersPaddingLeft = "12px";
export const BCodeSnippetCodeAreaHighlightRed = "#c72727";
export const BCodeSnippetCodeAreaHighlightOrange = "#9b5700";
export const BCodeSnippetCodeAreaHighlightGreen = "#067934";
export const BCodeSnippetCodeAreaHighlightBlue = "#0063d7";
export const BCodeSnippetEmptyStateBackgroundColor = "#f7f7f8";
export const BCodeSnippetEmptyStateMinHeight = "200px";
export const BCodeSnippetInlineCodeAreaContentPadding = "12px 8px 12px 8px";
export const BControlsGroupDescriptionColor = "#5c687c";
export const BControlsGroupLabelContainerGap = "4px";
export const BCountryLabelColor = "#00112c";
export const BDataGridTemplateGap = "12px";
export const BDataGridTemplateHeaderMargin = "20px 24px 0px 24px";
export const BDataGridTemplateHeaderZIndex = 3;
export const BDataGridTemplateAdditionalContentMargin = "0px 24px 0px 24px";
export const BDataGridTemplateTabsBackgroundColor = "#ffffff";
export const BDataGridTemplateTabsPadding = "0px 24px";
export const BDataGridTemplateDataGridFiltersPaddingRight = "24px";
export const BDataGridTemplateDataGridColumnsPaddingLeft = "10px";
export const BDataGridTemplateDataGridColumnsPaddingRight = "10px";
export const BDataGridTemplateDataGridCellsPaddingLeft = "10px";
export const BDataGridTemplateDataGridCellsPaddingRight = "10px";
export const BDataGridTemplateDataGridPaginationPaddingLeft = "24px";
export const BDataGridTemplateDataGridPaginationPaddingRight = "24px";
export const BDataGridBorderColor = "#eeeff1";
export const BDataGridBorderRadius = "8px";
export const BDataGridBorderWidth = "1px";
export const BDataGridHeaderPadding = "20px 24px";
export const BDataGridColumnHeadersZIndex = 2;
export const BDataGridColumnHeadersBackgroundColor = "#ffffff";
export const BDataGridColumnHeadersFocusWithinZIndex = 3;
export const BDataGridColumnsBorderColor = "#dbdee2";
export const BDataGridColumnsBorderWidth = "1px";
export const BDataGridColumnsFrozenBorderRightColor = "#dbdee2";
export const BDataGridColumnsFrozenBorderRightWidth = "1px";
export const BDataGridColumnsFrozenZIndex = 2;
export const BDataGridColumnsSelectColumnPaddingLeft = "8px";
export const BDataGridColumnsSelectColumnPaddingRight = "4px";
export const BDataGridColumnsSelectColumnGap = "4px";
export const BDataGridColumnBackgroundColor = "#ffffff";
export const BDataGridColumnBorderColor = "#000000";
export const BDataGridColumnHeight = "auto";
export const BDataGridColumnPadding = "14px 16px 13px 16px";
export const BDataGridColumnColor = "#00112c";
export const BDataGridColumnHoverBackgroundColor = "#eeeff1";
export const BDataGridColumnHoverSortIconColor = "#8d95a3";
export const BDataGridColumnSortPadding = "0 0 0 8px";
export const BDataGridColumnSortIconColor = "#8d95a3";
export const BDataGridColumnSortIconMarginTop = "2px";
export const BDataGridColumnSortIconDescendingIconMarginTop = "8px";
export const BDataGridColumnSeparatorBorderRightColor = "#eeeff1";
export const BDataGridColumnSeparatorBorderRightWidth = "1px";
export const BDataGridColumnSeparatorHeight = "24px";
export const BDataGridColumnSeparatorResizeableHoverColor = "#dbdee2";
export const BDataGridColumnSeparatorZIndex = 1;
export const BDataGridColumnHeaderGap = "8px";
export const BDataGridColumnDraggableGhostBackgroundColor = "#e3f0ff";
export const BDataGridColumnActionsGap = "16px";
export const BDataGridColumnActionsPaddingLeft = "12px";
export const BDataGridRowsFrozenBorderRightColor = "#dbdee2";
export const BDataGridRowsFrozenBorderRightWidth = "1px";
export const BDataGridRowsFrozenZIndex = 1;
export const BDataGridRowBackgroundColor = "#ffffff";
export const BDataGridRowBorderColor = "#eeeff1";
export const BDataGridRowBorderWidth = "1px";
export const BDataGridRowColor = "#00112c";
export const BDataGridRowHoverBackgroundColor = "#f7f7f8";
export const BDataGridRowDisabledColor = "#00112c";
export const BDataGridRowHighlightedBackgroundColor = "#e3f0ff";
export const BDataGridCellBackgroundColor = "#ffffff";
export const BDataGridCellPadding = "14px 16px";
export const BDataGridCellCondensedPadding = "6px 16px";
export const BDataGridCellActionsAnimationDuration = "100ms";
export const BDataGridCellActionsBackgroundGradientBackground = "linear-gradient(270deg, #f7f7f8 0%, rgba(247, 247, 248, 0.99) 8.59%, rgba(247, 247, 248, 0.98) 17.48%, rgba(247, 247, 248, 0.95) 26.53%, rgba(247, 247, 248, 0.92) 35.61%, rgba(247, 247, 248, 0.87) 44.59%, rgba(247, 247, 248, 0.82) 53.34%, rgba(247, 247, 248, 0.75) 61.73%, rgba(247, 247, 248, 0.68) 69.63%, rgba(247, 247, 248, 0.60) 76.9%, rgba(247, 247, 248, 0.52) 83.41%, rgba(247, 247, 248, 0.42) 89.03%, rgba(247, 247, 248, 0.33) 93.63%, rgba(247, 247, 248, 0.22) 97.08%, rgba(247, 247, 248, 0.11) 99.25%, rgba(247, 247, 248, 0.00) 100%)";
export const BDataGridCellActionsBackgroundGradientWidth = "32px";
export const BDataGridCellActionsGap = "8px";
export const BDataGridCellActionsPadding = "0 16px 0 0";
export const BDataGridSelectCellMarginLeft = "8px";
export const BDataGridResizerWidth = "8px";
export const BDataGridFooterPadding = "8px 24px";
export const BDataGridLazyLoadBackgroundColor = "#ffffff";
export const BDataGridLazyLoadZIndex = 2;
export const BDataGridLazyLoadBorder = "1px solid #eeeff1";
export const BDataGridLazyLoadPadding = "16px 16px 0px 16px";
export const BDataGridPaginationBackgroundColor = "#ffffff";
export const BDataGridPaginationZIndex = 2;
export const BDataGridPaginationBorderColor = "#eeeff1";
export const BDataGridPaginationBorderWidth = "1px";
export const BDataGridPaginationPadding = "16px 16px 0px 16px";
export const BDataGridPaginationOutlinePadding = "16px";
export const BDataGridPaginationStickyPadding = "16px";
export const BDataGridFiltersBorderColor = "#eeeff1";
export const BDataGridFiltersBorderWidth = "1px";
export const BDataGridFiltersPadding = "16px 0 16px 24px";
export const BDataGridFiltersBackgroundColor = "#ffffff";
export const BDataGridFiltersZIndex = 2;
export const BDataGridFiltersFocusWithinZIndex = 3;
export const BDataGridLoadingIndicatorMargin = "0 0 12px 0";
export const BDataGridConfigSettingsZIndex = 4;
export const BDataGridConfigSettingsButtonAndSeparatorBackgroundColor = "#ffffff";
export const BDataGridConfigSettingsButtonAndSeparatorPadding = 0;
export const BDataGridConfigSettingsButtonMargin = "16px 0";
export const BDataGridConfigSettingsButtonPadding = "0 16px";
export const BDataGridConfigSettingsColumnAndRowMargin = "48px";
export const BDataGridConfigCondensedViewBorderColor = "#eeeff1";
export const BDataGridConfigCondensedViewPadding = "0 16px 12px 16px";
export const BDataGridConfigCondensedViewTitlePadding = "0 0 12px 0";
export const BDataGridConfigPanelHeaderPadding = "0 16px 12px 16px";
export const BDataGridConfigColumnVisibilityPadding = "16px 0 0 0";
export const BDataGridConfigColumnVisibilityHeaderPadding = "0 16px 12px 16px";
export const BDataGridConfigColumnVisibilityShowAllBackgroundColor = "#f7f7f8";
export const BDataGridConfigColumnVisibilityShowAllBorderRadius = "8px";
export const BDataGridConfigColumnVisibilityShowAllMargin = "0 8px 4px 8px";
export const BDataGridConfigColumnVisibilityShowAllPadding = "8px";
export const BDataGridConfigColumnVisibilitySearchMargin = "0 16px 4px 16px";
export const BDataGridConfigColumnVisibilityEmptySearchMessageColor = "#5c687c";
export const BDataGridConfigColumnVisibilityEmptySearchMessagePadding = "12px 16px 0px";
export const BDataGridConfigColumnRowPadding = "8px 16px";
export const BDataGridConfigColumnRowHoverBackgroundColor = "#f7f7f8";
export const BDataGridConfigColumnRowFreezeIconColor = "#8d95a3";
export const BDataGridConfigColumnRowFreezeIconFrozenColor = "#00112c";
export const BDataGridReOrderColumnTargetSeparatorBackgroundColor = "#0070f5";
export const BDataGridReOrderColumnTargetSeparatorWidth = "3px";
export const BDataGridReOrderColumnTargetSeparatorZIndex = 1;
export const BDataGridEmptyStatePadding = "32px 0";
export const BDatePickerDescriptionColor = "#5c687c";
export const BDatePickerDescriptionMarginTop = "8px";
export const BDatePickerDescriptionErrorMarginTop = "4px";
export const BDatePickerLabelMarginBottom = "8px";
export const BDatePickerLabelRequirementColor = "#5c687c";
export const BDatePickerErrorMessageMarginTop = "8px";
export const BDateRangePickerDescriptionColor = "#5c687c";
export const BDateRangePickerDescriptionMarginTop = "8px";
export const BDateRangePickerDescriptionErrorMarginTop = "4px";
export const BDateRangePickerErrorMessageMarginTop = "8px";
export const BDateRangePickerLabelMarginBottom = "8px";
export const BDateRangePickerLabelRequirementColor = "#5c687c";
export const BDateRangePickerCalendarFlexDirection = "row";
export const BDateRangePickerCalendarCalendarsContainerBackgroundColor = "inherit";
export const BDateRangePickerCalendarCalendarsContainerBorderRight = "none";
export const BDateRangePickerCalendarCalendarsContainerGap = "24px";
export const BDateRangePickerCalendarCalendarsContainerPadding = "24px";
export const BDateRangePickerCalendarFormGap = "12px";
export const BDateRangePickerCalendarFormMinWidth = "auto";
export const BDateRangePickerCalendarFormPadding = "16px";
export const BDateRangePickerCalendarFormWidth = "332px";
export const BDateRangePickerCalendarFormContainerBorderRight = "1px solid #eeeff1";
export const BDateRangePickerCalendarFormDateFormatExampleColor = "#5c687c";
export const BDateRangePickerCalendarFormDateFormatExampleDisplay = "initial";
export const BDateRangePickerCalendarFormInputPaddingTop = "8px";
export const BDateRangePickerCalendarFormInputWrapperGap = "10px";
export const BDateRangePickerCalendarLabelFromDisplay = "initial";
export const BDateRangePickerCalendarLabelToDisplay = "initial";
export const BDividerBackgroundColor = "#eeeff1";
export const BDividerBackgroundColorInverse = "#2f3e56";
export const BDividerBorder = "none";
export const BDividerHeight = "1px";
export const BDropdownLabelColor = "#5c687c";
export const BDropdownLabelMarginBottom = "8px";
export const BDropdownDescriptionColor = "#5c687c";
export const BDropdownDescriptionMarginTop = "8px";
export const BDropdownDescriptionErrorMarginTop = "4px";
export const BDropdownDescriptionCondensedMarginTop = "6px";
export const BDropdownErrorMessageMarginTop = "8px";
export const BDropdownDefaultTextboxBackgroundColor = "#ffffff";
export const BDropdownDefaultTextboxBorderColor = "#8d95a3";
export const BDropdownDefaultTextboxBorderRadius = "8px";
export const BDropdownDefaultTextboxColor = "#00112c";
export const BDropdownDefaultTextboxHeight = "36px";
export const BDropdownDefaultTextboxIconColor = "#00112c";
export const BDropdownDefaultTextboxPadding = "0 12px";
export const BDropdownDefaultTextboxFocusBorderColor = "#00112c";
export const BDropdownDefaultTextboxFocusColor = "#00112c";
export const BDropdownDefaultTextboxFocusOutlineColor = "#00112c";
export const BDropdownDefaultTextboxFocusOutlineWidth = "0.5px";
export const BDropdownDefaultTextboxDisabledBackgroundColor = "#eeeff1";
export const BDropdownDefaultTextboxDisabledBorderColor = "#dbdee2";
export const BDropdownDefaultTextboxDisabledColor = "#8d95a3";
export const BDropdownDefaultTextboxDisabledIconColor = "#8d95a3";
export const BDropdownDefaultTextboxReadonlyBackgroundColor = "#eeeff1";
export const BDropdownDefaultTextboxReadonlyBorderColor = "#dbdee2";
export const BDropdownDefaultTextboxReadonlyIconColor = "#8d95a3";
export const BDropdownDefaultTextboxHoverBorderColor = "#00112c";
export const BDropdownDefaultTextboxCondensedMinHeight = "28px";
export const BDropdownDefaultTextboxCondensedPadding = "0 10px";
export const BDropdownDefaultTextboxErrorBorder = "1.5px solid #e22d2d";
export const BDropdownDefaultTextboxErrorFocusBorderColor = "#00112c";
export const BDropdownDefaultTextboxErrorFocusOutlineColor = "#00112c";
export const BDropdownDefaultTextboxOpenedBottomBorderRadius = "8px";
export const BDropdownDefaultTextboxAdditionalItemsBackgroundColor = "#f7f7f8";
export const BDropdownDefaultTextboxAdditionalItemsBorderRadius = "4px";
export const BDropdownDefaultTextboxAdditionalItemsPadding = "4px 6px";
export const BDropdownDefaultTextboxAdditionalItemsCondensedMaxHeight = "24px";
export const BDropdownDefaultTextboxBaseTextboxGap = "12px";
export const BDropdownDefaultTextboxInputSelectedElementMarginRight = "8px";
export const BDropdownOptionsContainerBorderColor = "unset";
export const BDropdownOptionsContainerBorderRadius = "8px";
export const BDropdownOptionsContainerBorderWidth = "0px";
export const BDropdownOptionsContainerBoxShadow = "0px 6px 12px rgba(0, 17, 44, 0.08), 0px 2px 4px rgba(0, 17, 44, 0.04), 0px 0px 0px 1px rgba(0, 17, 44, 0.06)";
export const BDropdownOptionsContainerMarginTop = "0px";
export const BDropdownSmallTextboxBackgroundColor = "#f7f7f8";
export const BDropdownSmallTextboxBorderColor = "#f7f7f8";
export const BDropdownSmallTextboxBorderRadius = "4px";
export const BDropdownSmallTextboxColor = "#00112c";
export const BDropdownSmallTextboxHeight = "28px";
export const BDropdownSmallTextboxPadding = "4px 6px";
export const BDropdownSmallTextboxFocusBorderColor = "#00112c";
export const BDropdownSmallTextboxHoverColor = "#5c687c";
export const BDropdownSmallTextboxDisabledBackgroundColor = "#eeeff1";
export const BDropdownSmallTextboxDisabledBorderColor = "#eeeff1";
export const BDropdownSmallTextboxDisabledColor = "#8d95a3";
export const BDropdownSmallTextboxReadonlyBackgroundColor = "#eeeff1";
export const BDropdownSmallTextboxReadonlyBorderColor = "#eeeff1";
export const BDropdownSmallTextboxReadonlyIconColor = "#8d95a3";
export const BDropdownSmallTextboxOpenedBottomBorderRadius = "4px";
export const BDropdownSmallTextboxAdditionalItemsBackgroundColor = "#f7f7f8";
export const BDropdownSmallTextboxAdditionalItemsPadding = "4px 6px";
export const BDropdownSmallTextboxAdditionalItemsMargin = "0 0 0 12px";
export const BDropdownSmallTextboxBaseTextboxGap = "2px";
export const BDropdownSmallTextboxBaseTextboxHeight = "auto";
export const BEmptyStateTitleMarginBottom = "4px";
export const BEmptyStateDetailsColor = "#5c687c";
export const BEmptyStateDetailsMarginBottom = "16px";
export const BErrorMessageColor = "#e22d2d";
export const BErrorMessageIconMarginRight = "8px";
export const BErrorMessageIconHeight = "20px";
export const BFieldLabelColor = "#00112c";
export const BFieldLabelMarginBottom = "8px";
export const BFieldLabelMarginRight = "4px";
export const BFieldLabelCondensedMarginBottom = "6px";
export const BFieldLabelRequirementColor = "#5c687c";
export const BFieldLabelTooltipWrapperMarginRight = "8px";
export const BFilterBarButtonBackgroundColor = "#ffffff";
export const BFilterBarButtonBorder = "1px solid #c9cdd3";
export const BFilterBarButtonBorderRadius = "8px";
export const BFilterBarButtonColor = "#00112c";
export const BFilterBarButtonHeight = "28px";
export const BFilterBarButtonActiveBorder = "1px solid #00112c";
export const BFilterBarButtonChevronPaddingRight = "12px";
export const BFilterBarButtonDisabledBackgroundColor = "#ffffff";
export const BFilterBarButtonDisabledColor = "#8d95a3";
export const BFilterBarButtonWithChevronContainerPaddingRight = "8px";
export const BFilterBarButtonContainerGap = "8px";
export const BFilterBarButtonContainerPadding = "4px 10px";
export const BFilterBarButtonCounterBackgroundColor = "#eeeff1";
export const BFilterBarButtonCounterBorderRadius = "4px";
export const BFilterBarButtonCounterColor = "#5c687c";
export const BFilterBarButtonCounterMarginRight = "4px";
export const BFilterBarButtonCounterPadding = "0 4px";
export const BFilterBarButtonCounterDisabledBackgroundColor = "#eeeff1";
export const BFilterBarButtonCounterDisabledColor = "#8d95a3";
export const BFilterBarButtonCounterHoverBackgroundColor = "#e3e5e9";
export const BFilterBarButtonFilterOpenBorder = "1px solid #00112c";
export const BFilterBarButtonHoverBackgroundColor = "#f7f7f8";
export const BFilterBarButtonHoverColor = "#5c687c";
export const BFilterBarButtonLabelContainerGap = "4px";
export const BFilterBarButtonLabelLineHeight = "20px";
export const BFilterBarButtonSecondaryLabelColor = "#5c687c";
export const BFilterBarGap = "8px";
export const BFilterBarPaddingRight = "16px";
export const BFilterBarZIndex = 5;
export const BFilterBarPersistentFiltersBorderColor = "#c9cdd3";
export const BFilterBarPersistentFiltersPaddingRight = "8px";
export const BFilterBarActionsGap = "8px";
export const BFilterBarBeforeFiltersGap = "8px";
export const BFilterBarContainerGap = "8px";
export const BFilterBarFiltersContainerGap = "8px";
export const BFilterBarScrollerContentGap = "8px";
export const BFilterBarFiltersWrapperGap = "8px";
export const BFilterBarFiltersWrapperPaddingRight = "16px";
export const BFilterBarInputFilterPadding = "12px 16px";
export const BFilterBarInputFilterOnlySlotPadding = "0px";
export const BFilterBarBooleanFilterPadding = "12px 16px";
export const BFilterBarBooleanFilterOnlySlotPadding = "0px";
export const BFilterBarRangeFilterPadding = "12px 16px";
export const BFilterBarRangeFilterOnlySlotPadding = "0px";
export const BFilterBarRangeFilterItemPadding = "12px";
export const BFilterBarSelectFilterDividerBackgroundColor = "#eeeff1";
export const BFilterBarSelectFilterDividerBorderWidth = "0";
export const BFilterBarSelectFilterDividerMargin = "8px 0px 0px";
export const BFilterBarSelectFilterSearchBarPadding = "12px 16px 0px";
export const BFilterBarDateFilterPadding = "16px 12px 16px";
export const BFilterBarDateFilterDividerMarginTop = "12px";
export const BFilterBarDateRangeFilterActionsPadding = "0 16px 12px 16px";
export const BFilterBarDateRangeFilterDividerMargin = "12px 0";
export const BFilterBarAllFiltersModalFiltersGap = "24px";
export const BFilterBarAllFiltersModalFiltersListGap = "6px";
export const BFilterBarAllFiltersModalFiltersListMargin = "0px";
export const BFilterBarAllFiltersModalFiltersListPadding = "0px";
export const BFilterBarAllFiltersModalFiltersListEmptySearchMessageColor = "#5c687c";
export const BFilterBarAllFiltersModalFiltersListEmptySearchMessagePadding = "0px 16px";
export const BFilterBarAllFiltersModalFiltersListTitleColor = "#5c687c";
export const BFilterBarAllFiltersModalFiltersListTitlePadding = "0px 16px";
export const BFilterBarAllFiltersModalSelectedFilterContentPadding = "8px 32px 8px 8px";
export const BFilterBarAllFiltersModalContentTextGap = "6px";
export const BFilterBarAllFiltersModalContentTextMarginBottom = "32px";
export const BFilterBarAllFiltersModalContentPadding = "0px";
export const BFilterBarAllFiltersModalClearButtonPaddingBottom = "8px";
export const BFilterBarAllFiltersModalSearchBarPadding = "0px 16px";
export const BFilterButtonBackgroundColor = "#ffffff";
export const BFilterButtonFontColor = "#00112c";
export const BFilterButtonPadding = "6px 16px";
export const BFilterButtonActiveBackgroundColor = "#edfaf3";
export const BFilterButtonDefaultContainerGap = "16px";
export const BFilterButtonDisabledBackgroundColor = "#ffffff";
export const BFilterButtonDisabledFontColor = "#8d95a3";
export const BFilterButtonCounterBackgroundColor = "#eeeff1";
export const BFilterButtonCounterBorderRadius = "4px";
export const BFilterButtonCounterFontColor = "#5c687c";
export const BFilterButtonCounterPadding = "4px 6px";
export const BFilterButtonCounterActiveBackgroundColor = "#dbdee2";
export const BFilterButtonCounterOpenBackgroundColor = "#dbdee2";
export const BFilterButtonCounterDisabledBackgroundColor = "#eeeff1";
export const BFilterButtonCounterDisabledFontColor = "#8d95a3";
export const BFilterButtonCounterHoverBackgroundColor = "#e3e5e9";
export const BFilterButtonHoverBackgroundColor = "#f7f7f8";
export const BFilterButtonHoverTextColor = "#5c687c";
export const BFilterButtonLabelLineHeight = "20px";
export const BFilterButtonModalCounterFocusBackgroundColor = "#dbdee2";
export const BFilterButtonModalFocusBackgroundColor = "#edfaf3";
export const BFilterButtonModalHoverBackgroundColor = "#f7f7f8";
export const BFilterButtonModalHoverColor = "#00112c";
export const BFilterButtonModalOpenBackgroundColor = "#edfaf3";
export const BFixedScrollerButtonMarginTop = "4px";
export const BFixedScrollerLeftBackground = "linear-gradient(90deg, #ffffff 75%, rgba(255, 255, 255, 0) 100%)";
export const BFixedScrollerLeftPaddingRight = "20px";
export const BFixedScrollerRightBackground = "linear-gradient(270deg, #ffffff 75%, rgba(255, 255, 255, 0) 100%)";
export const BFixedScrollerRightPaddingLeft = "20px";
export const BFixedScrollerSideSpacingPaddingBottom = "4px";
export const BFooterActionsWrapperRowGap = "8px";
export const BFormLayoutGap = "20px";
export const BFormLayoutActionsGap = "8px";
export const BFormLayoutActionsContainerMarginTop = "20px";
export const BFormLayoutErrorMessageMarginBottom = "20px";
export const BFormLayoutFieldTipGap = "12px";
export const BFormLayoutGroupGap = "10px";
export const BFormLayoutTitleDescriptionColor = "#5c687c";
export const BFormLayoutTitleSubsectionMarginTop = "20px";
export const BHeaderWrapperGap = "40px";
export const BHeaderTitleColor = "#00112c";
export const BHeaderTitleMarginTop = "2px";
export const BHeaderInfoIconMarginLeft = "10px";
export const BHeaderInfoIconMarginTop = "4px";
export const BHeaderTimestampColor = "#5c687c";
export const BHeaderTimestampMarginBlock = "4px";
export const BHeaderActionsMarginTop = "16px";
export const BHeaderActionsMarginTopDesktop = 0;
export const BHeaderComponentTitleMarginTop = "4px";
export const BInputFieldLabelColor = "#00112c";
export const BInputFieldLabelMarginBottom = "8px";
export const BInputFieldLabelRequirementColor = "#5c687c";
export const BInputFieldInputBoxBackgroundColor = "#ffffff";
export const BInputFieldInputBoxColor = "#00112c";
export const BInputFieldInputBoxBorderColor = "#8d95a3";
export const BInputFieldInputBoxBorderRadius = "8px";
export const BInputFieldInputBoxHeight = "36px";
export const BInputFieldInputBoxPadding = "6px 12px";
export const BInputFieldInputBoxHoverBorderColor = "#00112c";
export const BInputFieldInputBoxCondensedPadding = "4px 10px";
export const BInputFieldInputBoxCondensedWithPaymentMethodPaddingLeft = "6px";
export const BInputFieldInputBoxDisabledBackgroundColor = "#eeeff1";
export const BInputFieldInputBoxDisabledBorderColor = "#dbdee2";
export const BInputFieldInputBoxErrorBorder = "1.5px solid #e22d2d";
export const BInputFieldInputBoxErrorBorderColor = "#e22d2d";
export const BInputFieldInputBoxErrorOutline = "0.5px solid #e22d2d";
export const BInputFieldInputBoxFocusBorderColor = "#00112c";
export const BInputFieldInputBoxFocusBorderWidth = "1.5px";
export const BInputFieldInputBoxFocusOutline = "0.5px solid #00112c";
export const BInputFieldInputBoxSmallPaddingLeft = "6px 12px 6px 3px";
export const BInputFieldInputBoxSmallPaddingRight = "6px 3px 6px 12px";
export const BInputFieldInputColor = "#00112c";
export const BInputFieldInputMargin = "8px 12px";
export const BInputFieldInputPadding = "0px";
export const BInputFieldInputFontSize = "14px";
export const BInputFieldInputCondensedMargin = "4px 10px";
export const BInputFieldInputPlaceholderColor = "#8d95a3";
export const BInputFieldInputDisabledBackgroundColor = "#eeeff1";
export const BInputFieldInputDisabledColor = "#8d95a3";
export const BInputFieldIconAfterMarginLeft = "12px";
export const BInputFieldIconAfterMarginRight = "12px";
export const BInputFieldIconAfterCondensedMarginLeft = "10px";
export const BInputFieldIconAfterCondensedMarginRight = "10px";
export const BInputFieldIconBeforeMarginLeft = "12px";
export const BInputFieldIconBeforeMarginRight = "12px";
export const BInputFieldIconBeforeCondensedMarginLeft = "10px";
export const BInputFieldIconBeforeCondensedMarginRight = "10px";
export const BInputFieldIconsColor = "#8d95a3";
export const BInputFieldStaticValueBackgroundColor = "#f7f7f8";
export const BInputFieldStaticValueColor = "#00112c";
export const BInputFieldStaticValueMargin = "12px";
export const BInputFieldStaticValuePadding = "4px 6px";
export const BInputFieldStaticValueBorderRadius = "4px";
export const BInputFieldStaticValueDisabledBackgroundColor = "#eeeff1";
export const BInputFieldStaticValueDisabledColor = "#8d95a3";
export const BInputFieldStaticValueEndMarginRight = "4px";
export const BInputFieldStaticValueEndCondensedMarginLeft = "10px";
export const BInputFieldStaticValueEndCondensedMarginRight = "4px";
export const BInputFieldStaticValueStartMarginLeft = "4px";
export const BInputFieldStaticValueStartCondensedMarginLeft = "4px";
export const BInputFieldStaticValueStartCondensedMarginRight = "10px";
export const BInputFieldPaymentMethodMarginRight = "12px";
export const BInputFieldPaymentMethodMarginLeft = "6px";
export const BInputFieldPaymentMethodCondensedMarginLeft = "6px";
export const BInputFieldPaymentMethodCondensedMarginRight = "10px";
export const BInputFieldDropdownBorderRadius = "4px";
export const BInputFieldDropdownPaddingRight = "12px";
export const BInputFieldDropdownEndMarginRight = "4px";
export const BInputFieldDropdownEndCondensedMarginRight = "20px";
export const BInputFieldDropdownEndCondensedPaddingLeft = "10px";
export const BInputFieldDropdownSmallTextboxCondensedPadding = "0 6px";
export const BInputFieldDropdownStartMarginLeft = "4px";
export const BInputFieldDropdownStartCondensedMarginLeft = "4px";
export const BInputFieldDropdownStartCondensedPaddingLeft = "10px";
export const BInputFieldDescriptionColor = "#5c687c";
export const BInputFieldDescriptionMarginTop = "8px";
export const BInputFieldDescriptionPadding = "0px";
export const BInputFieldErrorMessageMarginTop = "8px";
export const BLinkBorderRadius = "2px";
export const BLinkColor = "#00112c";
export const BLinkTextDecoration = "underline";
export const BLinkHoverColor = "#5c687c";
export const BLinkHoverTextDecoration = "none";
export const BLinkActiveColor = "#8d95a3";
export const BLinkQuietColor = "#00112c";
export const BLinkQuietHoverColor = "#00112c";
export const BLinkQuietActiveColor = "#00112c";
export const BListboxBackgroundColor = "#ffffff";
export const BListboxPadding = "8px 0";
export const BListboxEmptySearchMessageColor = "#5c687c";
export const BListboxEmptySearchMessagePadding = "8px 16px";
export const BListboxLazyLoadMargin = "8px 0";
export const BListboxOptionPadding = "8px 16px";
export const BListboxOptionActiveBackgroundColor = "#eeeff1";
export const BListboxOptionDisabledFontColor = "#8d95a3";
export const BListboxOptionFocusBackgroundColor = "#eeeff1";
export const BListboxOptionHoverBackgroundColor = "#f7f7f8";
export const BListboxOptionSelectedDescriptionColor = "#5c687c";
export const BListboxOptionDescriptionColor = "#5c687c";
export const BListboxOptionDescriptionMarginTop = "4px";
export const BListboxOptionTagLabelColor = "#00112c";
export const BListboxOptionTagLabelPadding = "0 0 0 8px";
export const BListboxSingleSelectCheckIconMargin = "16px";
export const BListboxSingleSelectCategoryMarginBottom = "16px";
export const BListboxSingleSelectCategoryLabelTextPadding = "8px 16px";
export const BListboxSingleSelectCategoryLabelTextColor = "#5c687c";
export const BListboxMultiSelectDividerBackgroundColor = "#eeeff1";
export const BListboxMultiSelectDividerMargin = "8px 0px";
export const BListboxMultiSelectOptionsGap = "16px";
export const BListboxMultiSelectOptionsCategoryOptionPaddingLeft = "32px";
export const BListboxMultiSelectOptionsStaticCategoryOptionPaddingLeft = "16px";
export const BListboxMultiSelectCategoryDisabledLabelTextColor = "#8d95a3";
export const BListboxMultiSelectCategoryStaticCategoryGap = "16px";
export const BListboxMultiSelectCategoryStaticCategoryLabelTextColor = "#5c687c";
export const BListboxMultiSelectCategoryOptionGap = "16px";
export const BLoadingButtonPrimaryActiveBackgroundColor = "#8d95a3";
export const BLoadingButtonPrimaryActiveBackgroundColorInverse = "#edfaf3";
export const BLoadingButtonPrimarySuccessBackgroundColor = "#00112c";
export const BLoadingButtonPrimarySuccessColor = "#ffffff";
export const BLoadingButtonPrimarySuccessActiveBackgroundColor = "#8d95a3";
export const BLoadingButtonPrimarySuccessDisabledBackgroundColor = "#eeeff1";
export const BLoadingButtonPrimarySuccessDisabledColor = "#8d95a3";
export const BLoadingButtonPrimarySuccessHoverBackgroundColor = "#5c687c";
export const BLoadingButtonSecondaryActiveBackgroundColor = "#edfaf3";
export const BLoadingButtonSecondaryActiveBackgroundColorInverse = "#8d95a3";
export const BLoadingButtonSecondaryActiveColor = "#03401c";
export const BLoadingButtonSecondaryActiveColorInverse = "#eeeff1";
export const BLoadingButtonSecondarySuccessBackgroundColor = "#ffffff";
export const BLoadingButtonSecondarySuccessBorderColor = "#c9cdd3";
export const BLoadingButtonSecondarySuccessColor = "#00112c";
export const BLoadingButtonSecondarySuccessHoverBackgroundColor = "#f7f7f8";
export const BLoadingButtonSecondarySuccessHoverBorderColor = "#c9cdd3";
export const BLoadingButtonSecondarySuccessHoverBoxShadow = "unset";
export const BLoadingButtonSecondarySuccessHoverColor = "#5c687c";
export const BLoadingButtonSecondarySuccessActiveBackgroundColor = "#edfaf3";
export const BLoadingButtonSecondarySuccessActiveBorderColor = "#c9cdd3";
export const BLoadingButtonSecondarySuccessDisabledBackgroundColor = "#ffffff";
export const BLoadingButtonSecondarySuccessDisabledBorderColor = "#dbdee2";
export const BLoadingButtonSecondarySuccessDisabledColor = "#8d95a3";
export const BLoadingButtonTertiaryActiveBackgroundColor = "#000000";
export const BLoadingIndicatorBackgroundColor = "#00112c";
export const BLoadingIndicatorColor = "#00112c";
export const BLoadingIndicatorHeight = "24px";
export const BLoadingIndicatorWidth = "24px";
export const BLoadingIndicatorSmallBorderWidth = "2px";
export const BLoadingIndicatorSmallHeight = "16px";
export const BLoadingIndicatorSmallWidth = "16px";
export const BLoadingIndicatorLargeBorderWidth = "4px";
export const BLoadingIndicatorLargeHeight = "48px";
export const BLoadingIndicatorLargeWidth = "48px";
export const BLoadingIndicatorInverseBackgroundColor = "#ffffff";
export const BLoadingIndicatorInverseColor = "#ffffff";
export const BMenuItemColor = "#00112c";
export const BMenuItemColorInverse = "#ffffff";
export const BMenuItemPadding = "8px 16px";
export const BMenuItemActiveBackgroundColor = "#eeeff1";
export const BMenuItemActiveBackgroundColorInverse = "#8d95a3";
export const BMenuItemActiveCriticalBackgroundColor = "#fef4f4";
export const BMenuItemActiveCriticalBackgroundColorInverse = "#611313";
export const BMenuItemActiveCriticalColor = "#791818";
export const BMenuItemActiveCriticalColorInverse = "#fddfdf";
export const BMenuItemCriticalColor = "#e22d2d";
export const BMenuItemCriticalColorInverse = "#f99c9c";
export const BMenuItemDisabledColor = "#8d95a3";
export const BMenuItemDisabledColorInverse = "#747f8f";
export const BMenuItemFocusBackgroundColor = "#eeeff1";
export const BMenuItemFocusBackgroundColorInverse = "#8d95a3";
export const BMenuItemFocusCriticalBackgroundColor = "#fef4f4";
export const BMenuItemFocusCriticalBackgroundColorInverse = "#611313";
export const BMenuItemFocusCriticalColor = "#791818";
export const BMenuItemFocusCriticalColorInverse = "#fddfdf";
export const BMenuItemHoverBackgroundColor = "#f7f7f8";
export const BMenuItemHoverBackgroundColorInverse = "#5c687c";
export const BMenuItemHoverCriticalBackgroundColor = "#fef4f4";
export const BMenuItemHoverCriticalBackgroundColorInverse = "#611313";
export const BMenuItemHoverCriticalColor = "#c72727";
export const BMenuItemHoverCriticalColorInverse = "#fbbdbd";
export const BMenuItemSecondaryTextColor = "#5c687c";
export const BMenuItemSecondaryTextColorInverse = "#9ea6b1";
export const BMenuItemSecondaryTextPaddingLeft = "16px";
export const BMenuItemIconLeftPadding = "16px";
export const BMenuItemIconRightColor = "#00112c";
export const BMenuItemIconRightColorInverse = "#ffffff";
export const BMenuItemIconRightPadding = "16px";
export const BMenuItemIndentPaddingLeft = "48px";
export const BMenuItemListBackgroundColor = "#ffffff";
export const BMenuItemListBackgroundColorInverse = "#00112c";
export const BMenuItemListPadding = "8px 0px";
export const BMenuItemListDividerMarginBlock = "8px";
export const BMenuSubmenuBackgroundColor = "#ffffff";
export const BMenuSubmenuBackgroundColorInverse = "#00112c";
export const BMenuSubmenuDividerBorder = "1px solid #eeeff1";
export const BMenuSubmenuDividerBorderInverse = "1px solid #2f3e56";
export const BMenuSubmenuDividerPadding = "8px";
export const BMenuSubmenuDividerMarginBlock = "8px";
export const BMenuSubmenuAnimationDuration = "150ms";
export const BModalBorderRadius = "12px";
export const BModalPadding = "0px";
export const BModalContainerGap = "16px";
export const BModalContainerBackgroundColor = "#ffffff";
export const BModalContainerColumnLayoutBorder = "1px solid #eeeff1";
export const BModalContainerColumnLayoutPadding = "24px";
export const BModalHeaderContainerZIndex = 1;
export const BModalBackdropBackgroundColor = "#00112c";
export const BModalAnimationDuration = "150ms";
export const BPaginationNavigationSmallContainerGap = "10px";
export const BPaginationContextMargin = "0 12px";
export const BPaginationControlsGap = "16px";
export const BPaginationDividerBorderColor = "#eeeff1";
export const BPaginationDividerBorderWidth = "1px";
export const BPaginationDividerMargin = "0 16px";
export const BPaginationPageNavigatorSmallContainerGap = "16px";
export const BPaginationResultsPerPageMargin = "0 12px";
export const BPaymentMethodLabelColor = "#00112c";
export const BPaymentMethodLabelPadding = "0 0 0 12px";
export const BPopoverBackgroundColor = "#ffffff";
export const BPopoverBorder = "unset";
export const BPopoverBorderRadius = "8px";
export const BPopoverBoxShadow = "0px 6px 12px rgba(0, 17, 44, 0.08), 0px 2px 4px rgba(0, 17, 44, 0.04), 0px 0px 0px 1px rgba(0, 17, 44, 0.06)";
export const BPopoverZIndex = 5;
export const BPopoverDismissButtonOnImageIconColor = "#ffffff";
export const BPopoverDismissButtonOnImageIconHoveredColor = "#ffffff";
export const BPopoverDismissButtonOnImageIconActiveColor = "#ffffff";
export const BPopoverWithDividerFooterBackgroundColor = "#ffffff";
export const BPopoverWithDividerFooterBorderTop = "1px solid #eeeff1";
export const BPopoverWithDividerFooterPaddingTop = "12px";
export const BPopoverWithDividerContentPadding = "0px";
export const BPopoverWithoutSpacePadding = "0px";
export const BPopoverContentPadding = "16px";
export const BPopoverContentWithTitlePadding = "12px 16px";
export const BPopoverHeaderPadding = "12px 16px 0px 16px";
export const BPopoverHeaderTitleColumnGap = "8px";
export const BPopoverImageHeaderPadding = "16px 16px 0px 16px";
export const BPopoverImageDismissIconRight = "16px";
export const BPopoverImageDismissIconTop = "16px";
export const BPopoverFooterPadding = "0px 16px 12px 16px";
export const BPopoverAnimationDuration = "150ms";
export const BPopperZIndex = 5;
export const BRadioButtonOffsetTop = "2px";
export const BRadioButtonBorderRadius = "12px";
export const BRadioButtonSpaceAroundIncrease = "4px";
export const BRadioButtonUncheckedBackgroundColor = "#ffffff";
export const BRadioButtonUncheckedBorder = "1px solid #8d95a3";
export const BRadioButtonUncheckedHoverBackgroundColor = "#f7f7f8";
export const BRadioButtonUncheckedHoverBorderColor = "#8d95a3";
export const BRadioButtonUncheckedActiveBackgroundColor = "#edfaf3";
export const BRadioButtonUncheckedDisabledBackgroundColor = "#eeeff1";
export const BRadioButtonUncheckedDisabledBorderColor = "#dbdee2";
export const BRadioButtonUncheckedReadonlyBackgroundColor = "#eeeff1";
export const BRadioButtonUncheckedReadonlyBorderColor = "#dbdee2";
export const BRadioButtonUncheckedErrorBackgroundColor = "#e22d2d";
export const BRadioButtonUncheckedErrorBorder = "1.5px solid #e22d2d";
export const BRadioButtonCheckedBackgroundColor = "#00112c";
export const BRadioButtonCheckedBorderColor = "#00112c";
export const BRadioButtonCheckedHoverBackgroundColor = "#5c687c";
export const BRadioButtonCheckedHoverBorderColor = "#5c687c";
export const BRadioButtonCheckedActiveBackgroundColor = "#8d95a3";
export const BRadioButtonCheckedActiveBorderColor = "#8d95a3";
export const BRadioButtonCheckedDisabledBackgroundColor = "#eeeff1";
export const BRadioButtonCheckedDisabledBorderColor = "#dbdee2";
export const BRadioButtonCheckedReadonlyBackgroundColor = "#eeeff1";
export const BRadioButtonCheckedReadonlyBorderColor = "#dbdee2";
export const BRadioButtonCheckedErrorBackgroundColor = "#e22d2d";
export const BRadioButtonCheckedErrorBorderColor = "#e22d2d";
export const BRadioButtonLabelContainerPadding = "16px";
export const BRadioButtonLabelContainerRowGap = "4px";
export const BRadioButtonLabelFontColor = "#00112c";
export const BRadioButtonContentPadding = "12px 0px 8px";
export const BRadioButtonDescriptionFontColor = "#5c687c";
export const BRadioButtonDescriptionDisabledFontColor = "#5c687c";
export const BRadioButtonKnobColor = "#ffffff";
export const BRadioButtonKnobOffset = "3px";
export const BRadioButtonKnobOffsetError = "2.5px";
export const BRadioButtonKnobSize = "8px";
export const BRadioButtonKnobTransition = "transform 250ms cubic-bezier(0.12, 0.6, 0.4, 0.95)";
export const BRadioButtonKnobDisabledColor = "#8d95a3";
export const BRadioButtonKnobReadonlyColor = "#00112c";
export const BRadioGroupVerticalGap = "12px";
export const BRadioGroupHorizontalGap = "32px";
export const BRadioGroupErrorMessageMarginTop = "12px";
export const BSegmentedControlBackgroundColor = "#eeeff1";
export const BSegmentedControlBorderRadius = "12px";
export const BSegmentedControlPadding = "4px";
export const BSegmentedControlButtonBorderRadius = "8px";
export const BSegmentedControlButtonColor = "#00112c";
export const BSegmentedControlButtonPadding = "4px 20px";
export const BSegmentedControlButtonIconMarginRight = "8px";
export const BSegmentedControlButtonCheckedBackgroundColor = "#ffffff";
export const BSegmentedControlButtonCheckedPadding = "4px 8px";
export const BSegmentedControlButtonDisabledColor = "#8d95a3";
export const BSegmentedControlButtonHoverBackgroundColor = "#e3e5e9";
export const BSidepanelBackgroundColor = "#ffffff";
export const BSidepanelBorderRadius = "12px";
export const BSidepanelWidth = "480px";
export const BSidepanelCloseButtonHeight = "64px";
export const BSidepanelCloseButtonRight = "24px";
export const BSidepanelCloseButtonZIndex = 1;
export const BSidepanelAnimationDuration = "250ms";
export const BSidepanelBackdropBackgroundColor = "#00112c";
export const BSidepanelPageHeaderBackgroundColor = "#ffffff";
export const BSidepanelPageHeaderFlexBasis = "64px";
export const BSidepanelPageHeaderGap = "12px";
export const BSidepanelPageHeaderPadding = "16px 56px 16px 24px";
export const BSidepanelPageHeaderBorderBottom = "0";
export const BSidepanelPageHeaderWithBorderBorderBottom = "1px solid #dbdee2";
export const BSidepanelPageContentPadding = "0 24px";
export const BSidepanelPageActionsBackgroundColor = "#ffffff";
export const BSidepanelPageActionsHeight = "68px";
export const BSidepanelPageActionsPadding = "16px 24px";
export const BSidepanelPageActionsBorderTop = "0";
export const BSidepanelPageActionsWithBorderBorderTop = "1px solid #dbdee2";
export const BSidepanelPageAnimationDuration = "150ms";
export const BStatusColumnGap = "12px";
export const BStatusIndicatorSize = "10px";
export const BStatusIndicatorBlueBackgroundColor = "#0070f5";
export const BStatusIndicatorGreenBackgroundColor = "#09ab4b";
export const BStatusIndicatorGreyBackgroundColor = "#c0c5cc";
export const BStatusIndicatorOrangeBackgroundColor = "#dd7d00";
export const BStatusIndicatorRedBackgroundColor = "#e22d2d";
export const BStatusIndicatorYellowBackgroundColor = "#f7bc00";
export const BStepperGap = "6px";
export const BStepperMaxWidth = "200px";
export const BStepperTextStepperColor = "#5c687c";
export const BStepperStepButtonColor = "#5c687c";
export const BStepperStepButtonPadding = "6px 8px";
export const BStepperStepButtonGap = "8px";
export const BStepperStepButtonBorderRadius = "8px";
export const BStepperStepButtonHoverBackgroundColor = "#f7f7f8";
export const BStepperStepContentColor = "#5c687c";
export const BStepperStepContentHoverColor = "#00112c";
export const BStepperStepContentCompletedColor = "#00112c";
export const BStepperStepContentActiveColor = "#00112c";
export const BStepperStepIndexColor = "#ffffff";
export const BStepperStepIndexWidth = "20px";
export const BStepperStepIndexHeight = "20px";
export const BStepperStepIndexBorderRadius = "12px";
export const BStepperStepIndexBackgroundColor = "#c0c5cc";
export const BStepperStepIndexActiveBackgroundColor = "#00112c";
export const BStepperStepIndexCompletedBackgroundColor = "#07893c";
export const BStepperStepIndexCompletedColor = "#00112c";
export const BStepperStepSubstepsGap = "6px";
export const BStepperStepSubstepsMarginTop = "6px";
export const BStepperStepSubstepsButtonPaddingLeft = "24px";
export const BStepperStepSubstepsContentColor = "#5c687c";
export const BStepperStepSubstepsContentHoverColor = "#00112c";
export const BStepperStepSubstepsContentCompletedColor = "#00112c";
export const BStepperStepSubstepsContentActiveColor = "#00112c";
export const BStepperStepSubstepsIndexColor = "#c0c5cc";
export const BStepperStepSubstepsIndexCompletedColor = "#07893c";
export const BStepperStepSubstepsIndexActiveColor = "#00112c";
export const BStructuredListItemPadding = "6px 0";
export const BStructuredListItemPaddingCondensed = "0 0 4px 0";
export const BStructuredListItemMarginBottom = "12px";
export const BStructuredListItemMarginBottomCondensed = "6px";
export const BStructuredListItemLabelPadding = "8px";
export const BStructuredListItemLabelPaddingMobile = "2px 0";
export const BStructuredListItemLabelColor = "#5c687c";
export const BStructuredListItemLabelGap = "8px";
export const BStructuredListItemLabelIconColor = "#00112c";
export const BStructuredListItemItemValueColor = "#00112c";
export const BStructuredListItemItemValueGap = "16px";
export const BStructuredListItemItemValueMobilePadding = "2px 0";
export const BStructuredListItemIconMargin = "2px";
export const BSummaryGridGap = "8px";
export const BSummaryGridItemBackgroundColor = "#f7f7f8";
export const BSummaryGridItemBorderRadius = "8px";
export const BSummaryGridItemPadding = "16px 16px 12px";
export const BSummaryGridItemGap = "8px";
export const BSummaryGridItemHeaderMarginBottom = "4px";
export const BSummaryGridItemTitleIconsGap = "12px";
export const BSummaryGridItemTitleIconsHeight = "20px";
export const BSummaryGridItemTitleIconsMarginLeft = "12px";
export const BSummaryGridItemContentLabelFontWeight = 600;
export const BSummaryGridItemContentLabelLineHeight = "24px";
export const BSummaryGridItemContentTrendMarginLeft = "8px";
export const BSummaryGridItemAmountSecondaryLabelPaddingLeft = "4px";
export const BTabsBackgroundColor = "#ffffff";
export const BTabsTabListGap = "24px";
export const BTabsTabListHeight = "26px";
export const BTabsTabTitleBorderRadius = "2px";
export const BTabsTabTitleColor = "#5c687c";
export const BTabsTabTitleActiveColor = "#00112c";
export const BTabsTabTitleDisabledColor = "#8d95a3";
export const BTabsTabTitleHoverColor = "#00112c";
export const BTabsTabUnderlineColor = "#00112c";
export const BTabsTabUnderlineHeight = "1px";
export const BTabsTabUnderlineHeightActive = "2px";
export const BTabsTabUnderlineWidth = "20px";
export const BTabsNotificationDotBackgroundColor = "#0070f5";
export const BTabsNotificationDotHeight = "7px";
export const BTabsNotificationDotMarginLeft = "6px";
export const BTabsNotificationDotWidth = "7px";
export const BTagPadding = "2px 8px";
export const BTagDefaultColor = "#5c687c";
export const BTagDefaultBackgroundColor = "#f7f7f8";
export const BTagBlueColor = "#0063d7";
export const BTagBlueBackgroundColor = "#f2f8ff";
export const BTagGreenColor = "#067934";
export const BTagGreenBackgroundColor = "#edfaf3";
export const BTagOrangeColor = "#9b5700";
export const BTagOrangeBackgroundColor = "#fff5e9";
export const BTagRedColor = "#c72727";
export const BTagRedBackgroundColor = "#fef4f4";
export const BTagWhiteColor = "#00112c";
export const BTagWhiteBackgroundColor = "#ffffff";
export const BTextareaMinWidth = "200px";
export const BTextareaLabelColor = "#00112c";
export const BTextareaLabelMarginBottom = "8px";
export const BTextareaLabelItemColor = "#5c687c";
export const BTextareaLabelItemErrorColor = "#e22d2d";
export const BTextareaInputBoxMarginBottom = "8px";
export const BTextareaInputBorder = "1px solid #8d95a3";
export const BTextareaInputBorderRadius = "8px";
export const BTextareaInputPadding = "12px 12px 8px 12px";
export const BTextareaInputColor = "#00112c";
export const BTextareaInputMinHeight = "44px";
export const BTextareaInputDisabledBackgroundColor = "#eeeff1";
export const BTextareaInputDisabledBorderColor = "#dbdee2";
export const BTextareaInputDisabledColor = "#8d95a3";
export const BTextareaInputErrorBackgroundColor = "#e22d2d";
export const BTextareaInputFocusBorder = "1.5px solid #00112c";
export const BTextareaInputHoverBorderColor = "#00112c";
export const BTextareaInputPlaceholderColor = "#8d95a3";
export const BTextareaInputReadonlyBackgroundColor = "#eeeff1";
export const BTextareaInputReadonlyBorderColor = "#dbdee2";
export const BTextareaInputClearablePadding = "12px 40px 8px 12px";
export const BTextareaDescriptionColor = "#5c687c";
export const BTextareaDescriptionMarginTop = "4px";
export const BTextareaDescriptionPadding = 0;
export const BTextareaDescriptionErrorColor = "#e22d2d";
export const BTimelineMaxWidth = "500px";
export const BTimelineAddNoteSectionPaddingBottom = "24px";
export const BTimelineItemRowGap = "16px";
export const BTimelineItemContentPaddingBottom = "32px";
export const BTimelineItemIconMargin = "2px 0";
export const BTimelineItemIconColor = "#00112c";
export const BTimelineItemIconDefaultColor = "#00112c";
export const BTimelineItemIconDefaultMargin = "2px 0";
export const BTimelineItemIconSuccessColor = "#0abf53";
export const BTimelineItemIconCriticalColor = "#e22d2d";
export const BTimelineItemIconBlueColor = "#0070f5";
export const BTimelineItemIconGreenColor = "#09ab4b";
export const BTimelineItemIconGreyColor = "#c0c5cc";
export const BTimelineItemIconOrangeColor = "#dd7d00";
export const BTimelineItemIconRedColor = "#e22d2d";
export const BTimelineItemIconYellowColor = "#f7bc00";
export const BTimelineItemIconTimegapPaddingBottom = "6px";
export const BTimelineItemIconTimegapBorder = "1px solid #dbdee2";
export const BTimelineItemIconTimegapMargin = "6px auto";
export const BTimelineItemSeparatorBackgroundColor = "#dbdee2";
export const BTimelineItemTitleColor = "#00112c";
export const BTimelineItemTitleMarginBottom = "2px";
export const BTimelineItemTitlePaddingTop = "4px";
export const BTimelineItemTimestampColor = "#5c687c";
export const BTimelineItemTimestampGap = "6px";
export const BTimelineItemDescriptionColor = "#00112c";
export const BTimelineItemDescriptionMarginTop = "10px";
export const BTimelineItemDataListMarginTop = "10px";
export const BTimelineItemTagMarginTop = "10px";
export const BTimelineItemTimegapColor = "#5c687c";
export const BTimelineNoteTimestampMarginTop = "10px";
export const BToastItemBackgroundColor = "#f2f8ff";
export const BToastItemBorderColor = "#0070f5";
export const BToastItemBorderWidth = "1px";
export const BToastItemBorderRadius = "8px";
export const BToastItemColor = "#00112c";
export const BToastItemIconColor = "#0070f5";
export const BToastItemIconMargin = "16px";
export const BToastItemPadding = "16px";
export const BToastItemShadow = "0px 6px 12px rgba(0, 17, 44, 0.08), 0px 2px 4px rgba(0, 17, 44, 0.04), 0px 0px 0px 1px rgba(0, 17, 44, 0.06)";
export const BToastItemZIndex = 15;
export const BToggleHandleBackgroundColor = "#00112c";
export const BToggleHandleHeight = "12px";
export const BToggleHandleWidth = "12px";
export const BToggleHandleBorderRadius = "12px";
export const BToggleHandleTransition = "transform 150ms cubic-bezier(0.2, 0, 0.4, 0.9)";
export const BToggleHandleDisabledBackgroundColor = "#8d95a3";
export const BToggleHandleToggledBackgroundColor = "#ffffff";
export const BToggleHandleToggledHeight = "16px";
export const BToggleHandleToggledWidth = "16px";
export const BToggleHandleToggledColor = "#00112c";
export const BToggleHandleToggledDisabledBackgroundColor = "#eeeff1";
export const BToggleHandleToggledDisabledColor = "#8d95a3";
export const BToggleHandleToggledReadonlyBackgroundColor = "#eeeff1";
export const BToggleTrackHeight = "20px";
export const BToggleTrackWidth = "36px";
export const BToggleTrackPadding = "2px 4px";
export const BToggleTrackBorder = "1px solid #8d95a3";
export const BToggleTrackBorderRadius = "12px";
export const BToggleTrackBackgroundColor = "#ffffff";
export const BToggleTrackHoverBackgroundColor = "#f7f7f8";
export const BToggleTrackHoverBorderColor = "#6d7789";
export const BToggleTrackActiveBackgroundColor = "#edfaf3";
export const BToggleTrackActiveBorderColor = "#00112c";
export const BToggleTrackDisabledBackgroundColor = "#f7f7f8";
export const BToggleTrackDisabledBorderColor = "#dbdee2";
export const BToggleTrackReadonlyBackgroundColor = "#f7f7f8";
export const BToggleTrackReadonlyBorderColor = "#dbdee2";
export const BToggleTrackToggledBackgroundColor = "#00112c";
export const BToggleTrackToggledPadding = "2px";
export const BToggleTrackToggledBorder = "0";
export const BToggleTrackToggledHoverBackgroundColor = "#5c687c";
export const BToggleTrackToggledActiveBackgroundColor = "#8d95a3";
export const BToggleTrackToggledDisabledBackgroundColor = "#c0c5cc";
export const BToggleTrackToggledReadonlyBackgroundColor = "#c0c5cc";
export const BToggleLabelPadding = "16px";
export const BToggleDescriptionColor = "#5c687c";
export const BToggleDescriptionPadding = "4px";
export const BTooltipBackgroundColor = "#00112c";
export const BTooltipBorderRadius = "4px";
export const BTooltipColor = "#ffffff";
export const BTooltipMaxWidth = "300px";
export const BTooltipPadding = "4px 8px";
export const BTooltipZIndex = 5;
export const BTutorialPopoverColor = "#5c687c";
export const BTutorialPopoverStepsLineHeight = "26px";
export const BTutorialPopoverRippleBeforeBackgroundColor = "#09ab4b";
export const BTutorialPopoverRippleBeforeFilter = "blur(40px)";
export const BTutorialPopoverRippleBeforeHeight = "48px";
export const BTutorialPopoverRippleBeforeWidth = "64px";
export const BTutorialPopoverRippleBeforePosition = "-48px";
export const BTutorialPopoverRippleAfterHeight = "6px";
export const BTutorialPopoverRippleAfterWidth = "10px";
export const BTutorialPopoverRippleAfterPosition = "-24px";
export const BTutorialPopoverHeaderGap = "16px";
export const BTutorialPopoverHeaderPaddingBottom = "12px";
export const BTutorialPopoverFooterColumnGap = "8px";
export const BTutorialPopoverFooterPaddingTop = "12px";
export const BTutorialIndicatorBorder = "2px solid #09ab4b";
export const BTutorialIndicatorBorderOffset = "-2px";
export const BAnimationEasingStandard = "cubic-bezier(0.2, 0, 0.4, 0.9)";
export const BAnimationEasingEnter = "cubic-bezier(0.12, 0.6, 0.4, 0.95)";
export const BAnimationEasingExit = "cubic-bezier(0.4, 0, 1, 1)";
export const BAnimationEasingLinear = "linear";
export const BAnimationDurationFast = "100ms";
export const BAnimationDurationModerate = "150ms";
export const BAnimationDurationSlow = "250ms";
export const BBorderRadiusXs = "2px";
export const BBorderRadiusS = "4px";
export const BBorderRadiusM = "8px";
export const BBorderRadiusL = "12px";
export const BBorderRadiusXl = "24px";
export const BBorderWidthS = "1px";
export const BBorderWidthM = "2px";
export const BBorderWidthL = "3px";
export const BMediaQueryXsMin = "0px";
export const BMediaQueryXsMax = "375px";
export const BMediaQuerySMin = "376px";
export const BMediaQuerySMax = "499px";
export const BMediaQueryMMin = "500px";
export const BMediaQueryMMax = "767px";
export const BMediaQueryLMin = "768px";
export const BMediaQueryLMax = "1279px";
export const BMediaQueryXlMin = "1280px";
export const BMediaQueryXlMax = "1535px";
export const BMediaQueryXxl = "1536px";
export const BShadowLow = "0px 2px 4px rgba(0, 17, 44, 0.04), 0px 1px 2px rgba(0, 17, 44, 0.02)";
export const BShadowMedium = "0px 6px 12px rgba(0, 17, 44, 0.08), 0px 2px 4px rgba(0, 17, 44, 0.04)";
export const BShadowHigh = "0px 18px 36px rgba(0, 17, 44, 0.12), 0px 4px 8px rgba(0, 17, 44, 0.06)";
export const BShadowLowBorder = "0px 2px 4px rgba(0, 17, 44, 0.04), 0px 1px 2px rgba(0, 17, 44, 0.02), 0px 0px 0px 1px rgba(0, 17, 44, 0.06)";
export const BShadowMediumBorder = "0px 6px 12px rgba(0, 17, 44, 0.08), 0px 2px 4px rgba(0, 17, 44, 0.04), 0px 0px 0px 1px rgba(0, 17, 44, 0.06)";
export const BShadowHighBorder = "0px 18px 36px rgba(0, 17, 44, 0.12), 0px 4px 8px rgba(0, 17, 44, 0.06), 0px 0px 0px 1px rgba(0, 17, 44, 0.06)";
export const BSpacer100 = "32px";
export const BSpacer110 = "40px";
export const BSpacer120 = "48px";
export const BSpacer130 = "56px";
export const BSpacer140 = "64px";
export const BSpacer000 = "0px";
export const BSpacer010 = "2px";
export const BSpacer020 = "4px";
export const BSpacer030 = "6px";
export const BSpacer040 = "8px";
export const BSpacer050 = "10px";
export const BSpacer060 = "12px";
export const BSpacer070 = "16px";
export const BSpacer080 = "20px";
export const BSpacer090 = "24px";
