// @adyen/bento-design-tokens/dist/js/bento/es6-legacy.js
export const BColorBackgroundPrimary = "#ffffff";
export const BColorBackgroundPrimaryHover = "#f7f7f8";
export const BColorBackgroundPrimaryActive = "#eeeff1";
export const BColorBackgroundSecondary = "#f7f7f8";
export const BColorBackgroundSecondaryHover = "#eeeff1";
export const BColorBackgroundSecondaryActive = "#e3e5e9";
export const BColorBackgroundTertiary = "#eeeff1";
export const BColorBackgroundTertiaryHover = "#e3e5e9";
export const BColorBackgroundTertiaryActive = "#dbdee2";
export const BColorBackgroundQuaternary = "#c0c5cc";
export const BColorBackgroundQuaternaryHover = "#b8bdc6";
export const BColorBackgroundQuaternaryActive = "#afb5bf";
export const BColorBackgroundModal = "#ffffff";
export const BColorBackgroundModalHover = "#f7f7f8";
export const BColorBackgroundModalActive = "#eeeff1";
export const BColorBackgroundDisabled = "#eeeff1";
export const BColorBackgroundSelected = "#e3f0ff";
export const BColorBackgroundCriticalWeak = "#fef4f4";
export const BColorBackgroundCriticalStrong = "#e22d2d";
export const BColorBackgroundCriticalStrongHover = "#c72727";
export const BColorBackgroundCriticalStrongActive = "#791818";
export const BColorBackgroundWarningWeak = "#fff5e9";
export const BColorBackgroundWarningStrong = "#dd7d00";
export const BColorBackgroundSuccessWeak = "#edfaf3";
export const BColorBackgroundSuccessStrong = "#07893c";
export const BColorBackgroundHighlightWeak = "#f2f8ff";
export const BColorBackgroundHighlightStrong = "#0070f5";
export const BColorBackgroundInversePrimary = "#00112c";
export const BColorBackgroundInversePrimaryHover = "#5c687c";
export const BColorBackgroundInversePrimaryActive = "#8d95a3";
export const BColorBackgroundInverseSecondary = "#1a2a44";
export const BColorBackgroundInverseSecondaryHover = "#283750";
export const BColorBackgroundInverseSecondaryActive = "#37455d";
export const BColorBackgroundInverseDisabled = "#283750";
export const BColorBackgroundInverseCriticalWeak = "#611313";
export const BColorBackgroundInverseCriticalStrong = "#f99c9c";
export const BColorBackgroundInverseCriticalStrongHover = "#fbbdbd";
export const BColorBackgroundInverseCriticalStrongActive = "#fddfdf";
export const BColorBackgroundAlwaysLight = "#ffffff";
export const BColorBackgroundAlwaysLightHover = "#f7f7f8";
export const BColorBackgroundAlwaysLightActive = "#eeeff1";
export const BColorBackgroundAlwaysLightDisabled = "#eeeff1";
export const BColorBackgroundAlwaysLightSelected = "#e3f0ff";
export const BColorBackgroundAlwaysDark = "#00112c";
export const BColorBackgroundAlwaysDarkHover = "#5c687c";
export const BColorBackgroundAlwaysDarkActive = "#8d95a3";
export const BColorBackgroundAlwaysDarkDisabled = "#eeeff1";
export const BColorBackgroundAlwaysDarkSelected = "#002f68";
export const BColorLabelPrimary = "#00112c";
export const BColorLabelPrimaryHover = "#5c687c";
export const BColorLabelPrimaryActive = "#8d95a3";
export const BColorLabelSecondary = "#5c687c";
export const BColorLabelTertiary = "#8d95a3";
export const BColorLabelDisabled = "#8d95a3";
export const BColorLabelCritical = "#e22d2d";
export const BColorLabelCriticalHover = "#c72727";
export const BColorLabelCriticalActive = "#791818";
export const BColorLabelWarning = "#b06300";
export const BColorLabelHighlight = "#0070f5";
export const BColorLabelSuccess = "#07893c";
export const BColorLabelOnColor = "#ffffff";
export const BColorLabelInversePrimary = "#ffffff";
export const BColorLabelInversePrimaryHover = "#f7f7f8";
export const BColorLabelInversePrimaryActive = "#eeeff1";
export const BColorLabelInverseSecondary = "#9ea6b1";
export const BColorLabelInverseDisabled = "#747f8f";
export const BColorLabelInverseCritical = "#f99c9c";
export const BColorLabelInverseCriticalHover = "#fbbdbd";
export const BColorLabelInverseCriticalActive = "#fddfdf";
export const BColorLabelOnBackgroundCriticalWeak = "#c72727";
export const BColorLabelOnBackgroundWarningWeak = "#9b5700";
export const BColorLabelOnBackgroundSuccessWeak = "#067934";
export const BColorLabelOnBackgroundHighlightWeak = "#0063d7";
export const BColorLabelAlwaysLight = "#ffffff";
export const BColorLabelAlwaysDark = "#00112c";
export const BColorLinkPrimary = "#00112c";
export const BColorLinkPrimaryHover = "#5c687c";
export const BColorLinkPrimaryActive = "#8d95a3";
export const BColorLinkPrimaryDisabled = "#8d95a3";
export const BColorOutlinePrimary = "#dbdee2";
export const BColorOutlinePrimaryHover = "#c9cdd3";
export const BColorOutlinePrimaryActive = "#00112c";
export const BColorOutlineSecondary = "#c9cdd3";
export const BColorOutlineSecondaryHover = "#b8bdc6";
export const BColorOutlineSecondaryActive = "#00112c";
export const BColorOutlineTertiary = "#8d95a3";
export const BColorOutlineTertiaryHover = "#6d7789";
export const BColorOutlineTertiaryActive = "#00112c";
export const BColorOutlineDisabled = "#dbdee2";
export const BColorOutlineSelected = "#00112c";
export const BColorOutlineCritical = "#e22d2d";
export const BColorOutlineCriticalHover = "#c72727";
export const BColorOutlineCriticalActive = "#791818";
export const BColorOutlineSuccess = "#07893c";
export const BColorOutlineSuccessHover = "#067934";
export const BColorOutlineSuccessActive = "#04481f";
export const BColorOutlineInversePrimary = "#2f3e56";
export const BColorOutlineInversePrimaryHover = "#6d7789";
export const BColorOutlineInversePrimaryActive = "#9ea6b1";
export const BColorOutlineInverseDisabled = "#455368";
export const BColorOutlineInverseCritical = "#f99c9c";
export const BColorOutlineInverseCriticalHover = "#fbbdbd";
export const BColorOutlineInverseCriticalActive = "#fddfdf";
export const BColorSeparatorPrimary = "#dbdee2";
export const BColorSeparatorSecondary = "#c9cdd3";
export const BColorSeparatorInversePrimary = "#2f3e56";
export const BColorSeparatorInverseSecondary = "#5c687c";
export const BColorDecorativeGreen = "#09ab4b";
export const BColorDecorativeBlue = "#0070f5";
export const BColorDecorativeYellow = "#f7bc00";
export const BColorDecorativeOrange = "#dd7d00";
export const BColorDecorativeRed = "#e22d2d";
export const BColorDecorativeGrey = "#c0c5cc";
export const BColorSupportOverlay = "#00112c";
export const BColorSupportShadow = "#00112c";
export const BColorGrey0 = "#ffffff";
export const BColorGrey10 = "#f3f6f9";
export const BColorGrey15 = "#eef1f5";
export const BColorGrey20 = "#dce0e5";
export const BColorGrey30 = "#a5afbd";
export const BColorGrey40 = "#8390a3";
export const BColorGrey50 = "#69778c";
export const BColorGrey60 = "#818890";
export const BColorGrey65 = "#6f7988";
export const BColorGrey70 = "#707070";
export const BColorGrey80 = "#404d61";
export const BColorGrey90 = "#394962";
export const BColorGrey110 = "#00112c";
export const BColorGreen10 = "#e6f8ed";
export const BColorGreen20 = "#cef2dd";
export const BColorGreen40 = "#0abf53";
export const BColorGreen50 = "#08a648";
export const BColorGreen60 = "#055f29";
export const BColorGreen70 = "#03431d";
export const BColorBlue10 = "#e5efff";
export const BColorBlue20 = "#cce0ff";
export const BColorBlue30 = "#599bff";
export const BColorBlue40 = "#0066ff";
export const BColorBlue50 = "#0048b3";
export const BColorBlue60 = "#00337f";
export const BColorBlue70 = "#002459";
export const BColorOrange10 = "#fff4e5";
export const BColorOrange20 = "#ffeacc";
export const BColorOrange50 = "#ff9500";
export const BColorOrange70 = "#7f4a00";
export const BColorRed10 = "#fef5f5";
export const BColorRed20 = "#facccc";
export const BColorRed30 = "#e92929";
export const BColorRed50 = "#e50000";
export const BColorRed60 = "#a10000";
export const BColorRed70 = "#720000";
export const BColorRed80 = "#c00307";
export const BColorWhite = "#ffffff";
export const BColorFocus = "#0066ff";
export const BColorPrimary10 = "#e5efff";
export const BColorPrimary20 = "#cce0ff";
export const BColorPrimary30 = "#599bff";
export const BColorPrimary40 = "#0066ff";
export const BColorPrimary50 = "#0048b3";
export const BColorPrimary60 = "#00337f";
export const BColorAttention10 = "#e5efff";
export const BColorWarning10 = "#fff4e5";
export const BColorWarning20 = "#ffeacc";
export const BColorError10 = "#fef5f5";
export const BColorError20 = "#facccc";
export const BColorError50 = "#e50000";
export const BColorError60 = "#a10000";
export const BColorError70 = "#720000";
export const BColorDestructive10 = "#fef5f5";
export const BColorDestructive50 = "#e50000";
export const BColorDestructive60 = "#a10000";
export const BColorDestructive70 = "#720000";
export const BColorNeutral0 = "#ffffff";
export const BColorNeutral10 = "#f3f6f9";
export const BColorNeutral20 = "#dce0e5";
export const BColorNeutral30 = "#a5afbd";
export const BColorNeutral40 = "#8390a3";
export const BColorNeutral50 = "#69778c";
export const BColorNeutral60 = "#818890";
export const BColorNeutral65 = "#6f7988";
export const BColorNeutral70 = "#707070";
export const BColorNeutral80 = "#404d61";
export const BColorNeutral90 = "#394962";
export const BColorNeutral110 = "#00112c";
export const BColorSuccess10 = "#e6f8ed";
export const BColorSuccess20 = "#cef2dd";
export const BColorSuccess40 = "#0abf53";
export const BColorSuccess50 = "#08a648";
export const BColorSuccess60 = "#055f29";
export const BColorSuccess70 = "#03431d";
export const BColorNavy30 = "#334156";
export const BColorNavy40 = "#20304c";
export const BColorNavy50 = "#00112c";
export const BFocusRingColor = "#0066ff";
export const BFocusRingSpacer = "1px";
export const BFocusRingOutline = "3px";
export const BTextCaptionFontFamily = "Fakt, -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif";
export const BTextCaptionFontSize = "13px";
export const BTextCaptionFontWeight = "400";
export const BTextCaptionLineHeight = "24px";
export const BTextCaptionLetterSpacing = 0;
export const BTextCaptionWideLineHeight = "20px";
export const BTextCaptionStrongerFontWeight = "500";
export const BTextCaptionStrongerWideFontWeight = "500";
export const BTextCaptionStrongerWideLineHeight = "20px";
export const BTextBodyFontFamily = "Fakt, -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif";
export const BTextBodyFontSize = "15px";
export const BTextBodyFontWeight = "100";
export const BTextBodyLineHeight = "24px";
export const BTextBodyLetterSpacing = "normal";
export const BTextBodyWideLineHeight = "24px";
export const BTextBodyStrongerFontWeight = "100";
export const BTextBodyStrongerWideFontWeight = "500";
export const BTextBodyStrongerWideLineHeight = "24px";
export const BTextBodyStrongestFontWeight = "600";
export const BTextBodyStrongestWideFontWeight = "600";
export const BTextBodyStrongestWideLineHeight = "24px";
export const BTextSubtitleFontFamily = "Fakt, -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif";
export const BTextSubtitleFontSize = "16px";
export const BTextSubtitleFontWeight = "500";
export const BTextSubtitleLineHeight = "26px";
export const BTextSubtitleLetterSpacing = 0;
export const BTextSubtitleStrongerFontWeight = "600";
export const BTextTitleFontFamily = "Fakt, -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif";
export const BTextTitleFontSize = "16px";
export const BTextTitleFontWeight = "600";
export const BTextTitleLineHeight = "26px";
export const BTextTitleLetterSpacing = 0;
export const BTextTitleMobileFontSize = "16px";
export const BTextTitleMobileLineHeight = "26px";
export const BTextTitleMFontFamily = "Fakt, -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif";
export const BTextTitleMFontSize = "20px";
export const BTextTitleMFontWeight = "600";
export const BTextTitleMLineHeight = "30px";
export const BTextTitleMLetterSpacing = 0;
export const BTextTitleMMobileFontSize = "18px";
export const BTextTitleMMobileLineHeight = "28px";
export const BTextTitleLFontFamily = "Fakt, -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif";
export const BTextTitleLFontSize = "24px";
export const BTextTitleLFontWeight = "600";
export const BTextTitleLLineHeight = "34px";
export const BTextTitleLLetterSpacing = 0;
export const BTextTitleLMobileFontSize = "20px";
export const BTextTitleLMobileLineHeight = "30px";
export const BZIndexGlobalScreen = "auto";
export const BZIndexGlobalElevated = 5;
export const BZIndexGlobalNavigation = 10;
export const BZIndexGlobalInstantInteraction = 15;
export const BZIndexLocalTier1 = 1;
export const BZIndexLocalTier2 = 2;
export const BZIndexLocalTier3 = 3;
export const BZIndexLocalTier4 = 4;
export const BAccordionItemBackgroundColor = "#ffffff";
export const BAccordionItemBorder = "1px solid #dce0e5";
export const BAccordionItemButtonGap = "16px";
export const BAccordionItemButtonPadding = "16px";
export const BAccordionItemButtonHoverBackgroundColor = "#f3f6f9";
export const BAccordionItemButtonActiveBackgroundColor = "#dce0e5";
export const BAccordionItemContentPadding = "0 16px 24px 44px";
export const BAccordionItemAnimationDuration = "150ms";
export const BAccordionItemAnimationEasing = "cubic-bezier(0.2, 0, 0.4, 0.9)";
export const BActionBarBottom = "16px";
export const BActionBarBottomExtraSpacing = "48px";
export const BActionBarZIndex = 5;
export const BActionBarContainerBackgroundColor = "#00112c";
export const BActionBarContainerBorderRadius = "28px";
export const BActionBarContainerColor = "#ffffff";
export const BActionBarContainerPadding = "10px 24px";
export const BActionBarContainerWidth = "calc(100% - 16px - 16px)";
export const BActionBarSelectionMarginRight = "40px";
export const BActionBarSelectionMarginRightMobile = "16px";
export const BActionBarCloseButtonMarginRight = "24px";
export const BActionBarCloseButtonMarginRightMobile = "16px";
export const BActionBarAnimationDuration = "150ms";
export const BAlertBorderRadius = "12px";
export const BAlertPadding = "12px 24px";
export const BAlertPaddingMobile = "16px";
export const BAlertGap = "20px";
export const BAlertGapMobile = "16px";
export const BAlertIconHeight = "14px";
export const BAlertDescriptionMarginTop = "16px";
export const BAlertActionsMarginTop = "10px";
export const BAlertActionsGap = "16px";
export const BAlertSuccessBackgroundColor = "#edfaf3";
export const BAlertSuccessColor = "#00112c";
export const BAlertSuccessIconColor = "#07893c";
export const BAlertHighlightBackgroundColor = "#f3f6f9";
export const BAlertHighlightColor = "#00112c";
export const BAlertHighlightIconColor = "#394962";
export const BAlertHighlightGap = "16px";
export const BAlertWarningBackgroundColor = "#fff5e9";
export const BAlertWarningColor = "#00112c";
export const BAlertWarningIconColor = "#dd7d00";
export const BAlertCriticalBackgroundColor = "#fef4f4";
export const BAlertCriticalColor = "#00112c";
export const BAlertCriticalIconColor = "#e22d2d";
export const BAlertTipPadding = "10px 12px";
export const BAlertTipGap = "12px";
export const BAlertTipIconHeight = "20px";
export const BAvatarImageSmallHeight = "24px";
export const BAvatarImageSmallWidth = "24px";
export const BAvatarImageYellowBackgroundColor = "#a5afbd";
export const BAvatarImageYellowColor = "#ffffff";
export const BAvatarImageBlueBackgroundColor = "#a5afbd";
export const BAvatarImageBlueColor = "#ffffff";
export const BAvatarImageGreenBackgroundColor = "#a5afbd";
export const BAvatarImageGreenColor = "#ffffff";
export const BAvatarImageRedBackgroundColor = "#a5afbd";
export const BAvatarImageRedColor = "#ffffff";
export const BAvatarLabelColor = "#00112c";
export const BAvatarLabelPadding = "0 0 0 12px";
export const BAvatarColor = "#00112c";
export const BButtonActionsWithMenuVerticalStackRowGap = "8px";
export const BButtonActionsButtonsEndColumnGap = "8px";
export const BButtonActionsFillContainerColumnGap = "8px";
export const BButtonActionsVerticalStackRowGap = "8px";
export const BButtonBorderRadius = "6px";
export const BButtonBoxShadow = "inset 0 0 0 1px #000000";
export const BButtonLineHeight = "15px";
export const BButtonPadding = "12.5px 16px";
export const BButtonTransition = "100ms linear";
export const BButtonPrimaryBackgroundColor = "#0066ff";
export const BButtonPrimaryBackgroundColorInverse = "#ffffff";
export const BButtonPrimaryColor = "#ffffff";
export const BButtonPrimaryColorInverse = "#00112c";
export const BButtonPrimaryActiveBackgroundColor = "#00337f";
export const BButtonPrimaryActiveBackgroundColorInverse = "#eeeff1";
export const BButtonPrimaryDisabledBackgroundColor = "#dce0e5";
export const BButtonPrimaryDisabledBackgroundColorInverse = "#283750";
export const BButtonPrimaryDisabledColor = "#a5afbd";
export const BButtonPrimaryDisabledColorInverse = "#747f8f";
export const BButtonPrimaryDisabledBoxShadow = "inset 0 0 0 1px #eeeff1";
export const BButtonPrimaryDisabledBoxShadowInverse = "inset 0 0 0 1px #455368";
export const BButtonPrimaryHoverBackgroundColor = "#0048b3";
export const BButtonPrimaryHoverBackgroundColorInverse = "#f7f7f8";
export const BButtonPrimaryCriticalBackgroundColor = "#e50000";
export const BButtonPrimaryCriticalBackgroundColorInverse = "#f99c9c";
export const BButtonPrimaryCriticalColor = "#ffffff";
export const BButtonPrimaryCriticalColorInverse = "#00112c";
export const BButtonPrimaryCriticalActiveBackgroundColor = "#c00307";
export const BButtonPrimaryCriticalActiveBackgroundColorInverse = "#fddfdf";
export const BButtonPrimaryCriticalDisabledBackgroundColor = "#dce0e5";
export const BButtonPrimaryCriticalDisabledBackgroundColorInverse = "#283750";
export const BButtonPrimaryCriticalDisabledBoxShadow = "inset 0 0 0 1px #eeeff1";
export const BButtonPrimaryCriticalDisabledBoxShadowInverse = "inset 0 0 0 1px #455368";
export const BButtonPrimaryCriticalDisabledColor = "#a5afbd";
export const BButtonPrimaryCriticalDisabledColorInverse = "#747f8f";
export const BButtonPrimaryCriticalHoverBackgroundColor = "#e92929";
export const BButtonPrimaryCriticalHoverBackgroundColorInverse = "#fbbdbd";
export const BButtonPrimarySuccessBackgroundColor = "#08a648";
export const BButtonPrimarySuccessColor = "#ffffff";
export const BButtonPrimarySuccessActiveBackgroundColor = "#03431d";
export const BButtonPrimarySuccessDisabledBackgroundColor = "#8390a3";
export const BButtonPrimarySuccessDisabledColor = "#ffffff";
export const BButtonPrimarySuccessHoverBackgroundColor = "#055f29";
export const BButtonSecondaryBackgroundColor = "#ffffff";
export const BButtonSecondaryBackgroundColorInverse = "#00112c";
export const BButtonSecondaryBorder = "1px solid #dce0e5";
export const BButtonSecondaryBorderInverse = "1px solid #2f3e56";
export const BButtonSecondaryBoxShadow = "inset 0 0 0 1px #dce0e5";
export const BButtonSecondaryColor = "#394962";
export const BButtonSecondaryColorInverse = "#ffffff";
export const BButtonSecondaryPadding = "calc(12.5px - 1px) calc(16px - 1px)";
export const BButtonSecondaryActiveBackgroundColor = "#f3f6f9";
export const BButtonSecondaryActiveBackgroundColorInverse = "#8d95a3";
export const BButtonSecondaryActiveBorder = "1px solid #dce0e5";
export const BButtonSecondaryActiveBorderInverse = "1px solid #9ea6b1";
export const BButtonSecondaryActiveBoxShadow = "inset 0 0 0 1px #dce0e5, inset 0 0 0 1px #dce0e5";
export const BButtonSecondaryActiveColor = "#394962";
export const BButtonSecondaryActiveColorInverse = "#ffffff";
export const BButtonSecondaryHoverBackgroundColor = "#ffffff";
export const BButtonSecondaryHoverBackgroundColorInverse = "#5c687c";
export const BButtonSecondaryHoverBorder = "1px solid #dce0e5";
export const BButtonSecondaryHoverBorderInverse = "1px solid #6d7789";
export const BButtonSecondaryHoverBoxShadow = "0px 2px 4px rgba(0, 17, 44, 0.04), 0px 1px 2px rgba(0, 17, 44, 0.02)";
export const BButtonSecondaryHoverColor = "#394962";
export const BButtonSecondaryHoverColorInverse = "#ffffff";
export const BButtonSecondaryDisabledBackgroundColor = "#ffffff";
export const BButtonSecondaryDisabledBackgroundColorInverse = "#00112c";
export const BButtonSecondaryDisabledBorder = "1px solid #dce0e5";
export const BButtonSecondaryDisabledBorderInverse = "1px solid #455368";
export const BButtonSecondaryDisabledBoxShadow = "inset 0 0 0 1px #dce0e5";
export const BButtonSecondaryDisabledColor = "#a5afbd";
export const BButtonSecondaryDisabledColorInverse = "#747f8f";
export const BButtonSecondaryCriticalBackgroundColor = "#ffffff";
export const BButtonSecondaryCriticalBackgroundColorInverse = "#00112c";
export const BButtonSecondaryCriticalBorder = "1px solid #dce0e5";
export const BButtonSecondaryCriticalBorderInverse = "1px solid #f99c9c";
export const BButtonSecondaryCriticalBoxShadow = "inset 0 0 0 1px #dce0e5";
export const BButtonSecondaryCriticalColor = "#e50000";
export const BButtonSecondaryCriticalColorInverse = "#f99c9c";
export const BButtonSecondaryCriticalActiveBackgroundColor = "#fef5f5";
export const BButtonSecondaryCriticalActiveBackgroundColorInverse = "#611313";
export const BButtonSecondaryCriticalActiveBorder = "1px solid #dce0e5";
export const BButtonSecondaryCriticalActiveBorderInverse = "1px solid #fddfdf";
export const BButtonSecondaryCriticalActiveBoxShadow = "inset 0 0 0 1px #dce0e5";
export const BButtonSecondaryCriticalActiveColor = "#791818";
export const BButtonSecondaryCriticalActiveColorInverse = "#fddfdf";
export const BButtonSecondaryCriticalDisabledBackgroundColor = "#ffffff";
export const BButtonSecondaryCriticalDisabledBackgroundColorInverse = "#00112c";
export const BButtonSecondaryCriticalDisabledBorder = "1px solid #dce0e5";
export const BButtonSecondaryCriticalDisabledBorderInverse = "1px solid #455368";
export const BButtonSecondaryCriticalDisabledBoxShadow = "inset 0 0 0 1px #dce0e5";
export const BButtonSecondaryCriticalDisabledColor = "#a5afbd";
export const BButtonSecondaryCriticalDisabledColorInverse = "#747f8f";
export const BButtonSecondaryCriticalHoverBackgroundColor = "#fef4f4";
export const BButtonSecondaryCriticalHoverBackgroundColorInverse = "#611313";
export const BButtonSecondaryCriticalHoverBorder = "1px solid #dce0e5";
export const BButtonSecondaryCriticalHoverBorderInverse = "1px solid #fbbdbd";
export const BButtonSecondaryCriticalHoverBoxShadow = "inset 0 0 0 1px #dce0e5, 0px 2px 4px rgba(0, 17, 44, 0.04), 0px 1px 2px rgba(0, 17, 44, 0.02)";
export const BButtonSecondaryCriticalHoverColor = "#c72727";
export const BButtonSecondaryCriticalHoverColorInverse = "#fbbdbd";
export const BButtonTertiaryBackgroundColor = "#000000";
export const BButtonTertiaryBorderRadius = "6px";
export const BButtonTertiaryBoxShadow = "inset 0 0 0 1px #000000";
export const BButtonTertiaryColor = "#394962";
export const BButtonTertiaryColorInverse = "#ffffff";
export const BButtonTertiaryHeight = "auto";
export const BButtonTertiaryPadding = "0";
export const BButtonTertiaryTextDecoration = "none";
export const BButtonTertiaryHoverBackgroundColor = "#000000";
export const BButtonTertiaryHoverColor = "#8390a3";
export const BButtonTertiaryHoverColorInverse = "#f7f7f8";
export const BButtonTertiaryHoverTextDecoration = "none";
export const BButtonTertiaryActiveBackgroundColor = "#000000";
export const BButtonTertiaryActiveColor = "#69778c";
export const BButtonTertiaryActiveColorInverse = "#eeeff1";
export const BButtonTertiaryActiveTextDecoration = "none";
export const BButtonTertiaryDisabledBackgroundColor = "#000000";
export const BButtonTertiaryDisabledColor = "#a5afbd";
export const BButtonTertiaryDisabledColorInverse = "#747f8f";
export const BButtonTertiaryCriticalBackgroundColor = "#000000";
export const BButtonTertiaryCriticalColor = "#e50000";
export const BButtonTertiaryCriticalColorInverse = "#f99c9c";
export const BButtonTertiaryCriticalActiveBackgroundColor = "#000000";
export const BButtonTertiaryCriticalActiveColor = "#e50000";
export const BButtonTertiaryCriticalActiveColorInverse = "#fddfdf";
export const BButtonTertiaryCriticalDisabledBackgroundColor = "#000000";
export const BButtonTertiaryCriticalDisabledColor = "#a5afbd";
export const BButtonTertiaryCriticalDisabledColorInverse = "#747f8f";
export const BButtonTertiaryCriticalHoverBackgroundColor = "#000000";
export const BButtonTertiaryCriticalHoverBoxShadow = "inset 0 0 0 1px #dce0e5, 0px 2px 4px rgba(0, 17, 44, 0.04), 0px 1px 2px rgba(0, 17, 44, 0.02)";
export const BButtonTertiaryCriticalHoverColor = "#e50000";
export const BButtonTertiaryCriticalHoverColorInverse = "#fbbdbd";
export const BButtonCondensedBorderRadius = "6px";
export const BButtonCondensedFontSize = "13px";
export const BButtonCondensedLineHeight = "17px";
export const BButtonCondensedPadding = "3.5px 8px";
export const BButtonCondensedIconHeight = "10px";
export const BButtonCondensedIconWidth = "10px";
export const BButtonCondensedSecondaryPadding = "calc(3.5px - 1px) calc(8px - 1px)";
export const BButtonCondensedTertiaryHeight = "24px";
export const BButtonCondensedTertiaryPadding = "0px";
export const BButtonIconOnlyPadding = "10px";
export const BButtonIconOnlyCondensedPadding = "4px 6px";
export const BButtonIconOnlySecondaryPadding = "calc(10px - 1px)";
export const BButtonIconOnlySecondaryPaddingCondensed = "calc(4px - 1px) calc(6px - 1px)";
export const BCalendarRangeGap = "24px";
export const BCalendarMonthDayOfTheWeekColor = "#a5afbd";
export const BCalendarMonthDayOfTheWeekPaddingBottom = "6px";
export const BCalendarMonthMonthSelectorMarginBottom = "16px";
export const BCalendarMonthDayHeight = "36px";
export const BCalendarMonthDayWidth = "36px";
export const BCalendarMonthDayBeginningBorderRadius = "0";
export const BCalendarMonthDayBeginningAfterZIndex = 2;
export const BCalendarMonthDayCurrentDaySelectedColor = "#ffffff";
export const BCalendarMonthDayCurrentDaySelectedWrapperBackground = "#0066ff";
export const BCalendarMonthDayCurrentDayUnselectedHoverWrapperBackground = "transparent";
export const BCalendarMonthDayCurrentDayUnselectedWrapperBackground = "transparent";
export const BCalendarMonthDayDisabledSelectedColor = "#8d95a3";
export const BCalendarMonthDayDisabledSelectedBackground = "#eeeff1";
export const BCalendarMonthDayDisabledUnselectedColor = "#8d95a3";
export const BCalendarMonthDayEndBorderRadius = "0";
export const BCalendarMonthDayEndAfterZIndex = 2;
export const BCalendarMonthDayMiddleBackground = "#599bff";
export const BCalendarMonthDayMiddleColor = "#ffffff";
export const BCalendarMonthDayMiddleHoverBackground = "#599bff";
export const BCalendarMonthDayMiddleActiveBackground = "#599bff";
export const BCalendarMonthDaySelectedBackground = "#0066ff";
export const BCalendarMonthDaySelectedBorderRadius = "0";
export const BCalendarMonthDaySelectedColor = "#ffffff";
export const BCalendarMonthDaySelectedHoverBackground = "#0066ff";
export const BCalendarMonthDaySelectedActiveBackground = "#0066ff";
export const BCalendarMonthDayUnselectedHoverBackground = "none";
export const BCalendarMonthDayUnselectedActiveBackground = "transparent";
export const BCalendarMonthMonthContainerBorderCollapse = "collapse";
export const BCalendarMonthWeekHeaderBorderBottom = "1px solid #dce0e5";
export const BCardBackgroundColor = "#ffffff";
export const BCardBorderColor = "#dce0e5";
export const BCardBorderRadius = "4px";
export const BCardBorderWidth = "1px";
export const BCardHeaderBorderRadius = "4px";
export const BCardHeaderColor = "#00112c";
export const BCardHeaderMarginTop = "4px";
export const BCardHeaderPadding = "16px 24px 0px";
export const BCardHeaderPaddingBottom = "20px";
export const BCardHeaderHoverBackgroundColor = "#ffffff";
export const BCardHeaderActiveBackgroundColor = "#eeeff1";
export const BCardHeaderFocusBackgroundColor = "#f7f7f8";
export const BCardToggleHeight = "26px";
export const BCardToggleSmallTextHeight = "24px";
export const BCardBodyPadding = "0px 24px 16px";
export const BCardBodyPaddingTop = "4px";
export const BCardBodyExpandablePaddingTop = "4px";
export const BCardContentPadding = "16px 0px 0px";
export const BCardContentPaddingTop = "20px";
export const BCardDescriptionColor = "#5c687c";
export const BCardDescriptionMarginLeft = "32px";
export const BCardActionsMarginTop = "16px";
export const BCardActionsPaddingTop = "20px";
export const BCardActionsWithoutContentPaddingTop = "16px";
export const BCardExpandableHeaderPaddingBottom = "20px";
export const BCardDisabledBackgroundColor = "#f7f7f8";
export const BCardDisabledColor = "#8d95a3";
export const BCardSecondaryBackgroundColor = "#f3f6f9";
export const BCardSecondaryBorderColor = "#f3f6f9";
export const BCardSecondaryHoverBackgroundColor = "#eeeff1";
export const BCardSecondaryHeaderHoverBackgroundColor = "#eeeff1";
export const BCardSecondaryHeaderActiveBackgroundColor = "#e3e5e9";
export const BCardSecondaryHeaderFocusBackgroundColor = "#eeeff1";
export const BCardTitleWrapperGap = "16px";
export const BCardAnimationDuration = "150ms";
export const BCardAnimationEasing = "cubic-bezier(0.2, 0, 0.4, 0.9)";
export const BCheckboxGroupCheckboxesVerticalGap = "16px";
export const BCheckboxGroupCheckboxesHorizontalGap = "32px";
export const BCheckboxGroupErrorMessageMarginTop = "12px";
export const BCheckboxDescriptionColor = "#707070";
export const BCheckboxDescriptionDisabledColor = "#8390a3";
export const BCheckboxContentPadding = "12px 0px 8px";
export const BCheckboxIconColor = "#ffffff";
export const BCheckboxIconDisabledColor = "#8390a3";
export const BCheckboxIconReadonlyColor = "#00112c";
export const BCheckboxOutlineBorderRadius = "4px";
export const BCheckboxOutlineBorderWidth = "1px";
export const BCheckboxOutlineTopOffset = "4px";
export const BCheckboxOutlineCheckedBackgroundColor = "#0066ff";
export const BCheckboxOutlineCheckedBorderColor = "#0066ff";
export const BCheckboxOutlineCheckedActiveBackgroundColor = "#0066ff";
export const BCheckboxOutlineCheckedActiveBorderColor = "#dce0e5";
export const BCheckboxOutlineCheckedDisabledBackgroundColor = "#dce0e5";
export const BCheckboxOutlineCheckedDisabledBorderColor = "#dce0e5";
export const BCheckboxOutlineCheckedHoverBackgroundColor = "#0066ff";
export const BCheckboxOutlineCheckedHoverBorderColor = "#0048b3";
export const BCheckboxOutlineCheckedErrorBackgroundColor = "#e22d2d";
export const BCheckboxOutlineCheckedErrorBorderColor = "#e22d2d";
export const BCheckboxOutlineIndeterminateBackgroundColor = "#0066ff";
export const BCheckboxOutlineIndeterminateBorderColor = "#0066ff";
export const BCheckboxOutlineIndeterminateActiveBackgroundColor = "#0066ff";
export const BCheckboxOutlineIndeterminateActiveBorderColor = "#dce0e5";
export const BCheckboxOutlineIndeterminateDisabledBackgroundColor = "#dce0e5";
export const BCheckboxOutlineIndeterminateDisabledBorderColor = "#dce0e5";
export const BCheckboxOutlineIndeterminateHoverBackgroundColor = "#0066ff";
export const BCheckboxOutlineIndeterminateHoverBorderColor = "#0048b3";
export const BCheckboxOutlineIndeterminateErrorBackgroundColor = "#e22d2d";
export const BCheckboxOutlineIndeterminateErrorBorderColor = "#e22d2d";
export const BCheckboxOutlineUncheckedBackgroundColor = "#ffffff";
export const BCheckboxOutlineUncheckedBorderColor = "#dce0e5";
export const BCheckboxOutlineUncheckedActiveBackgroundColor = "#ffffff";
export const BCheckboxOutlineUncheckedActiveBorderColor = "#0066ff";
export const BCheckboxOutlineUncheckedDisabledBackgroundColor = "#dce0e5";
export const BCheckboxOutlineUncheckedDisabledBorderColor = "#dce0e5";
export const BCheckboxOutlineUncheckedHoverBackgroundColor = "#ffffff";
export const BCheckboxOutlineUncheckedHoverBorderColor = "#69778c";
export const BCheckboxOutlineUncheckedErrorBackgroundColor = "#e22d2d";
export const BCheckboxOutlineUncheckedErrorBorder = "1.5px solid #e22d2d";
export const BCheckboxOutlineReadonlyBackgroundColor = "#eeeff1";
export const BCheckboxOutlineReadonlyBorderColor = "#dbdee2";
export const BCheckboxTextColor = "#00112c";
export const BCheckboxTextGap = "0px";
export const BCheckboxTextPaddingLeft = "8px";
export const BCheckboxTextDisabledColor = "#8390a3";
export const BCodeSnippetActionBarBackgroundColor = "#eeeff1";
export const BCodeSnippetActionBarPadding = "8px 12px";
export const BCodeSnippetActionBarGap = "8px";
export const BCodeSnippetActionBarBorder = "0 solid #dbdee2";
export const BCodeSnippetActionBarBorderRadius = "8px 8px 0 0";
export const BCodeSnippetLanguagePickerBackgroundColor = "#ffffff";
export const BCodeSnippetActionButtonPadding = "0 4px 0 4px";
export const BCodeSnippetCodeAreaBackgroundColor = "#f7f7f8";
export const BCodeSnippetCodeAreaMargin = 0;
export const BCodeSnippetCodeAreaBorderColor = "#dbdee2";
export const BCodeSnippetCodeAreaBorderRadius = "0 0 8px 8px";
export const BCodeSnippetCodeAreaBorderWidth = "1px";
export const BCodeSnippetCodeAreaContentFontFamily = "Menlo, 'Roboto Mono', 'Oxygen Mono', 'IBM Plex Mono', 'Courier New', monospace";
export const BCodeSnippetCodeAreaContentPadding = "12px";
export const BCodeSnippetCodeAreaContentPaddingLeft = "8px";
export const BCodeSnippetCodeAreaGuttersPaddingLeft = "12px";
export const BCodeSnippetCodeAreaHighlightRed = "#c72727";
export const BCodeSnippetCodeAreaHighlightOrange = "#9b5700";
export const BCodeSnippetCodeAreaHighlightGreen = "#067934";
export const BCodeSnippetCodeAreaHighlightBlue = "#0063d7";
export const BCodeSnippetEmptyStateBackgroundColor = "#f7f7f8";
export const BCodeSnippetEmptyStateMinHeight = "200px";
export const BCodeSnippetInlineCodeAreaContentPadding = "12px 8px 12px 8px";
export const BControlsGroupDescriptionColor = "#5c687c";
export const BControlsGroupLabelContainerGap = "4px";
export const BCountryLabelColor = "#00112c";
export const BDataGridTemplateGap = "12px";
export const BDataGridTemplateHeaderMargin = "20px 24px 0px 24px";
export const BDataGridTemplateHeaderZIndex = 3;
export const BDataGridTemplateAdditionalContentMargin = "0px 24px 0px 24px";
export const BDataGridTemplateTabsBackgroundColor = "#ffffff";
export const BDataGridTemplateTabsPadding = "0px 24px";
export const BDataGridTemplateDataGridFiltersPaddingRight = "24px";
export const BDataGridTemplateDataGridColumnsPaddingLeft = "10px";
export const BDataGridTemplateDataGridColumnsPaddingRight = "10px";
export const BDataGridTemplateDataGridCellsPaddingLeft = "10px";
export const BDataGridTemplateDataGridCellsPaddingRight = "10px";
export const BDataGridTemplateDataGridPaginationPaddingLeft = "24px";
export const BDataGridTemplateDataGridPaginationPaddingRight = "24px";
export const BDataGridBorderColor = "#dce0e5";
export const BDataGridBorderRadius = "8px";
export const BDataGridBorderWidth = "1px";
export const BDataGridHeaderPadding = "20px 24px";
export const BDataGridColumnHeadersZIndex = 2;
export const BDataGridColumnHeadersBackgroundColor = "#ffffff";
export const BDataGridColumnHeadersFocusWithinZIndex = 3;
export const BDataGridColumnsBorderColor = "#c9cdd3";
export const BDataGridColumnsBorderWidth = "1px";
export const BDataGridColumnsFrozenBorderRightColor = "#c9cdd3";
export const BDataGridColumnsFrozenBorderRightWidth = "1px";
export const BDataGridColumnsFrozenZIndex = 2;
export const BDataGridColumnsSelectColumnPaddingLeft = "8px";
export const BDataGridColumnsSelectColumnPaddingRight = "4px";
export const BDataGridColumnsSelectColumnGap = "4px";
export const BDataGridColumnBackgroundColor = "#ffffff";
export const BDataGridColumnBorderColor = "#000000";
export const BDataGridColumnHeight = "24px";
export const BDataGridColumnPadding = "16px";
export const BDataGridColumnColor = "#00112c";
export const BDataGridColumnHoverBackgroundColor = "#eeeff1";
export const BDataGridColumnHoverSortIconColor = "#8d95a3";
export const BDataGridColumnHoverBorderLeftColor = "#dce0e5";
export const BDataGridColumnHoverBorderRightColor = "#dce0e5";
export const BDataGridColumnSortPadding = "0 0 0 8px";
export const BDataGridColumnSortIconColor = "#a5afbd";
export const BDataGridColumnSortIconMarginTop = "2px";
export const BDataGridColumnSortIconDescendingIconMarginTop = "8px";
export const BDataGridColumnSeparatorBorderRightColor = "#dbdee2";
export const BDataGridColumnSeparatorBorderRightWidth = "1px";
export const BDataGridColumnSeparatorHeight = "24px";
export const BDataGridColumnSeparatorResizeableHoverColor = "#c9cdd3";
export const BDataGridColumnSeparatorZIndex = 1;
export const BDataGridColumnHeaderGap = "8px";
export const BDataGridColumnDraggableGhostBackgroundColor = "#e3f0ff";
export const BDataGridColumnActionsGap = "16px";
export const BDataGridColumnActionsPaddingLeft = "12px";
export const BDataGridColumnSeperatorColor = "#dce0e5";
export const BDataGridColumnSeperatorHeight = "100%";
export const BDataGridRowsFrozenBorderRightColor = "#c9cdd3";
export const BDataGridRowsFrozenBorderRightWidth = "1px";
export const BDataGridRowsFrozenZIndex = 1;
export const BDataGridRowBackgroundColor = "#ffffff";
export const BDataGridRowBorderColor = "#dce0e5";
export const BDataGridRowBorderWidth = "1px";
export const BDataGridRowColor = "#00112c";
export const BDataGridRowHoverBackgroundColor = "#f3f6f9";
export const BDataGridRowDisabledColor = "#8390a3";
export const BDataGridRowHighlightedBackgroundColor = "#e3f0ff";
export const BDataGridCellBackgroundColor = "#ffffff";
export const BDataGridCellPadding = "16px 8px 16px 16px";
export const BDataGridCellCondensedPadding = "6px 16px";
export const BDataGridCellActionsAnimationDuration = "100ms";
export const BDataGridCellActionsBackgroundGradientBackground = "linear-gradient(270deg, #f7f7f8 0%, rgba(247, 247, 248, 0.99) 8.59%, rgba(247, 247, 248, 0.98) 17.48%, rgba(247, 247, 248, 0.95) 26.53%, rgba(247, 247, 248, 0.92) 35.61%, rgba(247, 247, 248, 0.87) 44.59%, rgba(247, 247, 248, 0.82) 53.34%, rgba(247, 247, 248, 0.75) 61.73%, rgba(247, 247, 248, 0.68) 69.63%, rgba(247, 247, 248, 0.60) 76.9%, rgba(247, 247, 248, 0.52) 83.41%, rgba(247, 247, 248, 0.42) 89.03%, rgba(247, 247, 248, 0.33) 93.63%, rgba(247, 247, 248, 0.22) 97.08%, rgba(247, 247, 248, 0.11) 99.25%, rgba(247, 247, 248, 0.00) 100%)";
export const BDataGridCellActionsBackgroundGradientWidth = "32px";
export const BDataGridCellActionsGap = "8px";
export const BDataGridCellActionsPadding = "0 16px 0 0";
export const BDataGridSelectCellMarginLeft = "8px";
export const BDataGridResizerWidth = "8px";
export const BDataGridFooterPadding = "8px 0";
export const BDataGridLazyLoadBackgroundColor = "#ffffff";
export const BDataGridLazyLoadZIndex = 2;
export const BDataGridLazyLoadBorder = "1px solid #dbdee2";
export const BDataGridLazyLoadPadding = "16px 16px 0px 16px";
export const BDataGridPaginationBackgroundColor = "#ffffff";
export const BDataGridPaginationZIndex = 2;
export const BDataGridPaginationBorderColor = "#dbdee2";
export const BDataGridPaginationBorderWidth = "1px";
export const BDataGridPaginationPadding = "16px 16px 0px 16px";
export const BDataGridPaginationOutlinePadding = "16px";
export const BDataGridPaginationStickyPadding = "16px";
export const BDataGridFiltersBorderColor = "#dbdee2";
export const BDataGridFiltersBorderWidth = "1px";
export const BDataGridFiltersPadding = "16px 0 16px 24px";
export const BDataGridFiltersBackgroundColor = "#ffffff";
export const BDataGridFiltersZIndex = 2;
export const BDataGridFiltersFocusWithinZIndex = 3;
export const BDataGridLoadingIndicatorMargin = "0 0 12px 0";
export const BDataGridConfigSettingsZIndex = 4;
export const BDataGridConfigSettingsButtonAndSeparatorBackgroundColor = "#ffffff";
export const BDataGridConfigSettingsButtonAndSeparatorPadding = "16px 0";
export const BDataGridConfigSettingsButtonMargin = "16px 0";
export const BDataGridConfigSettingsButtonPadding = "0 16px";
export const BDataGridConfigSettingsColumnAndRowMargin = "48px";
export const BDataGridConfigCondensedViewBorderColor = "#dbdee2";
export const BDataGridConfigCondensedViewPadding = "0 16px 12px 16px";
export const BDataGridConfigCondensedViewTitlePadding = "0 0 12px 0";
export const BDataGridConfigPanelHeaderPadding = "0 16px 12px 16px";
export const BDataGridConfigColumnVisibilityPadding = "16px 0 0 0";
export const BDataGridConfigColumnVisibilityHeaderPadding = "0 16px 12px 16px";
export const BDataGridConfigColumnVisibilityShowAllBackgroundColor = "#f3f6f9";
export const BDataGridConfigColumnVisibilityShowAllBorderRadius = "8px";
export const BDataGridConfigColumnVisibilityShowAllMargin = "0 8px 4px 8px";
export const BDataGridConfigColumnVisibilityShowAllPadding = "8px";
export const BDataGridConfigColumnVisibilitySearchMargin = "0 16px 4px 16px";
export const BDataGridConfigColumnVisibilityEmptySearchMessageColor = "#5c687c";
export const BDataGridConfigColumnVisibilityEmptySearchMessagePadding = "12px 16px 0px";
export const BDataGridConfigColumnRowPadding = "8px 16px";
export const BDataGridConfigColumnRowHoverBackgroundColor = "#f3f6f9";
export const BDataGridConfigColumnRowFreezeIconColor = "#8d95a3";
export const BDataGridConfigColumnRowFreezeIconFrozenColor = "#00112c";
export const BDataGridReOrderColumnTargetSeparatorBackgroundColor = "#0070f5";
export const BDataGridReOrderColumnTargetSeparatorWidth = "3px";
export const BDataGridReOrderColumnTargetSeparatorZIndex = 1;
export const BDataGridEmptyStatePadding = "32px 0";
export const BDatePickerDescriptionColor = "#5c687c";
export const BDatePickerDescriptionMarginTop = "8px";
export const BDatePickerDescriptionErrorMarginTop = "4px";
export const BDatePickerLabelMarginBottom = "8px";
export const BDatePickerLabelRequirementColor = "#5c687c";
export const BDatePickerErrorMessageMarginTop = "8px";
export const BDatePickerContainerBorder = "none";
export const BDatePickerContainerBoxShadow = "0 8px 16px rgba(0, 17, 44, 0.1)";
export const BDatePickerContainerWidth = "unset";
export const BDateRangePickerDescriptionColor = "#5c687c";
export const BDateRangePickerDescriptionMarginTop = "8px";
export const BDateRangePickerDescriptionErrorMarginTop = "4px";
export const BDateRangePickerErrorMessageMarginTop = "8px";
export const BDateRangePickerLabelMarginBottom = "8px";
export const BDateRangePickerLabelRequirementColor = "#5c687c";
export const BDateRangePickerCalendarFlexDirection = "row-reverse";
export const BDateRangePickerCalendarCalendarsContainerBackgroundColor = "#f3f6f9";
export const BDateRangePickerCalendarCalendarsContainerBorderRight = "1px solid #dce0e5";
export const BDateRangePickerCalendarCalendarsContainerGap = "24px";
export const BDateRangePickerCalendarCalendarsContainerPadding = "24px";
export const BDateRangePickerCalendarFormGap = "12px";
export const BDateRangePickerCalendarFormMinWidth = "200px";
export const BDateRangePickerCalendarFormPadding = "24px";
export const BDateRangePickerCalendarFormWidth = "auto";
export const BDateRangePickerCalendarFormContainerBorderRight = "none";
export const BDateRangePickerCalendarFormDateFormatExampleColor = "#5c687c";
export const BDateRangePickerCalendarFormDateFormatExampleDisplay = "none";
export const BDateRangePickerCalendarFormInputPaddingTop = "8px";
export const BDateRangePickerCalendarFormInputPadding = "0";
export const BDateRangePickerCalendarFormInputWrapperGap = "10px";
export const BDateRangePickerCalendarLabelFromDisplay = "none";
export const BDateRangePickerCalendarLabelToDisplay = "none";
export const BDateRangePickerContainerBorder = "1px solid #dce0e5";
export const BDateRangePickerContainerBorderRadius = "4px";
export const BDateRangePickerContainerWidth = "unset";
export const BDividerBackgroundColor = "#dce0e5";
export const BDividerBorder = "none";
export const BDividerHeight = "1px";
export const BDropdownLabelColor = "#00112c";
export const BDropdownLabelMarginBottom = "8px";
export const BDropdownDescriptionColor = "#707070";
export const BDropdownDescriptionMarginTop = "8px";
export const BDropdownDescriptionErrorMarginTop = "4px";
export const BDropdownDescriptionCondensedMarginTop = "6px";
export const BDropdownErrorMessageMarginTop = "8px";
export const BDropdownDefaultTextboxBackgroundColor = "#ffffff";
export const BDropdownDefaultTextboxBorderColor = "#dce0e5";
export const BDropdownDefaultTextboxBorderRadius = "6px";
export const BDropdownDefaultTextboxColor = "#00112c";
export const BDropdownDefaultTextboxHeight = "36px";
export const BDropdownDefaultTextboxIconColor = "#818890";
export const BDropdownDefaultTextboxPadding = "0 12px";
export const BDropdownDefaultTextboxPlaceholderColor = "#818890";
export const BDropdownDefaultTextboxFocusBorderColor = "#599bff";
export const BDropdownDefaultTextboxFocusColor = "#00112c";
export const BDropdownDefaultTextboxFocusOutlineColor = "#cce0ff";
export const BDropdownDefaultTextboxFocusOutlineWidth = "3px";
export const BDropdownDefaultTextboxDisabledBackgroundColor = "#f3f6f9";
export const BDropdownDefaultTextboxDisabledBorderColor = "#dce0e5";
export const BDropdownDefaultTextboxDisabledColor = "#69778c";
export const BDropdownDefaultTextboxDisabledIconColor = "#69778c";
export const BDropdownDefaultTextboxReadonlyBackgroundColor = "#eeeff1";
export const BDropdownDefaultTextboxReadonlyBorderColor = "#dbdee2";
export const BDropdownDefaultTextboxReadonlyIconColor = "#8d95a3";
export const BDropdownDefaultTextboxHoverBorderColor = "#a5afbd";
export const BDropdownDefaultTextboxCondensedMinHeight = "28px";
export const BDropdownDefaultTextboxCondensedPadding = "0 10px";
export const BDropdownDefaultTextboxErrorBorder = "1.5px solid #e92929";
export const BDropdownDefaultTextboxErrorFocusBorderColor = "#e92929";
export const BDropdownDefaultTextboxErrorFocusOutlineColor = "#facccc";
export const BDropdownDefaultTextboxErrorHoverBorderColor = "#e92929";
export const BDropdownDefaultTextboxOpenedBottomBorderRadius = 0;
export const BDropdownDefaultTextboxAdditionalItemsBackgroundColor = "#f3f6f9";
export const BDropdownDefaultTextboxAdditionalItemsBorderRadius = "4px";
export const BDropdownDefaultTextboxAdditionalItemsPadding = "4px 6px";
export const BDropdownDefaultTextboxAdditionalItemsCondensedMaxHeight = "24px";
export const BDropdownDefaultTextboxBaseTextboxGap = "12px";
export const BDropdownDefaultTextboxInputSelectedElementMarginRight = "8px";
export const BDropdownOptionsContainerBorderColor = "#dce0e5";
export const BDropdownOptionsContainerBorderRadius = "0 0 6px 6px";
export const BDropdownOptionsContainerBorderWidth = "1px";
export const BDropdownOptionsContainerBoxShadow = "0 8px 8px rgb(0 17 44 / 2%), 0 2px 4px rgb(0 17 44 / 4%)";
export const BDropdownOptionsContainerMarginTop = "-8px";
export const BDropdownSmallTextboxBackgroundColor = "#e5efff";
export const BDropdownSmallTextboxBorderColor = "#cce0ff";
export const BDropdownSmallTextboxBorderRadius = "4px";
export const BDropdownSmallTextboxColor = "#0066ff";
export const BDropdownSmallTextboxHeight = "25px";
export const BDropdownSmallTextboxPadding = "0 8px 4px 8px";
export const BDropdownSmallTextboxPlaceholderColor = "#0066ff";
export const BDropdownSmallTextboxFocusBorderColor = "#599bff";
export const BDropdownSmallTextboxHoverColor = "#5c687c";
export const BDropdownSmallTextboxDisabledBackgroundColor = "#eeeff1";
export const BDropdownSmallTextboxDisabledBorderColor = "#dbdee2";
export const BDropdownSmallTextboxDisabledColor = "#8d95a3";
export const BDropdownSmallTextboxReadonlyBackgroundColor = "#eeeff1";
export const BDropdownSmallTextboxReadonlyBorderColor = "#eeeff1";
export const BDropdownSmallTextboxReadonlyIconColor = "#8d95a3";
export const BDropdownSmallTextboxOpenedBottomBorderRadius = 0;
export const BDropdownSmallTextboxAdditionalItemsBackgroundColor = "#ffffff";
export const BDropdownSmallTextboxAdditionalItemsPadding = "4px 6px";
export const BDropdownSmallTextboxAdditionalItemsMargin = "0 0 0 12px";
export const BDropdownSmallTextboxBaseTextboxGap = "2px";
export const BDropdownSmallTextboxBaseTextboxHeight = "24px";
export const BEmptyStateTitleMarginBottom = "4px";
export const BEmptyStateDetailsColor = "#5c687c";
export const BEmptyStateDetailsMarginBottom = "16px";
export const BErrorMessageColor = "#e22d2d";
export const BErrorMessageIconMarginRight = "8px";
export const BErrorMessageIconHeight = "20px";
export const BFieldLabelColor = "#00112c";
export const BFieldLabelMarginBottom = "8px";
export const BFieldLabelMarginRight = "4px";
export const BFieldLabelCondensedMarginBottom = "6px";
export const BFieldLabelRequirementColor = "#5c687c";
export const BFieldLabelTooltipWrapperMarginRight = "8px";
export const BFilterBarButtonBackgroundColor = "#ffffff";
export const BFilterBarButtonBorder = "1px solid #dce0e5";
export const BFilterBarButtonBorderRadius = "8px";
export const BFilterBarButtonColor = "#394962";
export const BFilterBarButtonHeight = "28px";
export const BFilterBarButtonActiveBorder = "1px solid #00112c";
export const BFilterBarButtonChevronPaddingRight = "12px";
export const BFilterBarButtonDisabledBackgroundColor = "#f3f6f9";
export const BFilterBarButtonDisabledColor = "#8390a3";
export const BFilterBarButtonWithChevronContainerPaddingRight = "8px";
export const BFilterBarButtonContainerGap = "8px";
export const BFilterBarButtonContainerPadding = "4px 10px";
export const BFilterBarButtonCounterBackgroundColor = "#ffffff";
export const BFilterBarButtonCounterBorderRadius = "4px";
export const BFilterBarButtonCounterColor = "#394962";
export const BFilterBarButtonCounterMarginRight = "2px";
export const BFilterBarButtonCounterPadding = "2px 6px";
export const BFilterBarButtonCounterDisabledBackgroundColor = "#ffffff";
export const BFilterBarButtonCounterDisabledColor = "#a5afbd";
export const BFilterBarButtonCounterHoverBackgroundColor = "#ffffff";
export const BFilterBarButtonFilterOpenBorder = "1px solid #00112c";
export const BFilterBarButtonHoverBackgroundColor = "#ffffff";
export const BFilterBarButtonHoverColor = "#5c687c";
export const BFilterBarButtonLabelContainerGap = "4px";
export const BFilterBarButtonLabelLineHeight = "15px";
export const BFilterBarButtonSecondaryLabelColor = "#5c687c";
export const BFilterBarButtonWithCounterBackgroundColor = "#20304c";
export const BFilterBarButtonWithCounterBorder = "1px solid #20304c";
export const BFilterBarButtonWithCounterColor = "#ffffff";
export const BFilterBarButtonWithCounterActiveBackgroundColor = "#00112c";
export const BFilterBarButtonWithCounterActiveBorderColor = "#00112c";
export const BFilterBarButtonWithCounterActiveColor = "#ffffff";
export const BFilterBarButtonWithCounterActiveCounterColor = "#00112c";
export const BFilterBarGap = "8px";
export const BFilterBarPaddingRight = "16px";
export const BFilterBarZIndex = 5;
export const BFilterBarPersistentFiltersBorderColor = "#f3f6f9";
export const BFilterBarPersistentFiltersPaddingRight = "8px";
export const BFilterBarActionsGap = "8px";
export const BFilterBarBeforeFiltersGap = "8px";
export const BFilterBarContainerGap = "8px";
export const BFilterBarFiltersContainerGap = "8px";
export const BFilterBarScrollerContentGap = "8px";
export const BFilterBarFiltersWrapperGap = "8px";
export const BFilterBarFiltersWrapperPaddingRight = "16px";
export const BFilterBarInputFilterPadding = "12px 16px";
export const BFilterBarInputFilterOnlySlotPadding = "0px";
export const BFilterBarBooleanFilterPadding = "12px 16px";
export const BFilterBarBooleanFilterOnlySlotPadding = "0px";
export const BFilterBarRangeFilterPadding = "12px 16px";
export const BFilterBarRangeFilterOnlySlotPadding = "0px";
export const BFilterBarRangeFilterItemPadding = "12px";
export const BFilterBarSelectFilterDividerBackgroundColor = "#dbdee2";
export const BFilterBarSelectFilterDividerBorderWidth = "0";
export const BFilterBarSelectFilterDividerMargin = "8px 0px 0px";
export const BFilterBarSelectFilterSearchBarPadding = "12px 16px 0px";
export const BFilterBarDateFilterPadding = "16px 12px 16px";
export const BFilterBarDateFilterDividerMarginTop = "12px";
export const BFilterBarDateRangeFilterActionsPadding = "0 16px 12px 16px";
export const BFilterBarDateRangeFilterDividerMargin = "12px 0";
export const BFilterBarAllFiltersModalFiltersGap = "24px";
export const BFilterBarAllFiltersModalFiltersTitleColor = "#707070";
export const BFilterBarAllFiltersModalFiltersListGap = "6px";
export const BFilterBarAllFiltersModalFiltersListMargin = "0px";
export const BFilterBarAllFiltersModalFiltersListPadding = "0px";
export const BFilterBarAllFiltersModalFiltersListEmptySearchMessageColor = "#5c687c";
export const BFilterBarAllFiltersModalFiltersListEmptySearchMessagePadding = "0px 16px";
export const BFilterBarAllFiltersModalFiltersListTitleColor = "#5c687c";
export const BFilterBarAllFiltersModalFiltersListTitlePadding = "0px 16px";
export const BFilterBarAllFiltersModalSelectedFilterContentPadding = "8px 32px 8px 8px";
export const BFilterBarAllFiltersModalContentTextGap = "6px";
export const BFilterBarAllFiltersModalContentTextMarginBottom = "32px";
export const BFilterBarAllFiltersModalContentPadding = "0px";
export const BFilterBarAllFiltersModalClearButtonPaddingBottom = "8px";
export const BFilterBarAllFiltersModalSearchBarPadding = "0px 16px";
export const BFilterBarAllFiltersModalHeaderBackgroundColor = "#ffffff";
export const BFilterButtonBackgroundColor = "#ffffff";
export const BFilterButtonFontColor = "#394962";
export const BFilterButtonPadding = "6px 16px";
export const BFilterButtonActiveBackgroundColor = "#e5efff";
export const BFilterButtonDefaultContainerGap = "16px";
export const BFilterButtonDisabledBackgroundColor = "#f3f6f9";
export const BFilterButtonDisabledFontColor = "#8390a3";
export const BFilterButtonCounterBackgroundColor = "#ffffff";
export const BFilterButtonCounterBorderRadius = "4px";
export const BFilterButtonCounterFontColor = "#394962";
export const BFilterButtonCounterPadding = "2px 6px";
export const BFilterButtonCounterActiveBackgroundColor = "#0066ff";
export const BFilterButtonCounterOpenBackgroundColor = "#e5efff";
export const BFilterButtonCounterDisabledBackgroundColor = "#ffffff";
export const BFilterButtonCounterDisabledFontColor = "#a5afbd";
export const BFilterButtonCounterHoverBackgroundColor = "#e3e5e9";
export const BFilterButtonHoverBackgroundColor = "#ffffff";
export const BFilterButtonHoverTextColor = "#5c687c";
export const BFilterButtonLabelLineHeight = "15px";
export const BFilterButtonModalCounterFocusBackgroundColor = "#0066ff";
export const BFilterButtonModalCounterBackgroundColor = "#0066ff";
export const BFilterButtonModalCounterColor = "#ffffff";
export const BFilterButtonModalFocusBackgroundColor = "#ffffff";
export const BFilterButtonModalFocusBoxShadow = "inset 0 0 0 1px #dce0e5";
export const BFilterButtonModalHoverBackgroundColor = "#ffffff";
export const BFilterButtonModalHoverColor = "#00112c";
export const BFilterButtonModalOpenBackgroundColor = "#e5efff";
export const BFilterButtonModalBorderRadius = "4px";
export const BFilterButtonModalDisabledBackgroundColor = "#ffffff";
export const BFilterButtonModalDisabledColor = "#a5afbd";
export const BFilterButtonModalDisabledCounterBackgroundColor = "#a5afbd";
export const BFilterButtonModalDisabledCounterColor = "#ffffff";
export const BFixedScrollerButtonMarginTop = "4px";
export const BFixedScrollerLeftBackground = "linear-gradient(90deg, #ffffff 75%, rgba(255, 255, 255, 0) 100%)";
export const BFixedScrollerLeftPaddingRight = "20px";
export const BFixedScrollerRightBackground = "linear-gradient(270deg, #ffffff 75%, rgba(255, 255, 255, 0) 100%)";
export const BFixedScrollerRightPaddingLeft = "20px";
export const BFixedScrollerSideSpacingPaddingBottom = "4px";
export const BFooterActionsWrapperRowGap = "8px";
export const BFormLayoutGap = "20px";
export const BFormLayoutActionsGap = "8px";
export const BFormLayoutActionsContainerMarginTop = "20px";
export const BFormLayoutErrorMessageMarginBottom = "20px";
export const BFormLayoutFieldTipGap = "12px";
export const BFormLayoutGroupGap = "10px";
export const BFormLayoutTitleDescriptionColor = "#5c687c";
export const BFormLayoutTitleSubsectionMarginTop = "20px";
export const BHeaderWrapperGap = "40px";
export const BHeaderTitleColor = "#00112c";
export const BHeaderTitleMarginTop = "2px";
export const BHeaderInfoIconMarginLeft = "10px";
export const BHeaderInfoIconMarginTop = "4px";
export const BHeaderTimestampColor = "#5c687c";
export const BHeaderTimestampMarginBlock = "4px";
export const BHeaderActionsMarginTop = "16px";
export const BHeaderActionsMarginTopDesktop = 0;
export const BHeaderComponentTitleMarginTop = "4px";
export const BInputFieldLabelColor = "#00112c";
export const BInputFieldLabelMarginBottom = "8px";
export const BInputFieldLabelRequirementColor = "#5c687c";
export const BInputFieldInputBoxBackgroundColor = "#ffffff";
export const BInputFieldInputBoxColor = "#00112c";
export const BInputFieldInputBoxBorderColor = "#8d95a3";
export const BInputFieldInputBoxBorderRadius = "8px";
export const BInputFieldInputBoxHeight = "40px";
export const BInputFieldInputBoxPadding = "8px 16px";
export const BInputFieldInputBoxHoverBorderColor = "#00112c";
export const BInputFieldInputBoxCondensedPadding = "4px 10px";
export const BInputFieldInputBoxCondensedWithPaymentMethodPaddingLeft = "6px";
export const BInputFieldInputBoxDisabledBackgroundColor = "#eeeff1";
export const BInputFieldInputBoxDisabledBorderColor = "#dbdee2";
export const BInputFieldInputBoxErrorBorder = "1.5px solid #e22d2d";
export const BInputFieldInputBoxErrorBorderColor = "#e22d2d";
export const BInputFieldInputBoxErrorOutline = "0";
export const BInputFieldInputBoxFocusBorderColor = "#00112c";
export const BInputFieldInputBoxFocusBorderWidth = "1px";
export const BInputFieldInputBoxFocusOutline = "0";
export const BInputFieldInputBoxSmallPaddingLeft = "6px 12px 6px 3px";
export const BInputFieldInputBoxSmallPaddingRight = "6px 3px 6px 12px";
export const BInputFieldInputColor = "#00112c";
export const BInputFieldInputMargin = "8px 12px";
export const BInputFieldInputPadding = "0px";
export const BInputFieldInputFontSize = "15px";
export const BInputFieldInputCondensedMargin = "4px 10px";
export const BInputFieldInputPlaceholderColor = "#8d95a3";
export const BInputFieldInputDisabledBackgroundColor = "#eeeff1";
export const BInputFieldInputDisabledColor = "#8d95a3";
export const BInputFieldIconAfterMarginLeft = "12px";
export const BInputFieldIconAfterMarginRight = "12px";
export const BInputFieldIconAfterCondensedMarginLeft = "10px";
export const BInputFieldIconAfterCondensedMarginRight = "10px";
export const BInputFieldIconBeforeMarginLeft = "12px";
export const BInputFieldIconBeforeMarginRight = "12px";
export const BInputFieldIconBeforeCondensedMarginLeft = "10px";
export const BInputFieldIconBeforeCondensedMarginRight = "10px";
export const BInputFieldIconsColor = "#8d95a3";
export const BInputFieldStaticValueBackgroundColor = "#f3f6f9";
export const BInputFieldStaticValueColor = "#00112c";
export const BInputFieldStaticValueMargin = "12px";
export const BInputFieldStaticValuePadding = "4px 6px";
export const BInputFieldStaticValueBorderRadius = "4px";
export const BInputFieldStaticValueDisabledBackgroundColor = "#eeeff1";
export const BInputFieldStaticValueDisabledColor = "#8d95a3";
export const BInputFieldStaticValueEndMarginRight = "4px";
export const BInputFieldStaticValueEndCondensedMarginLeft = "10px";
export const BInputFieldStaticValueEndCondensedMarginRight = "4px";
export const BInputFieldStaticValueStartMarginLeft = "4px";
export const BInputFieldStaticValueStartCondensedMarginLeft = "4px";
export const BInputFieldStaticValueStartCondensedMarginRight = "10px";
export const BInputFieldPaymentMethodMarginRight = "12px";
export const BInputFieldPaymentMethodMarginLeft = "6px";
export const BInputFieldPaymentMethodCondensedMarginLeft = "6px";
export const BInputFieldPaymentMethodCondensedMarginRight = "10px";
export const BInputFieldDropdownBorderRadius = "8px";
export const BInputFieldDropdownPaddingRight = "12px";
export const BInputFieldDropdownEndMarginRight = "4px";
export const BInputFieldDropdownEndCondensedMarginRight = "20px";
export const BInputFieldDropdownEndCondensedPaddingLeft = "10px";
export const BInputFieldDropdownSmallTextboxCondensedPadding = "0 6px";
export const BInputFieldDropdownStartMarginLeft = "4px";
export const BInputFieldDropdownStartCondensedMarginLeft = "4px";
export const BInputFieldDropdownStartCondensedPaddingLeft = "10px";
export const BInputFieldDescriptionColor = "#5c687c";
export const BInputFieldDescriptionMarginTop = "8px";
export const BInputFieldDescriptionPadding = "0px";
export const BInputFieldErrorMessageMarginTop = "8px";
export const BLinkBorderRadius = "2px";
export const BLinkColor = "#0066ff";
export const BLinkTextDecoration = "none";
export const BLinkHoverColor = "#0066ff";
export const BLinkHoverTextDecoration = "underline";
export const BLinkActiveColor = "#0066ff";
export const BLinkQuietColor = "#0066ff";
export const BLinkQuietHoverColor = "#0066ff";
export const BLinkQuietActiveColor = "#0066ff";
export const BListboxBackgroundColor = "#ffffff";
export const BListboxPadding = "0px";
export const BListboxEmptySearchMessageColor = "#5c687c";
export const BListboxEmptySearchMessagePadding = "8px 16px";
export const BListboxLazyLoadMargin = "8px 0";
export const BListboxOptionPadding = "8px 16px";
export const BListboxOptionActiveBackgroundColor = "#0066ff";
export const BListboxOptionActiveFontColor = "#ffffff";
export const BListboxOptionActiveDescriptionColor = "#ffffff";
export const BListboxOptionDisabledFontColor = "#8d95a3";
export const BListboxOptionFocusBackgroundColor = "#eeeff1";
export const BListboxOptionFocusFontColor = {"100":{"value":"#f7f7f7","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#F7F7F7","type":"color"},"name":"BColorBlack100","attributes":{"category":"color","type":"black","item":"100"},"path":["color","black","100"]},"200":{"value":"#efefef","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#EFEFEF","type":"color"},"name":"BColorBlack200","attributes":{"category":"color","type":"black","item":"200"},"path":["color","black","200"]},"300":{"value":"#e5e5e5","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#E5E5E5","type":"color"},"name":"BColorBlack300","attributes":{"category":"color","type":"black","item":"300"},"path":["color","black","300"]},"400":{"value":"#dddddd","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#DDDDDD","type":"color"},"name":"BColorBlack400","attributes":{"category":"color","type":"black","item":"400"},"path":["color","black","400"]},"500":{"value":"#d5d5d5","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#D5D5D5","type":"color"},"name":"BColorBlack500","attributes":{"category":"color","type":"black","item":"500"},"path":["color","black","500"]},"600":{"value":"#cdcdcd","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#CDCDCD","type":"color"},"name":"BColorBlack600","attributes":{"category":"color","type":"black","item":"600"},"path":["color","black","600"]},"700":{"value":"#c5c5c5","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#C5C5C5","type":"color"},"name":"BColorBlack700","attributes":{"category":"color","type":"black","item":"700"},"path":["color","black","700"]},"800":{"value":"#bdbdbd","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#BDBDBD","type":"color"},"name":"BColorBlack800","attributes":{"category":"color","type":"black","item":"800"},"path":["color","black","800"]},"900":{"value":"#b5b5b5","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#B5B5B5","type":"color"},"name":"BColorBlack900","attributes":{"category":"color","type":"black","item":"900"},"path":["color","black","900"]},"1000":{"value":"#adadad","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#ADADAD","type":"color"},"name":"BColorBlack1000","attributes":{"category":"color","type":"black","item":"1000"},"path":["color","black","1000"]},"1100":{"value":"#a5a5a5","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#A5A5A5","type":"color"},"name":"BColorBlack1100","attributes":{"category":"color","type":"black","item":"1100"},"path":["color","black","1100"]},"1200":{"value":"#9d9d9d","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#9D9D9D","type":"color"},"name":"BColorBlack1200","attributes":{"category":"color","type":"black","item":"1200"},"path":["color","black","1200"]},"1300":{"value":"#959595","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#959595","type":"color"},"name":"BColorBlack1300","attributes":{"category":"color","type":"black","item":"1300"},"path":["color","black","1300"]},"1400":{"value":"#8d8d8d","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#8D8D8D","type":"color"},"name":"BColorBlack1400","attributes":{"category":"color","type":"black","item":"1400"},"path":["color","black","1400"]},"1500":{"value":"#868686","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#868686","type":"color"},"name":"BColorBlack1500","attributes":{"category":"color","type":"black","item":"1500"},"path":["color","black","1500"]},"1600":{"value":"#7e7e7e","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#7E7E7E","type":"color"},"name":"BColorBlack1600","attributes":{"category":"color","type":"black","item":"1600"},"path":["color","black","1600"]},"1700":{"value":"#767676","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#767676","type":"color"},"name":"BColorBlack1700","attributes":{"category":"color","type":"black","item":"1700"},"path":["color","black","1700"]},"1800":{"value":"#6f6f6f","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#6F6F6F","type":"color"},"name":"BColorBlack1800","attributes":{"category":"color","type":"black","item":"1800"},"path":["color","black","1800"]},"1900":{"value":"#686868","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#686868","type":"color"},"name":"BColorBlack1900","attributes":{"category":"color","type":"black","item":"1900"},"path":["color","black","1900"]},"2000":{"value":"#616161","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#616161","type":"color"},"name":"BColorBlack2000","attributes":{"category":"color","type":"black","item":"2000"},"path":["color","black","2000"]},"2100":{"value":"#595959","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#595959","type":"color"},"name":"BColorBlack2100","attributes":{"category":"color","type":"black","item":"2100"},"path":["color","black","2100"]},"2200":{"value":"#525252","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#525252","type":"color"},"name":"BColorBlack2200","attributes":{"category":"color","type":"black","item":"2200"},"path":["color","black","2200"]},"2300":{"value":"#4b4b4b","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#4B4B4B","type":"color"},"name":"BColorBlack2300","attributes":{"category":"color","type":"black","item":"2300"},"path":["color","black","2300"]},"2400":{"value":"#454545","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#454545","type":"color"},"name":"BColorBlack2400","attributes":{"category":"color","type":"black","item":"2400"},"path":["color","black","2400"]},"2500":{"value":"#3e3e3e","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#3E3E3E","type":"color"},"name":"BColorBlack2500","attributes":{"category":"color","type":"black","item":"2500"},"path":["color","black","2500"]},"2600":{"value":"#373737","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#373737","type":"color"},"name":"BColorBlack2600","attributes":{"category":"color","type":"black","item":"2600"},"path":["color","black","2600"]},"2700":{"value":"#313131","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#313131","type":"color"},"name":"BColorBlack2700","attributes":{"category":"color","type":"black","item":"2700"},"path":["color","black","2700"]},"2800":{"value":"#2a2a2a","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#2A2A2A","type":"color"},"name":"BColorBlack2800","attributes":{"category":"color","type":"black","item":"2800"},"path":["color","black","2800"]},"2900":{"value":"#242424","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#242424","type":"color"},"name":"BColorBlack2900","attributes":{"category":"color","type":"black","item":"2900"},"path":["color","black","2900"]},"3000":{"value":"#1e1e1e","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#1E1E1E","type":"color"},"name":"BColorBlack3000","attributes":{"category":"color","type":"black","item":"3000"},"path":["color","black","3000"]},"3100":{"value":"#181818","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#181818","type":"color"},"name":"BColorBlack3100","attributes":{"category":"color","type":"black","item":"3100"},"path":["color","black","3100"]},"3200":{"value":"#121212","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#121212","type":"color"},"name":"BColorBlack3200","attributes":{"category":"color","type":"black","item":"3200"},"path":["color","black","3200"]},"3300":{"value":"#0c0c0c","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#0C0C0C","type":"color"},"name":"BColorBlack3300","attributes":{"category":"color","type":"black","item":"3300"},"path":["color","black","3300"]},"3400":{"value":"#070707","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#070707","type":"color"},"name":"BColorBlack3400","attributes":{"category":"color","type":"black","item":"3400"},"path":["color","black","3400"]},"_":{"value":"#000000","type":"color","filePath":"tokens/bento/base/definitions/_color.yaml","isSource":false,"original":{"value":"#000","type":"color"},"name":"BColorBlack","attributes":{"category":"color","type":"black","item":"_"},"path":["color","black","_"]}};
export const BListboxOptionHoverBackgroundColor = "#f3f6f9";
export const BListboxOptionHoverFontColor = "#0066ff";
export const BListboxOptionSelectedDescriptionColor = "#ffffff";
export const BListboxOptionSelectedFontColor = "#ffffff";
export const BListboxOptionSelectedBackgroundColor = "#0066ff";
export const BListboxOptionDescriptionColor = "#707070";
export const BListboxOptionDescriptionMarginTop = "4px";
export const BListboxOptionTagLabelColor = "#00112c";
export const BListboxOptionTagLabelPadding = "0 0 0 8px";
export const BListboxSingleSelectCheckIconMargin = "16px";
export const BListboxSingleSelectCategoryMarginBottom = "16px";
export const BListboxSingleSelectCategoryLabelTextPadding = "8px 16px";
export const BListboxSingleSelectCategoryLabelTextColor = "#5c687c";
export const BListboxMultiSelectDividerBackgroundColor = "#dbdee2";
export const BListboxMultiSelectDividerMargin = "0px";
export const BListboxMultiSelectOptionsGap = "16px";
export const BListboxMultiSelectOptionsCategoryOptionPaddingLeft = "32px";
export const BListboxMultiSelectOptionsStaticCategoryOptionPaddingLeft = "16px";
export const BListboxMultiSelectCategoryDisabledLabelTextColor = "#8d95a3";
export const BListboxMultiSelectCategoryStaticCategoryGap = "16px";
export const BListboxMultiSelectCategoryStaticCategoryLabelTextColor = "#5c687c";
export const BListboxMultiSelectCategoryOptionGap = "16px";
export const BLoadingButtonPrimaryActiveBackgroundColor = "#0048b3";
export const BLoadingButtonPrimarySuccessBackgroundColor = "#0abf53";
export const BLoadingButtonPrimarySuccessColor = "#ffffff";
export const BLoadingButtonPrimarySuccessActiveBackgroundColor = "#0abf53";
export const BLoadingButtonPrimarySuccessDisabledBackgroundColor = "#a5afbd";
export const BLoadingButtonPrimarySuccessDisabledColor = "#dce0e5";
export const BLoadingButtonPrimarySuccessHoverBackgroundColor = "#0abf53";
export const BLoadingButtonSecondaryActiveBackgroundColor = "#ffffff";
export const BLoadingButtonSecondaryActiveColor = "#a5afbd";
export const BLoadingButtonSecondarySuccessBackgroundColor = "#0abf53";
export const BLoadingButtonSecondarySuccessBorderColor = "#000000";
export const BLoadingButtonSecondarySuccessColor = "#ffffff";
export const BLoadingButtonSecondarySuccessHoverBackgroundColor = "#0abf53";
export const BLoadingButtonSecondarySuccessHoverBorderColor = "#000000";
export const BLoadingButtonSecondarySuccessHoverBoxShadow = "unset";
export const BLoadingButtonSecondarySuccessHoverColor = "#ffffff";
export const BLoadingButtonSecondarySuccessActiveBackgroundColor = "#0abf53";
export const BLoadingButtonSecondarySuccessActiveBorderColor = "#000000";
export const BLoadingButtonSecondarySuccessDisabledBackgroundColor = "#dce0e5";
export const BLoadingButtonSecondarySuccessDisabledBorderColor = "#000000";
export const BLoadingButtonSecondarySuccessDisabledColor = "#a5afbd";
export const BLoadingButtonTertiaryActiveBackgroundColor = "#eef1f5";
export const BLoadingIndicatorBackgroundColor = "#0066ff";
export const BLoadingIndicatorColor = "#0066ff";
export const BLoadingIndicatorHeight = "24px";
export const BLoadingIndicatorWidth = "24px";
export const BLoadingIndicatorSmallBorderWidth = "2px";
export const BLoadingIndicatorSmallHeight = "16px";
export const BLoadingIndicatorSmallWidth = "16px";
export const BLoadingIndicatorLargeBorderWidth = "4px";
export const BLoadingIndicatorLargeHeight = "48px";
export const BLoadingIndicatorLargeWidth = "48px";
export const BLoadingIndicatorInverseBackgroundColor = "#ffffff";
export const BLoadingIndicatorInverseColor = "#ffffff";
export const BMenuItemColor = "#00112c";
export const BMenuItemColorInverse = "#ffffff";
export const BMenuItemPadding = "8px 16px";
export const BMenuItemActiveBackgroundColor = "#eeeff1";
export const BMenuItemActiveBackgroundColorInverse = "#8d95a3";
export const BMenuItemActiveCriticalBackgroundColor = "#fef4f4";
export const BMenuItemActiveCriticalBackgroundColorInverse = "#611313";
export const BMenuItemActiveCriticalColor = "#791818";
export const BMenuItemActiveCriticalColorInverse = "#fddfdf";
export const BMenuItemCriticalColor = "#e22d2d";
export const BMenuItemCriticalColorInverse = "#f99c9c";
export const BMenuItemDisabledColor = "#8d95a3";
export const BMenuItemDisabledColorInverse = "#747f8f";
export const BMenuItemFocusBackgroundColor = "#dce0e5";
export const BMenuItemFocusCriticalBackgroundColor = "#fef4f4";
export const BMenuItemFocusCriticalBackgroundColorInverse = "#611313";
export const BMenuItemFocusCriticalColor = "#791818";
export const BMenuItemFocusCriticalColorInverse = "#fddfdf";
export const BMenuItemHoverBackgroundColor = "#f3f6f9";
export const BMenuItemHoverCriticalBackgroundColor = "#fef4f4";
export const BMenuItemHoverCriticalBackgroundColorInverse = "#611313";
export const BMenuItemHoverCriticalColor = "#c72727";
export const BMenuItemHoverCriticalColorInverse = "#fbbdbd";
export const BMenuItemSecondaryTextColor = "#5c687c";
export const BMenuItemSecondaryTextColorInverse = "#9ea6b1";
export const BMenuItemSecondaryTextPaddingLeft = "16px";
export const BMenuItemIconLeftPadding = "16px";
export const BMenuItemIconRightColor = "#00112c";
export const BMenuItemIconRightColorInverse = "#ffffff";
export const BMenuItemIconRightPadding = "16px";
export const BMenuItemIndentPaddingLeft = "48px";
export const BMenuItemListBackgroundColor = "#ffffff";
export const BMenuItemListBackgroundColorInverse = "#00112c";
export const BMenuItemListPadding = "8px 0px";
export const BMenuItemListDividerMarginBlock = "8px";
export const BMenuSubmenuBackgroundColor = "#ffffff";
export const BMenuSubmenuBackgroundColorInverse = "#00112c";
export const BMenuSubmenuDividerBorder = "1px solid #dbdee2";
export const BMenuSubmenuDividerBorderInverse = "1px solid #2f3e56";
export const BMenuSubmenuDividerPadding = "8px";
export const BMenuSubmenuDividerMarginBlock = "8px";
export const BMenuSubmenuAnimationDuration = "150ms";
export const BModalBorderRadius = "12px";
export const BModalPadding = "0px";
export const BModalContainerGap = "16px";
export const BModalContainerBackgroundColor = "#ffffff";
export const BModalContainerColumnLayoutBorder = "1px solid #dce0e5";
export const BModalContainerColumnLayoutPadding = "24px";
export const BModalHeaderContainerZIndex = 1;
export const BModalBackdropBackgroundColor = "#00112c";
export const BModalAnimationDuration = "150ms";
export const BModalWrapperPadding = "24px";
export const BModalWrapperGap = "24px";
export const BPaginationNavigationSmallContainerGap = "10px";
export const BPaginationContextMargin = "0 12px";
export const BPaginationControlsGap = "16px";
export const BPaginationDividerBorderColor = "#dbdee2";
export const BPaginationDividerBorderWidth = "1px";
export const BPaginationDividerMargin = "0 16px";
export const BPaginationPageNavigatorSmallContainerGap = "16px";
export const BPaginationResultsPerPageMargin = "0 12px";
export const BPaymentMethodLabelColor = "#00112c";
export const BPaymentMethodLabelPadding = "0 0 0 12px";
export const BPopoverBackgroundColor = "#ffffff";
export const BPopoverBorder = "1px solid #dce0e5";
export const BPopoverBorderRadius = "4px";
export const BPopoverBoxShadow = "0px 8px 8px rgba(0, 17, 44, 0.04), 0px 2px 4px rgba(0, 17, 44, 0.08);";
export const BPopoverZIndex = 5;
export const BPopoverDismissButtonOnImageIconColor = "#ffffff";
export const BPopoverDismissButtonOnImageIconHoveredColor = "#ffffff";
export const BPopoverDismissButtonOnImageIconActiveColor = "#ffffff";
export const BPopoverWithDividerFooterBackgroundColor = "#f3f6f9";
export const BPopoverWithDividerFooterBorderTop = "1px solid #dbdee2";
export const BPopoverWithDividerFooterPaddingTop = "12px";
export const BPopoverWithDividerContentPadding = "0px";
export const BPopoverWithoutSpacePadding = "0px";
export const BPopoverContentPadding = "16px";
export const BPopoverContentWithTitlePadding = "12px 16px";
export const BPopoverHeaderPadding = "12px 16px 0px 16px";
export const BPopoverHeaderTitleColumnGap = "8px";
export const BPopoverImageHeaderPadding = "16px 16px 0px 16px";
export const BPopoverImageDismissIconRight = "16px";
export const BPopoverImageDismissIconTop = "16px";
export const BPopoverFooterPadding = "0px 16px 12px 16px";
export const BPopoverAnimationDuration = "150ms";
export const BPopperZIndex = 5;
export const BRadioButtonOffsetTop = "4px";
export const BRadioButtonBorderRadius = "12px";
export const BRadioButtonSpaceAroundIncrease = "4px";
export const BRadioButtonUncheckedBackgroundColor = "#ffffff";
export const BRadioButtonUncheckedBorder = "1px solid #dce0e5";
export const BRadioButtonUncheckedHoverBackgroundColor = "#ffffff";
export const BRadioButtonUncheckedHoverBorderColor = "#69778c";
export const BRadioButtonUncheckedActiveBackgroundColor = "#ffffff";
export const BRadioButtonUncheckedDisabledBackgroundColor = "#dce0e5";
export const BRadioButtonUncheckedDisabledBorderColor = "#dce0e5";
export const BRadioButtonUncheckedReadonlyBackgroundColor = "#eeeff1";
export const BRadioButtonUncheckedReadonlyBorderColor = "#dbdee2";
export const BRadioButtonUncheckedErrorBackgroundColor = "#e22d2d";
export const BRadioButtonUncheckedErrorBorder = "1.5px solid #e22d2d";
export const BRadioButtonUncheckedErrorBorderWidth = "1.5px;";
export const BRadioButtonCheckedBackgroundColor = "#0066ff";
export const BRadioButtonCheckedBorderColor = "#0066ff";
export const BRadioButtonCheckedHoverBackgroundColor = "#0066ff";
export const BRadioButtonCheckedHoverBorderColor = "#0048b3";
export const BRadioButtonCheckedActiveBackgroundColor = "#0066ff";
export const BRadioButtonCheckedActiveBorderColor = "#0066ff";
export const BRadioButtonCheckedDisabledBackgroundColor = "#dce0e5";
export const BRadioButtonCheckedDisabledBorderColor = "#dce0e5";
export const BRadioButtonCheckedReadonlyBackgroundColor = "#eeeff1";
export const BRadioButtonCheckedReadonlyBorderColor = "#dbdee2";
export const BRadioButtonCheckedErrorBackgroundColor = "#e22d2d";
export const BRadioButtonCheckedErrorBorderColor = "#e22d2d";
export const BRadioButtonLabelContainerPadding = "16px";
export const BRadioButtonLabelContainerRowGap = "4px";
export const BRadioButtonLabelFontColor = "#00112c";
export const BRadioButtonLabelDisabledFontColor = "#8390a3";
export const BRadioButtonContentPadding = "12px 0px 8px";
export const BRadioButtonDescriptionFontColor = "#5c687c";
export const BRadioButtonDescriptionDisabledFontColor = "#5c687c";
export const BRadioButtonKnobColor = "#ffffff";
export const BRadioButtonKnobOffset = "4px";
export const BRadioButtonKnobOffsetError = "3.5px";
export const BRadioButtonKnobSize = "6px";
export const BRadioButtonKnobTransition = "transform 250ms cubic-bezier(0.12, 0.6, 0.4, 0.95)";
export const BRadioButtonKnobDisabledColor = "#707070";
export const BRadioButtonKnobReadonlyColor = "#00112c";
export const BRadioButtonSubLabelFontColor = "#707070";
export const BRadioButtonSubLabelDisabledFontColor = "#8390a3";
export const BRadioGroupVerticalGap = "12px";
export const BRadioGroupHorizontalGap = "32px";
export const BRadioGroupErrorMessageMarginTop = "12px";
export const BSegmentedControlBackgroundColor = "#eeeff1";
export const BSegmentedControlBorderRadius = "12px";
export const BSegmentedControlPadding = "4px";
export const BSegmentedControlButtonBorderRadius = "8px";
export const BSegmentedControlButtonColor = "#00112c";
export const BSegmentedControlButtonPadding = "4px 20px";
export const BSegmentedControlButtonIconMarginRight = "8px";
export const BSegmentedControlButtonCheckedBackgroundColor = "#ffffff";
export const BSegmentedControlButtonCheckedPadding = "4px 8px";
export const BSegmentedControlButtonDisabledColor = "#8d95a3";
export const BSegmentedControlButtonHoverBackgroundColor = "#e3e5e9";
export const BSidepanelBackgroundColor = "#ffffff";
export const BSidepanelBorderRadius = "0";
export const BSidepanelWidth = "480px";
export const BSidepanelCloseButtonHeight = "64px";
export const BSidepanelCloseButtonRight = "24px";
export const BSidepanelCloseButtonZIndex = 1;
export const BSidepanelAnimationDuration = "250ms";
export const BSidepanelBackdropBackgroundColor = "#00112c";
export const BSidepanelPageHeaderBackgroundColor = "#f3f6f9";
export const BSidepanelPageHeaderFlexBasis = "64px";
export const BSidepanelPageHeaderGap = "12px";
export const BSidepanelPageHeaderPadding = "16px 56px 16px 40px";
export const BSidepanelPageHeaderBorderBottom = "1px solid #dce0e5";
export const BSidepanelPageHeaderWithBorderBorderBottom = "1px solid #dce0e5";
export const BSidepanelPageContentPadding = "0 40px";
export const BSidepanelPageActionsBackgroundColor = "#ffffff";
export const BSidepanelPageActionsHeight = "68px";
export const BSidepanelPageActionsPadding = "16px 40px";
export const BSidepanelPageActionsBorderTop = "1px solid #dce0e5";
export const BSidepanelPageActionsWithBorderBorderTop = "1px solid #dce0e5";
export const BSidepanelPageAnimationDuration = "150ms";
export const BStatusColumnGap = "8px";
export const BStatusIndicatorSize = "12px";
export const BStatusIndicatorBlueBackgroundColor = "#0066ff";
export const BStatusIndicatorGreenBackgroundColor = "#0abf53";
export const BStatusIndicatorGreyBackgroundColor = "#c0c5cc";
export const BStatusIndicatorOrangeBackgroundColor = "#ff9500";
export const BStatusIndicatorRedBackgroundColor = "#e50000";
export const BStatusIndicatorYellowBackgroundColor = "#f7bc00";
export const BStepperGap = "6px";
export const BStepperMaxWidth = "200px";
export const BStepperTextStepperColor = "#5c687c";
export const BStepperStepButtonColor = "#5c687c";
export const BStepperStepButtonPadding = "6px 8px";
export const BStepperStepButtonGap = "8px";
export const BStepperStepButtonBorderRadius = "8px";
export const BStepperStepButtonHoverBackgroundColor = "#f7f7f8";
export const BStepperStepContentColor = "#5c687c";
export const BStepperStepContentHoverColor = "#00112c";
export const BStepperStepContentCompletedColor = "#00112c";
export const BStepperStepContentActiveColor = "#00112c";
export const BStepperStepIndexColor = "#ffffff";
export const BStepperStepIndexWidth = "20px";
export const BStepperStepIndexHeight = "20px";
export const BStepperStepIndexBorderRadius = "12px";
export const BStepperStepIndexBackgroundColor = "#c0c5cc";
export const BStepperStepIndexActiveBackgroundColor = "#00112c";
export const BStepperStepIndexCompletedBackgroundColor = "#07893c";
export const BStepperStepIndexCompletedColor = "#00112c";
export const BStepperStepSubstepsGap = "6px";
export const BStepperStepSubstepsMarginTop = "6px";
export const BStepperStepSubstepsButtonPaddingLeft = "24px";
export const BStepperStepSubstepsContentColor = "#5c687c";
export const BStepperStepSubstepsContentHoverColor = "#00112c";
export const BStepperStepSubstepsContentCompletedColor = "#00112c";
export const BStepperStepSubstepsContentActiveColor = "#00112c";
export const BStepperStepSubstepsIndexColor = "#c0c5cc";
export const BStepperStepSubstepsIndexCompletedColor = "#07893c";
export const BStepperStepSubstepsIndexActiveColor = "#00112c";
export const BStructuredListItemPadding = "6px 0";
export const BStructuredListItemPaddingCondensed = "0 0 4px 0";
export const BStructuredListItemMarginBottom = "12px";
export const BStructuredListItemMarginBottomCondensed = "6px";
export const BStructuredListItemLabelPadding = "8px";
export const BStructuredListItemLabelPaddingMobile = "2px 0";
export const BStructuredListItemLabelColor = "#5c687c";
export const BStructuredListItemLabelGap = "8px";
export const BStructuredListItemLabelIconColor = "#00112c";
export const BStructuredListItemItemValueColor = "#00112c";
export const BStructuredListItemItemValueGap = "16px";
export const BStructuredListItemItemValueMobilePadding = "2px 0";
export const BStructuredListItemIconMargin = "2px";
export const BSummaryGridGap = "12px";
export const BSummaryGridItemBackgroundColor = "#f3f6f9";
export const BSummaryGridItemBorderRadius = "8px";
export const BSummaryGridItemPadding = "16px 16px 12px";
export const BSummaryGridItemGap = "8px";
export const BSummaryGridItemHeaderMarginBottom = "4px";
export const BSummaryGridItemTitleIconsGap = "12px";
export const BSummaryGridItemTitleIconsHeight = "24px";
export const BSummaryGridItemTitleIconsMarginLeft = "12px";
export const BSummaryGridItemContentLabelFontWeight = "500";
export const BSummaryGridItemContentLabelLineHeight = "24px";
export const BSummaryGridItemContentTrendMarginLeft = "8px";
export const BSummaryGridItemAmountSecondaryLabelPaddingLeft = "4px";
export const BTabsBackgroundColor = "#ffffff";
export const BTabsTabListGap = "24px";
export const BTabsTabListHeight = "44px";
export const BTabsTabTitleBorderRadius = "2px";
export const BTabsTabTitleColor = "#5c687c";
export const BTabsTabTitleActiveColor = "#00112c";
export const BTabsTabTitleDisabledColor = "#8d95a3";
export const BTabsTabTitleHoverColor = "#00112c";
export const BTabsTabUnderlineColor = "#00112c";
export const BTabsTabUnderlineHeight = "3px";
export const BTabsTabUnderlineHeightActive = "3px";
export const BTabsTabUnderlineWidth = "20px";
export const BTabsTabUnderlineBorderRadius = 0;
export const BTabsNotificationDotBackgroundColor = "#0070f5";
export const BTabsNotificationDotHeight = "7px";
export const BTabsNotificationDotMarginLeft = "6px";
export const BTabsNotificationDotWidth = "7px";
export const BTagPadding = "0 8px";
export const BTagDefaultColor = "#394962";
export const BTagDefaultBackgroundColor = "#dce0e5";
export const BTagBlueColor = "#0048b3";
export const BTagBlueBackgroundColor = "#e5efff";
export const BTagGreenColor = "#08a648";
export const BTagGreenBackgroundColor = "#e6f8ed";
export const BTagOrangeColor = "#ff9500";
export const BTagOrangeBackgroundColor = "#fff4e5";
export const BTagRedColor = "#a10000";
export const BTagRedBackgroundColor = "#fef5f5";
export const BTagWhiteColor = "#00112c";
export const BTagWhiteBackgroundColor = "#ffffff";
export const BTextareaMinWidth = "200px";
export const BTextareaLabelColor = "#00112c";
export const BTextareaLabelMarginBottom = "8px";
export const BTextareaLabelItemColor = "#5c687c";
export const BTextareaLabelItemErrorColor = "#e22d2d";
export const BTextareaInputBoxMarginBottom = "8px";
export const BTextareaInputBorder = "1px solid #8d95a3";
export const BTextareaInputBorderRadius = "8px";
export const BTextareaInputPadding = "12px 12px 8px 12px";
export const BTextareaInputColor = "#00112c";
export const BTextareaInputMinHeight = "44px";
export const BTextareaInputDisabledBackgroundColor = "#eeeff1";
export const BTextareaInputDisabledBorderColor = "#dbdee2";
export const BTextareaInputDisabledColor = "#8d95a3";
export const BTextareaInputErrorBackgroundColor = "#e22d2d";
export const BTextareaInputFocusBorder = "1.5px solid #00112c";
export const BTextareaInputHoverBorderColor = "#00112c";
export const BTextareaInputPlaceholderColor = "#8d95a3";
export const BTextareaInputReadonlyBackgroundColor = "#eeeff1";
export const BTextareaInputReadonlyBorderColor = "#dbdee2";
export const BTextareaInputClearablePadding = "12px 40px 8px 12px";
export const BTextareaDescriptionColor = "#5c687c";
export const BTextareaDescriptionMarginTop = "4px";
export const BTextareaDescriptionPadding = 0;
export const BTextareaDescriptionErrorColor = "#e22d2d";
export const BTimelineMaxWidth = "500px";
export const BTimelineAddNoteSectionPaddingBottom = "24px";
export const BTimelineItemRowGap = "8px";
export const BTimelineItemContentPaddingBottom = "16px";
export const BTimelineItemIconMargin = "4px 0";
export const BTimelineItemIconColor = "#00112c";
export const BTimelineItemIconDefaultColor = "#a5afbd";
export const BTimelineItemIconDefaultMargin = "4px 0";
export const BTimelineItemIconSuccessColor = "#07893c";
export const BTimelineItemIconCriticalColor = "#e22d2d";
export const BTimelineItemIconBlueColor = "#0070f5";
export const BTimelineItemIconGreenColor = "#09ab4b";
export const BTimelineItemIconGreyColor = "#c0c5cc";
export const BTimelineItemIconOrangeColor = "#dd7d00";
export const BTimelineItemIconRedColor = "#e22d2d";
export const BTimelineItemIconYellowColor = "#f7bc00";
export const BTimelineItemIconTimegapPaddingBottom = "4px";
export const BTimelineItemIconTimegapBorder = "1px solid #a5afbd";
export const BTimelineItemIconTimegapMargin = "6px auto";
export const BTimelineItemSeparatorBackgroundColor = "#a5afbd";
export const BTimelineItemTitleColor = "#00112c";
export const BTimelineItemTitleMarginBottom = "0";
export const BTimelineItemTitlePaddingTop = "4px";
export const BTimelineItemTimestampColor = "#69778c";
export const BTimelineItemTimestampGap = "6px";
export const BTimelineItemDescriptionColor = "#00112c";
export const BTimelineItemDescriptionMarginTop = "0";
export const BTimelineItemDataListMarginTop = "0";
export const BTimelineItemTagMarginTop = "10px";
export const BTimelineItemTimegapColor = "#69778c";
export const BTimelineNoteTimestampMarginTop = "10px";
export const BToastItemBackgroundColor = "#00112c";
export const BToastItemBorderColor = "#0070f5";
export const BToastItemBorderWidth = "1px";
export const BToastItemBorderRadius = "6px";
export const BToastItemColor = "#ffffff";
export const BToastItemIconColor = "#8390a3";
export const BToastItemIconMargin = "16px";
export const BToastItemPadding = "12px 16px";
export const BToastItemShadow = "transparent";
export const BToastItemZIndex = 15;
export const BToastItemQueuedBackgroundColor = "#8390a3";
export const BToggleHandleBackgroundColor = "#ffffff";
export const BToggleHandleHeight = "13px";
export const BToggleHandleWidth = "13px";
export const BToggleHandleBorderRadius = "12px";
export const BToggleHandleTransition = "transform 150ms cubic-bezier(0.2, 0, 0.4, 0.9)";
export const BToggleHandleDisabledBackgroundColor = "#8d95a3";
export const BToggleHandleToggledBackgroundColor = "#ffffff";
export const BToggleHandleToggledHeight = "13px";
export const BToggleHandleToggledWidth = "13px";
export const BToggleHandleToggledColor = "transparent";
export const BToggleHandleToggledOffset = "4px";
export const BToggleHandleToggledDisabledBackgroundColor = "#8390a3";
export const BToggleHandleToggledDisabledColor = "transparent";
export const BToggleHandleToggledReadonlyBackgroundColor = "#ffffff";
export const BToggleHandleToggledReadonlyColor = "transparent";
export const BToggleTrackHeight = "17px";
export const BToggleTrackWidth = "33px";
export const BToggleTrackPadding = "1px";
export const BToggleTrackBorder = "1px solid transparent";
export const BToggleTrackBorderRadius = "12px";
export const BToggleTrackBackgroundColor = "#a5afbd";
export const BToggleTrackMarginTop = "3px;";
export const BToggleTrackHoverBackgroundColor = "#a5afbd";
export const BToggleTrackHoverBorderColor = "#0048b3";
export const BToggleTrackActiveBackgroundColor = "#a5afbd";
export const BToggleTrackActiveBorderColor = "transparent";
export const BToggleTrackDisabledBackgroundColor = "#dce0e5";
export const BToggleTrackDisabledBorderColor = "#dbdee2";
export const BToggleTrackReadonlyBackgroundColor = "#dce0e5";
export const BToggleTrackReadonlyBorderColor = "#dbdee2";
export const BToggleTrackToggledBackgroundColor = "#0066ff";
export const BToggleTrackToggledPadding = "1px solid transparent";
export const BToggleTrackToggledBorder = "1px solid transparent";
export const BToggleTrackToggledHoverBackgroundColor = "#0066ff";
export const BToggleTrackToggledHoverBorderColor = "#0048b3";
export const BToggleTrackToggledActiveBackgroundColor = "#0066ff";
export const BToggleTrackToggledDisabledBackgroundColor = "#dce0e5";
export const BToggleTrackToggledReadonlyBackgroundColor = "#dce0e5";
export const BToggleLabelPadding = "10px";
export const BToggleDescriptionColor = "#707070";
export const BToggleDescriptionPadding = "4px";
export const BTooltipBackgroundColor = "#20304c";
export const BTooltipBorderRadius = "4px";
export const BTooltipColor = "#ffffff";
export const BTooltipMaxWidth = "400px";
export const BTooltipPadding = "4px 8px";
export const BTooltipZIndex = 5;
export const BTutorialPopoverColor = "#5c687c";
export const BTutorialPopoverStepsLineHeight = "26px";
export const BTutorialPopoverRippleBeforeBackgroundColor = "#09ab4b";
export const BTutorialPopoverRippleBeforeFilter = "blur(40px)";
export const BTutorialPopoverRippleBeforeHeight = "48px";
export const BTutorialPopoverRippleBeforeWidth = "64px";
export const BTutorialPopoverRippleBeforePosition = "-48px";
export const BTutorialPopoverRippleAfterHeight = "6px";
export const BTutorialPopoverRippleAfterWidth = "10px";
export const BTutorialPopoverRippleAfterPosition = "-24px";
export const BTutorialPopoverHeaderGap = "16px";
export const BTutorialPopoverHeaderPaddingBottom = "12px";
export const BTutorialPopoverFooterColumnGap = "8px";
export const BTutorialPopoverFooterPaddingTop = "12px";
export const BTutorialIndicatorBorder = "2px solid #09ab4b";
export const BTutorialIndicatorBorderOffset = "-2px";
export const BFontSize13 = "13px";
export const BFontSize15 = "15px";
export const BFontSize21 = "21px";
export const BFontSize30 = "30px";
export const BFontSize45 = "45px";
export const BFontSizeSmall = "13px";
export const BFontSizeRegular = "15px";
export const BFontSizeMedium = "21px";
export const BFontSizeLarge = "30px";
export const BFontSizeXlarge = "45px";
export const BFontWeight100 = "100";
export const BFontWeight400 = "400";
export const BFontWeight500 = "500";
export const BFontWeight600 = "600";
export const BFontWeightThin = "100";
export const BFontWeightRegular = "400";
export const BFontWeightMedium = "500";
export const BFontWeightSemiBold = "600";
export const BFontFamilyPrimary = "Fakt, -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif";
export const BAnimationEasingStandard = "cubic-bezier(0.2, 0, 0.4, 0.9)";
export const BAnimationEasingEnter = "cubic-bezier(0.12, 0.6, 0.4, 0.95)";
export const BAnimationEasingExit = "cubic-bezier(0.4, 0, 1, 1)";
export const BAnimationEasingLinear = "linear";
export const BAnimationDurationFast = "100ms";
export const BAnimationDurationModerate = "150ms";
export const BAnimationDurationSlow = "250ms";
export const BBorderRadiusXs = "2px";
export const BBorderRadiusS = "4px";
export const BBorderRadiusM = "8px";
export const BBorderRadiusL = "12px";
export const BBorderRadiusXl = "24px";
export const BBorderWidthS = "1px";
export const BBorderWidthM = "2px";
export const BBorderWidthL = "3px";
export const BMediaQueryXsMin = "0px";
export const BMediaQueryXsMax = "375px";
export const BMediaQuerySMin = "376px";
export const BMediaQuerySMax = "499px";
export const BMediaQueryMMin = "500px";
export const BMediaQueryMMax = "767px";
export const BMediaQueryLMin = "768px";
export const BMediaQueryLMax = "1279px";
export const BMediaQueryXlMin = "1280px";
export const BMediaQueryXlMax = "1535px";
export const BMediaQueryXxl = "1536px";
export const BShadowLow = "0px 2px 4px rgba(0, 17, 44, 0.04), 0px 1px 2px rgba(0, 17, 44, 0.02)";
export const BShadowMedium = "0px 6px 12px rgba(0, 17, 44, 0.08), 0px 2px 4px rgba(0, 17, 44, 0.04)";
export const BShadowHigh = "0px 18px 36px rgba(0, 17, 44, 0.12), 0px 4px 8px rgba(0, 17, 44, 0.06)";
export const BShadowLowBorder = "0px 2px 4px rgba(0, 17, 44, 0.04), 0px 1px 2px rgba(0, 17, 44, 0.02), 0px 0px 0px 1px rgba(0, 17, 44, 0.06)";
export const BShadowMediumBorder = "0px 6px 12px rgba(0, 17, 44, 0.08), 0px 2px 4px rgba(0, 17, 44, 0.04), 0px 0px 0px 1px rgba(0, 17, 44, 0.06)";
export const BShadowHighBorder = "0px 18px 36px rgba(0, 17, 44, 0.12), 0px 4px 8px rgba(0, 17, 44, 0.06), 0px 0px 0px 1px rgba(0, 17, 44, 0.06)";
export const BSpacer100 = "32px";
export const BSpacer110 = "40px";
export const BSpacer120 = "48px";
export const BSpacer130 = "56px";
export const BSpacer140 = "64px";
export const BSpacer000 = "0px";
export const BSpacer010 = "2px";
export const BSpacer020 = "4px";
export const BSpacer030 = "6px";
export const BSpacer040 = "8px";
export const BSpacer050 = "10px";
export const BSpacer060 = "12px";
export const BSpacer070 = "16px";
export const BSpacer080 = "20px";
export const BSpacer090 = "24px";
