// @adyen/bento-design-tokens/dist/js/adl/es6.js
export const AdlBackgroundInteractive = "#dee1e6";
export const AdlBoxSizingIncludeChildren = "border-box";
export const AdlColorPrimary30 = "#334156";
export const AdlColorPrimary40 = "#1a2941";
export const AdlColorPrimary50 = "#00112c";
export const AdlColorAttention10 = "#e5efff";
export const AdlColorWarning10 = "#fff4e5";
export const AdlColorError10 = "#fce5e5";
export const AdlColorError50 = "#e50000";
export const AdlColorError60 = "#a10000";
export const AdlColorError70 = "#720000";
export const AdlColorDestructive10 = "#fce5e5";
export const AdlColorDestructive50 = "#e50000";
export const AdlColorDestructive60 = "#a10000";
export const AdlColorDestructive70 = "#720000";
export const AdlColorNeutral0 = "#ffffff";
export const AdlColorNeutral10 = "#f7f8f9";
export const AdlColorNeutral20 = "#eef0f3";
export const AdlColorNeutral30 = "#dee1e6";
export const AdlColorNeutral40 = "#bfc4ca";
export const AdlColorNeutral50 = "#9fa6b0";
export const AdlColorNeutral60 = "#818890";
export const AdlColorNeutral65 = "#6f7988";
export const AdlColorNeutral70 = "#707070";
export const AdlColorNeutral80 = "#404d61";
export const AdlColorNeutral90 = "#202f46";
export const AdlColorNeutral110 = "#00112c";
export const AdlColorFocus = "#0066ff";
export const AdlColorSuccess10 = "#e6f8ed";
export const AdlColorSuccess40 = "#0abf53";
export const AdlColorSuccess50 = "#08a648";
export const AdlColorSuccess60 = "#055f29";
export const AdlColorSuccess70 = "#03431d";
export const AdlColorBlack = "#000000";
export const AdlColorWhite = "#ffffff";
export const AdlColorNavy30 = "#334156";
export const AdlColorNavy40 = "#1a2941";
export const AdlColorNavy50 = "#00112c";
export const AdlColorGrey0 = "#ffffff";
export const AdlColorGrey10 = "#f7f8f9";
export const AdlColorGrey20 = "#eef0f3";
export const AdlColorGrey30 = "#dee1e6";
export const AdlColorGrey40 = "#bfc4ca";
export const AdlColorGrey50 = "#9fa6b0";
export const AdlColorGrey60 = "#818890";
export const AdlColorGrey65 = "#6f7988";
export const AdlColorGrey70 = "#707070";
export const AdlColorGrey80 = "#404d61";
export const AdlColorGrey90 = "#202f46";
export const AdlColorGrey110 = "#00112c";
export const AdlColorRed10 = "#fce5e5";
export const AdlColorRed30 = "#fce5e5";
export const AdlColorRed50 = "#e50000";
export const AdlColorRed60 = "#a10000";
export const AdlColorRed70 = "#720000";
export const AdlColorBlue10 = "#e5efff";
export const AdlColorBlue50 = "#0066ff";
export const AdlColorBlue60 = "#0048b3";
export const AdlColorOrange10 = "#fff4e5";
export const AdlColorOrange50 = "#ff9500";
export const AdlColorGreen10 = "#e6f8ed";
export const AdlColorGreen20 = "#cef2dd";
export const AdlColorGreen40 = "#0abf53";
export const AdlColorGreen50 = "#08a648";
export const AdlColorGreen60 = "#055f29";
export const AdlColorGreen70 = "#03431d";
export const AdlColorTransparent = "#000000";
export const AdlComponentBaseFontFamily = "Fakt";
export const AdlComponentBaseFontSize = "14px";
export const AdlComponentBaseFontWeight = "500";
export const AdlComponentBaseBorderBoxIncludeChildren = "border-box";
export const AdlComponentBaseTextColor = "#000000";
export const AdlComponentBaseTextLineHeight = "24px";
export const AdlErrorRingColor = "#e50000";
export const AdlFocusRingColor = "#0066ff";
export const AdlTextStyleFormFieldsBodyMediumFontSize = "14px";
export const AdlTextStyleFormFieldsBodyMediumFontWeight = "500";
export const AdlTextStyleFormFieldsBodyMediumLineHeight = "22px";
export const AdlTextStyleFormFieldsBodySmallFontSize = "13px";
export const AdlTextStyleFormFieldsBodySmallFontWeight = "500";
export const AdlTextStyleFormFieldsBodySmallLineHeight = "22px";
export const AdlTextStyleFormFieldsCaptionMediumFontSize = "12px";
export const AdlTextStyleFormFieldsCaptionMediumFontWeight = "500";
export const AdlTextStyleFormFieldsCaptionMediumLineHeight = "16px";
export const AdlTextStyleFormFieldsCaptionSmallFontSize = "10px";
export const AdlTextStyleFormFieldsCaptionSmallFontWeight = "500";
export const AdlTextStyleFormFieldsCaptionSmallLineHeight = "16px";
export const AdlTextStyleHeadingMediumFontSize = "16px";
export const AdlTextStyleHeadingMediumFontWeight = "500";
export const AdlTextStyleHeadingMediumLineHeight = "24px";
export const AdlTextStyleParagraphMediumFontSize = "14px";
export const AdlTextStyleParagraphMediumFontWeight = "400";
export const AdlTextStyleParagraphMediumLineHeight = "22px";
export const AdlTextStyleParagraphSmallFontSize = "13px";
export const AdlTextStyleParagraphSmallFontWeight = "400";
export const AdlTextStyleParagraphSmallLineHeight = "22px";
export const AdlSubHeadingMediumFontSize = "12px";
export const AdlSubHeadingMediumFontWeight = "500";
export const AdlSubHeadingMediumLineHeight = "22px";
export const AdlTextColor = "#000000";
export const AdlTextLineHeight = "24px";
export const AdlFontSize10 = "10px";
export const AdlFontSize12 = "12px";
export const AdlFontSize13 = "13px";
export const AdlFontSize14 = "14px";
export const AdlFontSize16 = "16px";
export const AdlFontSize18 = "18px";
export const AdlFontSize20 = "20px";
export const AdlFontSize24 = "24px";
export const AdlFontSizeMicro = "10px";
export const AdlFontSizeXxsmall = "12px";
export const AdlFontSizeXsmall = "13px";
export const AdlFontSizeSmall = "14px";
export const AdlFontSizeRegular = "16px";
export const AdlFontSizeMedium = "18px";
export const AdlFontSizeLarge = "20px";
export const AdlFontSizeXlarge = "24px";
export const AdlFontWeight100 = "100";
export const AdlFontWeight400 = "400";
export const AdlFontWeight500 = "500";
export const AdlFontWeight600 = "600";
export const AdlFontWeightThin = "100";
export const AdlFontWeightRegular = "400";
export const AdlFontWeightMedium = "500";
export const AdlFontWeightSemiBold = "600";
export const AdlFontFamilyFakt = "Fakt";
export const AdlFontFamilyInter = "Inter";
export const AdlLineHeight16 = "16px";
export const AdlLineHeight20 = "20px";
export const AdlLineHeight22 = "22px";
export const AdlLineHeight24 = "24px";
export const AdlLineHeight28 = "28px";
export const AdlLineHeight32 = "32px";
export const AdlLineHeightXsmall = "16px";
export const AdlLineHeightSmall = "20px";
export const AdlLineHeightRegular = "22px";
export const AdlLineHeightMedium = "24px";
export const AdlLineHeightLarge = "28px";
export const AdlLineHeightXlarge = "32px";
export const AdlAccordionBorderColor = "#dee1e6";
export const AdlAccordionBorderRadius = "6px";
export const AdlAccordionItemBackgroundColor = "#ffffff";
export const AdlAccordionItemMargin = "40px";
export const AdlAccordionItemPadding = "8px";
export const AdlAccordionItemGroupedPadding = "12px";
export const AdlAccordionItemHeaderPadding = "12px";
export const AdlAccordionItemHeaderMarginBottom = "20px";
export const AdlAccordionItemHeaderColor = "#00112c";
export const AdlAccordionItemHeaderHoverBackgroundColor = "#f7f8f9";
export const AdlAccordionItemHeaderHoverBorderRadius = "4px";
export const AdlAccordionItemHeaderPictureMargin = "1rem";
export const AdlAccordionItemHeaderPictureSize = "40px";
export const AdlAccordionItemHeaderTitleColor = "#00112c";
export const AdlAccordionItemHeaderTitleFontWeight = "500";
export const AdlAccordionItemHeaderTitleFontSize = "14px";
export const AdlAccordionItemHeaderSubtitleFontWeight = "400";
export const AdlAccordionItemHeaderSubtitleFontSize = "12px";
export const AdlAccordionItemHeaderSubtitleColor = "#707070";
export const AdlAccordionItemCaretMarginRight = "10px";
export const AdlAccordionItemCaretMarginBottom = "2px";
export const AdlAccordionItemDescriptionPaddingX = "8px";
export const AdlAccordionItemDescriptionPaddingY = "12px";
export const AdlAccordionItemContentFontWeight = "400";
export const AdlAccordionItemContentFontSize = "14px";
export const AdlAccordionItemContentColor = "#202f46";
export const AdlAccordionItemContentLineHeight = "24px";
export const AdlBadgeFontWeight = "400";
export const AdlBadgeGap = "6px";
export const AdlBadgeHeight = "24px";
export const AdlBadgePadding = "0 8px";
export const AdlBadgeDefaultBackgroundColor = "#eef0f3";
export const AdlBadgeAttentionBackgroundColor = "#e5efff";
export const AdlBadgeWarningBackgroundColor = "#fff4e5";
export const AdlBadgeCriticalBackgroundColor = "#fce5e5";
export const AdlBadgeSuccessBackgroundColor = "#e6f8ed";
export const AdlPopoverBackgroundColor = "#ffffff";
export const AdlPopoverArrowDisplay = "none";
export const AdlPopoverBorderColor = "#dee1e6";
export const AdlPopoverBoxShadow = "0px 1px 2px rgba(0, 0, 0, 0.05)";
export const AdlPopoverContentPadding = "8px 12px";
export const AdlPopoverFontSize = "12px";
export const AdlPopoverFontWeight = "400";
export const AdlSegmentedControlBackgroundColor = "#eef0f3";
export const AdlSegmentedControlBorder = "none";
export const AdlSegmentedControlBorderRadius = "6px";
export const AdlSegmentedControlPadding = "2px";
export const AdlSegmentedOptionBorderRadius = "4px";
export const AdlSegmentedOptionColor = "#00112c";
export const AdlSegmentedOptionFontSize = "14px";
export const AdlSegmentedOptionGap = "6px";
export const AdlSegmentedOptionPadding = "6px 24px";
export const AdlSegmentedOptionHeight = "36px";
export const AdlSegmentedOptionSelectedBackgroundColor = "#ffffff";
export const AdlSegmentedOptionSelectedBoxShadow = "0 1px 3px rgb(16 24 40 / 10%), 0 1px 2px rgb(16 24 40 / 6%);";
export const AdlSegmentedOptionSelectedColor = "inherit";
export const AdlSegmentedOptionDisabledColor = "#9fa6b0";
export const AdlSegmentedOptionDisabledSelectedBackgroundColor = "#ffffff";
export const AdlSegmentedOptionDisabledSelectedColor = "#9fa6b0";
export const AdlSegmentedOptionBeforeBorderLeft = "0px";
export const AdlSegmentedOptionHoverBackgroundColor = "transparent";
export const AdlTabsBorderBottomColor = "#eef0f3";
export const AdlTabsHandleFontSize = "14px";
export const AdlTabsHandleMarginRight = "16px";
export const AdlTabsHandlePadding = "0 12px 12px 12px";
export const AdlTabsHandleHoverBackgroundColor = "#f7f8f9";
export const AdlTabsHandleHoverBoxShadowHeight = "2px";
export const AdlTabsHandleHoverBoxShadowColor = "#ffffff";
export const AdlTabsHandleActiveColor = "#00112c";
export const AdlTabsHandleActiveBoxShadowHeight = "2px";
export const AdlTabsHandleActiveBoxShadowColor = "#00112c";
export const AdlTabsHandleDisabledColor = "#707070";
export const AdlTabsHandleTitlePadding = "6px 12px";
export const AdlTabsHandleTitleHeight = "36px";
export const AdlTextareaFontWeight = "400";
export const AdlTextareaDescriptionBackgroundColor = "unset";
export const AdlTextareaDescriptionColor = "#707070";
export const AdlTextareaDescriptionDisplay = "block";
export const AdlTextareaDescriptionFocusDisplay = "block";
export const AdlTextareaDescriptionMarginTop = "8px";
export const AdlTextareaDescriptionPadding = 0;
export const AdlTextareaLabelColor = "#00112c";
export const AdlTextareaLabelFontSize = "14px";
export const AdlTextareaLabelFontWeight = "500";
export const AdlTextareaLabelLineHeight = "inherit";
export const AdlTextareaLabelMarginBottom = "8px";
export const AdlTextareaInputColor = "#202f46";
export const AdlTextareaInputPlaceholderColor = "#6f7988";
export const AdlTextareaInputBorderColor = "#dee1e6";
export const AdlTextareaInputBorderRadius = "8px";
export const AdlTextareaInputFontSize = "14px";
export const AdlTextareaInputMinWidth = "182px";
export const AdlTextareaInputMinHeight = "90px";
export const AdlTextareaInputPadding = "6px 12px";
export const AdlSpacing1 = "0.125rem";
export const AdlSpacing2 = "0.25rem";
export const AdlSpacing3 = "0.5rem";
export const AdlSpacing4 = "1rem";
export const AdlSpacing5 = "1.5rem";
export const AdlSpacing6 = "2rem";
export const AdlSpacing7 = "2.5rem";
export const AdlSpacing8 = "3rem";
export const AdlSpacing9 = "3.5rem";
export const AdlSpacing10 = "4rem";
export const AdlSpacing11 = "4.5rem";
export const AdlSpacing12 = "5rem";
export const AdlSpacing13 = "5.5rem";
export const AdlSpacing14 = "6rem";
export const AdlSpacing15 = "6.5rem";
export const AdlSpacing16 = "7rem";
export const AdlSpacing17 = "7.5rem";
